{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This notebook investigates output from `langchain` pdf loaders and splitters. Loaders are responsible for processing raw pdf files, while splitters define heuristics for chunking documents for more efficient processing by LLMs."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import time\n",
    "from langchain.docstore.document import Document\n",
    "from langchain.text_splitter import RecursiveCharacterTextSplitter\n",
    "\n",
    "pdf_dir = \"/Users/kkumbier/github/persisters/papers/\"\n",
    "paper = \"Hata/Cabanos2021.pdf\"\n",
    "pdf_file = os.path.join(pdf_dir, paper)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# PDF loaders\n",
    "Below we examine the output of various pdf loaders with default splitters."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.9931268692016602\n",
      "19\n",
      "page_content='A UGUST  2021\\u2003 CANCER DISCOVERY \\u2002|\\u2002OF1 \\nResea Rch B Rief\\nClinical Acquired Resistance to KRASG12C \\nInhibition through a Novel KRAS Switch-II \\nPocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation  \\n \\nNoritaka Tanaka1, Jessica J. Lin1, Chendi Li1, Meagan B. Ryan1, Junbing Zhang1, Lesli A. Kiedrowski2,  \\nAlexa G. Michel1, Mohammed U. Syed1, Katerina A. Fella1, Mustafa Sakhi1, Islam Baiev1, Dejan Juric1,  \\nJustin F . Gainor1, Samuel J. Klempner1, Jochen K. Lennerz3, Giulia Siravegna1, Liron Bar-Peled1,  \\nAaron N. Hata1, Rebecca S. Heist1, and Ryan B. Corcoran1\\naBstRact Mutant-selective KRASG12C inhibitors, such as MRTX849 (adagrasib) and AMG 510 \\n(sotorasib), have demonstrated efficacy in KRASG12C-mutant cancers, including \\nnon–small cell lung cancer (NSCLC). However, mechanisms underlying clinical acquired resistance to \\nKRASG12C inhibitors remain undetermined. To begin to define the mechanistic spectrum of acquired \\nresistance, we describe a patient with KRASG12C NSCLC who developed polyclonal acquired resistance \\nto MRTX849 with the emergence of 10 heterogeneous resistance alterations in serial cell-free DNA spanning four genes (KRAS, NRAS, BRAF , MAP2K1), all of which converge to reactivate RAS–MAPK signaling. Notably, a novel KRAS\\nY96D mutation affecting the switch-II pocket, to which MRTX849 and \\nother inactive-state inhibitors bind, was identified that interferes with key protein–drug interactions and confers resistance to these inhibitors in engineered and patient-derived KRAS\\nG12C cancer models. \\nInterestingly, a novel, functionally distinct tricomplex KRASG12C active-state inhibitor RM-018 retained \\nthe ability to bind and inhibit KRASG12C/Y96D and could overcome resistance.\\nS\\nIGNI\\nf\\nICANC\\ne: In one of the first reports of clinical acquired resistance to KRASG12C inhibitors, our \\ndata suggest polyclonal RAS–MAPK reactivation as a central resistance mechanism. We also identify a novel KRAS switch-II pocket mutation that impairs binding and drives resistance to inactive-state inhibitors but is surmountable by a functionally distinct KRAS\\nG12C inhibitor.\\n1Massachusetts General Hospital Cancer Center and Department of  \\nMedicine, Harvard Medical School, Boston, Massachusetts. 2Guardant Health, \\nRedwood City, California. 3Department of Pathology, Massachusetts  \\nGeneral Hospital, Boston, Massachusetts.\\nNote: Supplementary data for this article are available at Cancer Discovery \\nOnline (http://cancerdiscovery.aacrjournals.org/).\\nN. Tanaka, J.J. Lin, and C. Li contributed equally to this work.Corresponding Authors: Aaron N. Hata, Massachusetts General Hospital \\nCancer Center, 149 13th Street,  7th Floor, Boston, MA 02129. Phone: 617-724-3442; E-mail: ahata@mgh.harvard.edu; Rebecca S. Heist, Massachu\\n-\\ns\\netts General Hospital Cancer Center, 55 Fruit Street, Boston, MA 02114. \\nPhone: 617-724-4000; E-mail: rheist@partners.org; and Ryan B. Corcoran,  \\nMassachusetts General Hospital Cancer Center, 149 13th Street, 7th Floor, Boston, MA 02129. Phone: 617-726-8599; E-mail: rbcorcoran@partners.org\\nCancer Discov 2021;11:1\\n–10\\ndoi: 10.1158/2159-8290.CD-21-0365\\n©2021 American Association for Cancer ResearchiNtRODU cti ON\\nThe development of compounds that bind covalently to \\ncysteine 12 in KRASG12C cancers has ushered in a new era in \\nefforts to target KRAS directly. Biochemically, these agents \\nlock KRAS in its inactive GDP-bound conformation, thereby inhibiting downstream signaling, leading to preclinical  antitumor responses (1–3). The lead clinical compounds soto-rasib (AMG 510) and adagrasib (MRTX849) have advanced rapidly and demonstrated tolerability and single-agent activity across KRAS\\nG12C-mutant cancers (4, 5). In patients \\nwith advanced non–small cell lung cancer (NSCLC) harbor -\\ning KRASG12C (which comprises approximately 13% of all  \\nlung adenocarcinomas), AMG 510 and MRTX849 have' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 0}\n",
      "page_content='with advanced non–small cell lung cancer (NSCLC) harbor -\\ning KRASG12C (which comprises approximately 13% of all  \\nlung adenocarcinomas), AMG 510 and MRTX849 have  \\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 0}\n",
      "page_content='Tanaka et al.ReSeARCH BRI ef\\nOF2 \\u2002|\\u2002CANCER DISCOVERY \\u2003AUGUST  2021  AACRJournals.orgdemonstrated meaningful efficacy with objective response \\nrates of 37% and 45%, as well as disease control rates of 81% and 96%, respectively (6, 7). AMG 510 has recently received Breakthrough Therapy designation from the FDA for the treatment of patients with advanced KRAS\\nG12C-mutant \\nNSCLC following at least one prior systemic therapy. Mul-tiple ongoing trials seek to augment responses to KRAS\\nG12C \\ninhibitors through combination strategies.\\nPreclinical studies with MRTX849 and other KRASG12C \\ninhibitors have suggested several mechanisms of up-front resistance, including reactivation of ERK-dependent sig\\n-\\nnaling \\nto bypass KRASG12C blockade (4). Prior work by our \\ngroup and others has identified adaptive RAS pathway feed-back reactivation as a key mechanism of primary resistance to KRAS\\nG12C inhibition (8–13). However, the key mecha-\\nnisms of clinical acquired resistance to KRASG12C inhibitors \\nare currently unknown. Here, as an initial effort to charac-terize the clinical landscape of potential acquired resistance mechanisms to KRAS\\nG12C inhibitors, we present a patient \\nwith KRASG12C-mutant NSCLC who developed acquired \\nresistance to MRTX849, characterized by the emergence of 10 individual resistance alterations involving four RAS–MAPK genes. All of these resistance alterations converge to reactivate RAS–MAPK signaling, implicating this as a potential central mechanism of acquired resistance. We also identify a novel KRAS\\nY96D resistance mutation in the \\nswitch-II pocket of KRAS through serial cell-free DNA (cfDNA) analysis. Through structural modeling and in vitro  \\nfunctional studies, we find that KRAS\\nY96D confers resist-\\nance to multiple KRASG12C inhibitors currently in clinical \\ndevelopment but identify a novel active-state KRASG12C \\ninhibitor, RM-018, that is able to overcome KRASG12C/Y96D-\\nmediated resistance.\\nResULts\\nHeterogeneous Acquired Resistance Alterations \\nConverge on RAS–MAPK Reactivation\\nA 67-year-old woman with metastatic KRASG12C-mutant \\nNSCLC was treated on the dose-expansion cohort of the \\nphase 1 study of MRTX849 (NCT03785249; Methods; fur -\\nther detailed in Supplementary Methods). Initial scans showed a 32% reduction in tumor size (by RECIST v1.1), but after approximately 4 months of treatment, the patient developed progressive disease, and the patient discontinued therapy at 5.5 months (Fig. 1A). To identify putative mecha-nisms of acquired resistance to MRTX849 in this patient, we assessed cfDNA using a targeted next-generation sequencing assay (Guardant360; Guardant Health) and droplet digital PCR (ddPCR). Upon development of acquired resistance, the original KRAS\\nG12C and TP53F338fs variants present in pretreat-\\nment tumor and cfDNA were again detected in cfDNA but were accompanied by the emergence of 10 distinct muta-tions affecting RAS–MAPK components KRAS, NRAS, BRAF, and MAP2K1 (which encodes the MEK1 protein) identified \\nacross cfDNA specimens obtained after disease progression (Fig. 1B; Supplementary Table S1). The lower allele frequen-cies of these alterations relative to the truncal KRAS\\nG12C \\nand TP53 mutations are consistent with the emergence of these mutations in heterogeneous subclonal populations. These included three activating NRAS mutations (NRAS\\nQ61L, \\nNRASQ61K, NRASQ61R), which can drive active RAS signaling \\nin a KRAS-independent manner, and BRAFV600E, which can \\nmaintain MAPK signaling downstream of KRASG12C in the \\npresence of MRTX849 (Supplementary Fig. S1). Three MAP2K1   \\nmutations (MAP2K1K57N, MAP2K1Q56P, MAP2K1E102-I103del) pre -\\nviously demonstrated to be activating and kno\\nwn to be \\ninvolved in resistance to upstream MAPK pathway inhibitors (i.e., BRAF inhibitors) were also identified (14, 15).\\nIn addition, three KRAS mutations emerged in the postpro-\\ngression cfDNA. Two of these mutations are the known activat -\\ning mutations KRAS\\nG13D and KRASG12V, and mutant-selective' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 1}\n",
      "page_content='In addition, three KRAS mutations emerged in the postpro-\\ngression cfDNA. Two of these mutations are the known activat -\\ning mutations KRAS\\nG13D and KRASG12V, and mutant-selective \\nKRASG12C inhibitors have previously been shown to be inef-\\nfective against these mutations (4, 16). A deeper analysis of individual sequencing reads from cfDNA suggested that these mutations seemed to occur in trans to the original KRAS\\nG12C \\nmutation (Supplementary Fig. S2A and S2B), likely arising in the remaining wild-type copy of KRAS, which appeared to be retained based on pretreatment tumor sequencing (Sup-plementary Table S2). However, it is not possible from the cfDNA data to confirm that these mutations coexist in cells that also harbor the original KRAS\\nG12C mutation. Notably, a \\nsingle, well-supported family of sequencing reads from the same original template molecule showed the concurrent pres-ence of both nucleotide changes corresponding to KRAS\\nG12C \\nand KRASG12V in cis on the same strand, which would encode \\nfor a KRASG12F mutation. While it is not possible to confirm \\nthe presence of this mutation based on a single read family, this finding raises the possibility that cis mutations resulting \\nin “loss” of the original KRAS\\nG12C mutation and conversion \\nto a different KRAS mutation might be another potential mechanism of resistance. Notably, all putative resistance mutations identified are predicted to converge on reactiva-tion of RAS–MAPK pathway signaling, suggesting that this may represent a common primary mechanism of acquired resistance to KRAS\\nG12C inhibitors (Fig. 1C).\\nInterestingly, the third KRAS mutation identified, KRASY96D,  \\nrepresents a novel mutation that is not known to be activat-ing. Notably, although KRAS is the most commonly mutated oncogene in human cancer, a search of two large tumor muta-tional databases—COSMIC and GENIE, which collectively contain >450,000 molecularly characterized cancers (17, \\n18)—did not reveal a single previously identified mutation at the KRAS\\nY96 locus among >75,000 cases with documented \\nKRAS mutations (Supplementary Table S3). However, the Y96 residue is associated with the switch-II pocket to which MRTX849 and other inactive-state KRAS\\nG12C inhibitors bind, \\nsuggesting that the previously undescribed Y96D mutation may have a novel and specific role in driving resistance to KRAS\\nG12C inhibitors.\\nStructural Modeling of KRASG12C/Y96D\\nTo understand the significance of the acquired KRASY96D \\nmutation, we performed structural modeling of the G12C-mutant and G12C/Y96D double-mutant KRAS proteins bound to the KRAS\\nG12C inhibitors MRTX849, AMG 510, and \\nARS-1620 (Fig. 2). These three inhibitors bind the GDP state of KRAS\\nG12C and exploit a cryptic pocket formed by the central \\nβ sheet of RAS and switch-II (first identified by Ostrem and \\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 1}\n",
      "page_content='Clinical Acquired Resistance to KRASG12C InhibitionReSeARCH BRI ef\\n A UGUST  2021\\u2003 CANCER DISCOVERY \\u2002|\\u2002OF3 \\nfigure 1.\\u2002 Acquired resistance to KRASG12C inhibitor MRTX849 (adagrasib). A, Computed tomography images of the patient’s axillary lymph node \\nmetastasis at baseline, during response to MRTX849, and at progression on MRTX849. B, Variant allele fractions of mutations detected in the patient’s \\nserial plasma samples. †, indicates the mutations were detected by ddPCR but not by plasma next-generation sequencing. C, Alterations detected in post-MRTX849 cfDNA include acquired mutations in KRAS as well as multiple components of the MAPK signaling cascade. *, KRAS\\nG12F represents a potential \\nresistance mechanism supported by limited sequencing reads, as shown in Supplementary Fig. S2.A\\nBaseline 6 weeks 4 months \\nMRTX849 \\nPre-\\nMRTX849Pre-\\nMRTX849Days post-MRTX849\\ndiscontinuation:\\n09 51\\nTP53 F338fs 36.8% 0.22% 8.8% 10.1% 14.3%\\nKRAS G12C 21.3% 0.12% 31.7% 47.1% 24.9%\\nKRAS G12V -- -- 0.09%\\nKRAS G13D -- - 0.13%†0.04%\\nKRAS Y96D -- 0.4% 0.2% -\\nNRAS Q61L -- - 0.2% -\\nNRAS Q61R -- -- 0.02%\\nNRAS Q61K -- 0.6% 0.6% 0.9%\\nBRAF V600E -- 0.1% 0.1% 0.5%\\nMAP2K1 K57N -- 0.05%†- 0.3%\\nMAP2K1 Q56P -- -- 0.1%\\nMAP2K1 E102_I103del -- - 0.12%†0.2%cfDNA Tumor B C\\nA/B/ \\nCRAF KRAS \\n G12C \\nMEK RTK \\nProliferation and Survival  \\nC\\n RTK \\nWT \\nRAS \\nERK NRAS Q61K \\nNRAS Q61R \\nNRAS Q61L \\nKRAS G12V \\nKRAS G13D KRAS G12F* KRAS Y96D \\nBRAF V600E \\nMAP2K1 K57N MAP2K1 Q56P \\nMAP2K1 E102_I103 del Secondary  \\nCis* \\nTrans \\ncolleagues; ref. 3). To determine the effects of the amino acid \\nsubstitution at the Y96 locus, crystal structures of MRTX849, AMG 510, and ARS-1620 bound to KRAS\\nG12C were modeled \\nfor interactions with the Y96 residue within the switch-II pocket (16, 19–21). The hydroxyl group of Y96 forms a direct hydrogen bond with the pyrimidine ring of MTRX849, which is abolished with the Y96D mutation. Y96D also disrupts the water-mediated hydrogen bond between Y96 and a carboxyl group on AMG 510. Finally, although Y96 does not form a direct hydrogen bond with ARS-1620, it stabilizes the inter -\\naction with ARS-1620 through pi-stacking with the phenyl ring of Y96, which is disrupted with the Y96D mutation. In addition, by introducing a negatively charged amino acid, the Y96D mutation changes the hydrophobic nature of the bind-ing pocket for all three compounds to a substantially more hydrophilic pocket, which may further destabilize binding.\\nfunctional Characterization of KRASY96D\\nTo assess whether KRASY96D can mediate resistance to \\nMRTX849 and other inactive-state KRASG12C inhibitors, we \\nexpressed KRASG12C or the KRASG12C/Y96D double mutant \\nin NCI-H358 (KRASG12C-mutant NSCLC), MIA PaCa-2 \\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 2}\n",
      "page_content='Tanaka et al.ReSeARCH BRI ef\\nOF4 \\u2002|\\u2002CANCER DISCOVERY \\u2003AUGUST  2021  AACRJournals.orgfigure 2.\\u2002 Structural basis for resistance to KRASG12C inhibition conferred by KRASY96D. Shown are the modeled crystal structures of MRTX849 (6UT0), \\nAMG 510 (6OIM), and ARS-1620 (5V9U) bound to KRASG12C (top) and KRASG12C/Y96D (bottom), highlighting the loss of the hydrogen bonds between \\nMRTX849 or AMG 510 and the Y96 residue and the disruption of the switch-II pocket dynamics between ARS-1620 and KRASG12C/Y96D.\\nARS-1620 MRTX849 AMG 510G12C G12C/Y96D\\n(KRASG12C-mutant pancreatic ductal adenocarcinoma), and \\nBa/F3 cells, which lack endogenous KRASG12C but become \\noncogene dependent upon withdrawal of IL3. In cell via-\\nbility assays, relative to KRASG12C-expressing controls, cells \\nexpressing KRASG12C/Y96D showed marked resistance to \\nthree KRASG12C inhibitors, with IC 50 shifts of >100-fold for \\nMRTX849 and AMG 510 and ∼20-fold for ARS-1620 (Fig. 3A; Supplementary Table S4).\\nConsistent with the effects on cell viability, RAS–MAPK \\npathway activity, as measured by levels of phosphorylated  \\nERK (pERK) and pRSK, was sustained in KRAS\\nG12C/Y96D-\\nexpressing MIA PaCa-2 cells even at high concentrations of MRTX849, relative to cells expressing KRAS\\nG12C alone \\n(Fig. 3B). Similarly, in KRASG12C-mutant NSCLC cells in \\nwhich PI3K signaling is driven by mutant KRAS, including an existing patient-derived model MGH1138-1, persistent pERK and pAKT levels were observed with KRAS\\nG12C/Y96D \\nin the presence of MRTX849, relative to KRASG12C expres-\\nsion alone (Fig. 3C; Supplementary Fig. S3). KRASG12C/Y96D \\nalso drove marked resistance to MRTX849 in the patient-derived MGH1138-1 model. Furthermore, in 293T cells, which lack endogenous KRAS\\nG12C expression, MRTX849 \\nwas unable to inhibit pERK levels driven by KRASG12C/Y96D  \\n(Fig. 3D). Because MRTX849 and other inactive-state KRAS\\nG12C inhibitors bind covalently to KRASG12C, an elec-\\ntrophoretic mobility shift of drug-adducted KRASG12C can \\nbe observed upon drug binding due to increased molecular weight. However, this mobility shift was no longer observed when 293T cells expressing KRAS\\nG12C/Y96D were treated with \\nMRTX849, suggesting that the Y96D mutation may abro-gate inhibitor binding. Notably, KRAS\\nG12C/Y96D appeared to \\nhave higher basal activation than KRASG12C, as measured \\nby a higher proportion of the active GTP-bound form of KRAS, although activation still appeared to be partly depend-ent on upstream pathway input (Supplementary Fig. S4A and S4B). Finally, although a decrease in GTP-bound KRAS (representing the active state) was observed in KRAS\\nG12C-expressing cells treated with MRTX849, levels of active  \\nGTP-bound KRAS were maintained in KRASG12C/Y96D-expressing \\ncells (Fig. 3E; ref. 22). These results suggest that the KRASY96D \\nmutation disrupts KRASG12C inhibitor binding, leading to \\nsustained KRAS signaling and therapeutic resistance.\\nThe Active State KRASG12C Inhibitor RM-018 \\nOvercomes KRASG12C/Y96D\\nAs KRASG12C/Y96D conferred resistance to multiple KRASG12C \\ninhibitors currently in clinical development, suggestive of shared vulnerability for this class of inhibitors, we sought to identify whether a structurally and functionally distinct KRAS\\nG12C inhibitor might retain potency against this resist-\\nance mutation. RM-018 is a novel KRASG12C inhibitor that \\nbinds specifically to the GTP-bound, active [“RAS(ON)”] state of KRAS\\nG12C. RM-018 is a “tricomplex” KRAS inhibitor, which \\nexploits a highly abundant chaperone protein, cyclophilin A, to bind and inhibit KRAS\\nG12C, as previously described (Fig. 4A;  \\nstructure shown in Supplementary Fig. S5; refs. 23, 24). Briefly, upon entering the cell, RM-018 forms a “binary com-plex” with cyclophilin A. This binary complex can associate with the active state of KRAS\\nG12C, aided by protein–protein \\nsurface interactions between cyclophilin A and KRAS, and forms a covalent bond with KRAS\\nG12C in a mutant-selec-' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 3}\n",
      "page_content='G12C, aided by protein–protein \\nsurface interactions between cyclophilin A and KRAS, and forms a covalent bond with KRAS\\nG12C in a mutant-selec-\\ntive manner. This resultant “tricomplex” inhibits KRASG12C \\nthrough binding of cyclophilin A, leading to steric occlu-sion and preventing the association of downstream effector proteins. Given the markedly different mechanism of action of this class of inhibitor, we hypothesized that the inhibitory activity of RM-018 might be differentially affected by KRAS\\nY96D \\ncompared with inactive-state KRASG12C inhibitors.\\nRM-018 demonstrated selectivity for KRASG12C-driven \\ncells, exhibiting low nanomolar potency in KRASG12C-mutant \\nH358 cells while not impairing the viability of cells driven by KRAS\\nG12D, BRAFV600E, or RTK-driven signaling through \\nwild-type RAS (Fig. 4B). Interestingly, although KRASG12C/Y96D  \\nexpression led to marked IC 50 shifts of > 100-fold for \\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 3}\n",
      "page_content='Clinical Acquired Resistance to KRASG12C InhibitionReSeARCH BRI ef\\n A UGUST  2021\\u2003 CANCER DISCOVERY \\u2002|\\u2002OF5 \\nfigure 3.\\u2002 Cellular characterization of KRASY96D in KRASG12C-mutant models. A, Cell viability assays performed with NCI-H358, MIA PaCa-2, and Ba/F3 \\ncells infected with retrovirus packaging KRAS (G12C or G12C/Y96D). Cell lines were treated with indicated drugs for 72 hours and the viabilities were \\nmeasured with CellTiter-Glo. B, Western blot analysis was performed after treating MIA PaCa-2 cells stably expressing KRASG12C or KRASG12C/Y96D with \\nMRTX849 for 4 hours. C, MGH1138-1 cells expressing KRASG12C or KRASG12C/Y96D were treated with MRTX849 for 4 hours and subjected to Western blot \\nanalysis (left) and cell viability assay following 72 hours of treatment with the indicated concentrations of MRTX849 (right). D, Western blot analysis of HEK293T cells transiently expressing KRAS mutants after treatment with MRTX849 for 4 hours. \\ne\\n, RAS-GTP pulldown was performed after treating \\nHEK293T stably expressing KRAS mutants with MRTX849.B\\n0 3 10 30 100 300 0 3 10 30 100 300 G12C G12C/Y96D MRTX849\\n(nmol/L):MIA PaCa-2 \\npERK\\npRSK\\nGAPDH\\nERAF-RBD Input \\npERK\\npRSKKRASKRAS-GTP\\nGAPDHRASRAS-GTPMRTX849\\n(nmol/L):G12C \\n0 3 10 100 30 300 G12C/Y96D \\n0 3 10 100 30 300 \\n293T KRAS\\npERK\\npRSK0 3 10 30 100 300 0 3 10 30 100 300 MRTX849\\n(nmol/L):G12C G12C/Y96D \\nGAPDH\\nDA\\n293T \\nCAMG 510 MRTX849 Relative Cell Titer (%) \\nLog ARS-1620 (mol/L) ARS-1620 H358 MIA PaCa-2 KRASG12C\\nKRASG12C/Y96D BaF3 Log ARS-1620 (mol/L) \\nLog AMG 510 (mol/L) Log MRTX849 (mol/L) Log ARS-1620 (mol/L) Log MRTX849 (mol/L) Log AMG 510 (mol/L) \\nLog AMG 510 (mol/L) \\npERK\\nERK MRTX849\\n(nmol/L):\\nGAPDHpAKT\\nAKT\\n0 30 300 0 30 300 G12C G12C/Y96D MGH1138-1 \\n150\\n100\\n50\\n0Relative Cell Titer (%) 150100\\n50\\n0Relative Cell Titer (%) 150100\\n50\\n0150100\\n50\\n0150100\\n50\\n0150100\\n50\\n0150100\\n50\\n0150100\\n50\\n0150100\\n50\\n0\\n−11 −10 −9 −8 −7 −6\\nLog MRTX849 (mol/L) −11\\n−12 −11 −10 −9 −8 −7 −11 −10 −9 −8 −7 −6 −10 −9 −8 −7 −6 −5−10 −9 −8 −7 −6 −11 −10 −10 −9 −8 −7 −6 −5 −9 −8 −7 −6−11 −10 −10 −9 −8 −7 −6 −5 −9 −8 −7 −6\\nKRASG12C \\nKRASG12C/Y96D \\n−9 −8 −7 −6 −5050100150\\nLog MRTX849 (mol/L)Relative Cell Titer (%)\\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 4}\n",
      "page_content='Tanaka et al.ReSeARCH BRI ef\\nOF6 \\u2002|\\u2002CANCER DISCOVERY \\u2003AUGUST  2021  AACRJournals.orgfigure 4.\\u2002 Novel KRAS inhibitor RM-018 overcomes KRASG12C/Y96D. A, Mechanism of action of RM-018. B, RM-018 selectively inhibits cell viability in \\ncells harboring KRASG12C. C, Cell viability assays performed with NCI-H358, MIA PaCa-2, Ba/F3, and MGH1138-1 cells stably infected with KRASG12C or \\nKRASG12C/Y96D treated for 72 hours with RM-018. D  and e , Western blot analysis performed in MIA PaCa-2 stably expressing KRASG12C or KRASG12C/Y96D (D) \\nand HEK293T cells transiently expressing KRAS mutants (\\ne) after treatment of RM-018 for 4 hours.\\n \\nf\\n, Western blot analysis of MGH1138-1 cells transiently \\nexpressing KRASG12C or KRASG12C/Y96D after treatment with RM-018 for 4 hours. G, HEK293T cells transiently expressing KRAS mutants were treated with \\nthe indicated drug at 100 nmol/L each for 4 hours and then subjected to Western blot analysis.150\\n100\\n50\\n0\\n−11 −10 −9 −8 −7 −6AB\\nC\\nRM-018 MIA PaCa-2 \\nBaF3 KRASG12C \\nKRASG12C/Y96D \\nLog RM-018 (mol/L) −11 −10 −9 −8 −7 −6\\nLog RM-018 (mol/L) −11 −10 −9 −8 −7 −6\\nLog RM-018 (mol/L) −9 −8 −7 −6 −5\\nLog RM-018 (mol/L) Relative Cell Titer (%) 150\\n100\\n50\\n0Relative Cell Titer (%) 150\\n100\\n50\\n0Relative Cell Titer (%) 150\\n100\\n50\\n0Relative Cell Titer (%) H358 \\nDMIA PaCa-2 \\n0 1 3 10 30 100 0 1 3 10 30 100 G12C G12C/Y96D RM-018\\n(nmol/L):\\npERK\\npRSK\\nGAPDHKRAS\\nGE\\n0 3 10 30 100 0 3 10 30 100 RM-018\\n (nmol/L):G12C G12C/Y96D \\n1 1 \\npERK\\npRSK\\nGAPDHKRAS293T \\nKRAS\\npERK\\npRSK\\nGAPDH\\nDMSO \\nMRTX849 AMG 510 RM-018 \\nDMSO MRTX849 AMG 510 RM-018 G12C G12C/Y96D 293T \\n150\\nInhibitor\\nCYPAKRASG12C(ON)\\n100\\n50\\n0\\n−10 −9 −8 −7 −6NCI-H358 (KRASG12C) \\nAsPC-1 (KRASG12D) \\nNCI-H1975 (EGFRT790M,L858R, KRASWT) \\nA375 (BRAFV600E) Relative Cell Titer (%) \\nLog RM-018 (mol/L) \\nMGH1138-1 \\nKRAS\\npERK\\nGAPDHpRSK\\n0 10 30 100 300 1,000 0 10 30 100 300 1,000 G12C G12C/Y96D RM-018\\n(nmol/L):F \\nMGH 1138-1 \\nMRTX849 and AMG 510 and ∼20-fold for ARS-1620 (Fig. 3A) \\nrelative to KRASG12C expression alone, the efficacy of RM-018 \\non cell viability was largely unaffected by KRASG12C/Y96D  \\nexpression, with IC 50 shifts of only ∼2-fold (Fig. 4C; Sup-\\nplementary Table S4). In addition, RM-018 was able to inhibit pERK and pRSK levels with similar potency in the \\npresence of KRASG12C or KRASG12C/Y96D expression in MIA \\nPaCa-2, 293T cells, and the patient-derived KRASG12C-mutant \\nNSCLC cell line MGH1138-1 (Fig. 4D–F). Inhibition of cell viability by RM-018 was also unaffected by KRAS\\nG12C/Y96D  \\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 5}\n",
      "page_content='Clinical Acquired Resistance to KRASG12C InhibitionReSeARCH BRI ef\\n A UGUST  2021\\u2003 CANCER DISCOVERY \\u2002|\\u2002OF7 \\nexpression in the patient-derived MGH1138-1 model. Fur -\\nthermore, the KRAS mobility shift induced by covalent bind-\\ning of RM-018 was observed in both cell lines in the presence of either KRAS\\nG12C or KRASG12C/Y96D expression, suggesting \\nthat binding of RM-018 to KRAS is not abrogated by the KRAS\\nY96D mutation. Indeed, although a KRAS mobility shift \\ndue to covalent drug binding was observed in 293T cells expressing KRAS\\nG12C for MRTX849, AMG 510, and RM-018, \\nonly RM-018 exhibited this same mobility shift and was able to inhibit downstream signaling in the presence of the KRAS\\nG12C/Y96D mutation (Fig. 4G). Taken together, these data \\nsuggest that RM-018 retains the ability to bind and inhibit KRAS\\nG12C/Y96D and may represent a potential therapeutic \\nstrategy to overcome this acquired resistance mechanism.\\nDisc Ussi ON\\nThe arrival of covalent KRASG12C-selective inhibitors in the \\nclinic and early signs of activity demonstrated by MRTX849 and AMG 510 have generated great enthusiasm (4, 5). How-ever, our experiences across targeted therapies in lung cancer and other cancers collectively demonstrate that acquired resistance to the KRAS\\nG12C inhibitors will represent an inevi-\\ntable challenge going forward. Although preclinical studies have nominated putative mechanisms of up-front resistance, including RAS–MAPK pathway reactivation (4, 8), mecha-nisms of acquired resistance to MRTX849 or AMG 510 caus-ing disease relapse in patients remain unknown.\\nThe acquired resistance demonstrated in this patient is \\ninstructive in highlighting several points. First, 10 distinct resistance alterations arose in this patient, all converging on the reactivation of RAS–MAPK signaling, suggesting that this may be a central common mechanism of acquired resistance. RAS reactivation occurred by multiple different mechanisms, including (i) activation of another RAS isoform (in this case, NRAS); (ii) other KRAS activating mutations in trans (G13D, G12V); (iii) potential loss of KRAS\\nG12C through a mutational \\nswitch to a different KRAS mutation in cis, although sup-ported by limited sequencing reads; and (iv) a novel secondary alteration in KRAS (i.e., Y96D), which alters inhibitor binding.\\nStructural modeling predicted that this KRAS\\nY96D muta-\\ntion disrupts critical hydrogen bonding between the Y96 residue of KRAS and MRTX849. Importantly, we found that this KRAS\\nY96D mutation conferred resistance not only to \\nMRTX849 but also to additional KRASG12C-selective inhibi-\\ntors in clinical development, AMG 510 and ARS-1620, high-lighting that mutations affecting the Y96 residue of KRAS may represent a shared vulnerability for the currently avail-able KRAS\\nG12C inhibitors. Continued investigation into clini-\\ncal mechanisms of resistance to KRASG12C inhibitors in larger \\ncohorts of patients will be required to define the spectrum and frequency of KRAS\\nY96 mutations (as well as other on-\\ntarget mutations). Nonetheless, in light of our observations, it may be necessary to develop novel compounds that are able to target KRAS\\nG12C/Y96D to overcome resistance to KRASG12C \\ninhibitors in clinic. RM-018 was identified as a novel KRASG12C \\ninhibitor with a distinct mechanism of action for targeting KRAS, which was uniquely able to overcome KRAS\\nG12C/Y96D \\nacross multiple models. These results suggest that a novel, distinct KRAS inhibitor could theoretically be used to target  resistance after an acquired KRAS resistance mutation emerges on the initial KRAS\\nG12C inhibitor, and they support \\nefforts toward rational design of next-generation inhibitors.\\nOf note, however, the abundance of mutations down-\\nstream of RAS suggests that MAPK reactivation alone may be sufficient to drive resistance in at least some KRAS\\nG12C \\ncancers. At the time of disease progression on MRTX849, this patient’s plasma biopsy specimen revealed a low allele fraction of KRAS' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 6}\n",
      "page_content='G12C \\ncancers. At the time of disease progression on MRTX849, this patient’s plasma biopsy specimen revealed a low allele fraction of KRAS\\nY96D, particularly compared with the sig-\\nnificantly higher allele fraction of KRASG12C (0.4% vs. 31.7%, \\nrespectively), suggestive of the presence of polyclonal resist-ance to MRTX849 (with KRAS\\nG12C/Y96D-harboring tumor cells \\nrepresenting a minor subclone). Concordant with this notion and prior reports based on preclinical models, subsequent posttreatment plasma samples in this case revealed numer -\\nous mutations affecting various nodes of the RAS–MAPK pathway (including NRAS, BRAF, and MAP2K1 mutations). \\nThus, rational combinations of KRAS\\nG12C inhibitors with \\ndownstream MAPK pathway inhibitors may be needed to suc-cessfully prevent or overcome resistance. Although analysis of additional patients is clearly needed to more fully define the spectrum of potential acquired resistance mechanisms to KRAS\\nG12C inhibitors, this study begins to delineate some \\nof the potential resistance alterations that may be observed clinically. Ultimately, an iterative discovery process of identi-fying mechanisms of resistance and validating in vitro with a detailed structural and molecular understanding should help advance the development of novel strategies for therapeutic targeting of KRAS-mutant cancers.\\nMeth ODs\\nPatient Treatment and Specimen Collection\\nThe patient was treated with MRTX849 dosed 600 mg twice \\ndaily on the phase I study (KRYSTAL-1) after providing written \\ninformed consent (ClinicalTrials.gov identifier: NCT03785249). She had received two prior lines of therapy. All pre- and posttreat-ment biopsies and genotyping were performed in accordance with the Massachusetts General Hospital (MGH) institutional review board–approved protocol and in accordance with the Declaration of Helsinki. The pretreatment tumor specimen was analyzed using the MGH SNaPshot next-generation sequencing assay (25). All cfDNA samples were sequenced using the commercially available Guardant360 assay (Guardant Health). More detailed patient history is available in the Supplement.\\nCell Lines and Reagents\\nBa/F3 cells were obtained from the RIKEN BRC Cell Bank (RIKEN \\nBioResource Center). MGH1138-1 cells were generated from a patient with KRAS\\nG12C-mutant NSCLC using methods that have \\nbeen previously described (26). Prior to cell line generation, the patient provided written informed consent to participate in a Dana-Farber/Harvard Cancer Center institutional review board–approved protocol giving permission for research to be performed on their sample. The remaining cell lines were obtained from ATCC or the Center for Molecular Therapeutics at the MGH Cancer Center, which routinely performs cell line authentication testing by SNP and short-tandem repeat analysis. HEK293T cells were maintained in DMEM supplemented with 10% FBS. MIA PaCa-2 and NCI-H358 cells were maintained in DMEM/F12 supplemented with 10% FBS. LU-65 and MGH1138-1 cells were maintained in RPMI supplemented with 10% FBS. Ba/F3 cells were maintained in DMEM supplemented \\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 6}\n",
      "page_content='Tanaka et al.ReSeARCH BRI ef\\nOF8 \\u2002|\\u2002CANCER DISCOVERY \\u2003AUGUST  2021  AACRJournals.orgwith 10% FBS and 10 ng/mL IL3. The \\nKRAS (G12C or G12C/Y96D) \\ngene was inserted in pMXs-Puro Retroviral Expression Vector, which \\nwas purchased from Cell Biolabs. Retrovirus packaging mutated KRAS genes were produced with HEK293T cells. After concentra-tion of virus with Retro-Concentin Retro Concentration Reagent  \\n(System Biosciences), MIA PaCa-2, NCI-H358, and Ba/F3 cells were infected with the virus packaging either the KRAS\\nG12C or KRASG12C/Y96D \\ngene. After 48 hours of incubation, the cells were treated with puro-mycin (1–2 \\nμg/mL) for another 48 hours. IL3 was withdrawn to \\nselect for Ba/F3 cells dependent on mutant KRAS signaling after 48 hours of puromycin treatment. The remaining cells were maintained in media supplemented with puromycin. For transient expression experiments, a day after seeding the cells, pMXs-Puro-KRAS\\nG12C or \\npMXs-Puro-KRASG12C/Y96D vectors were induced with Lipofectamine \\n2000 Transfection Reagent (ThermoFisher Scientific) following the manufacturer’s protocol. After 16 to 24 hours of incubation, cells were treated with inhibitors for 4 hours. AMG 510 was purchased from MedChemExpress. MRTX849 and ARS-1620 were purchased from Selleck Chemicals. RM-018 was provided by Revolution Medi-cines, and details of the chemical synthesis can be found in Interna-tional Patent Application No. PCT/US2020/058841.\\nCell Viability Assays\\nCell lines were seeded in a 96-well plate at 2 to 10 × 103 cells/well \\ndepending on cell lines and after 24 hours treated with a serial dilu-tion of drugs and incubated for 72 hours. Cell viability was measured with CellTiter-Glo (Promega).\\nWestern Blot Analysis\\nCell lines were treated with MRTX849, AMG 510, or RM-018 \\nfor 4 hours and lysates were prepared as described previously (27). All antibodies were diluted in 5% bovine serum albumin as fol-lows: KRAS (Sigma), pERK (Thr202/Tyr204,1:1,000; Cell Signal-ing Technology), p44/42 MAPK (ERK1/2) (1:1,000; Cell Signaling  \\nTechnology), phospho-RSK1 (T359+S363, 1:1,000; Abcam), phospho-  \\nAKT (Ser473, 1:1,000; Cell Signaling Technology), AKT (1:1,000; Cell Signaling Technology), and GAPDH (1:1,000; MilliporeSigma).\\nRAS-GTP Pulldown\\nAfter indicated inhibitor treatment, RAS activity was assessed by \\nGST-RAF-RBD pulldown (Cell Signaling Technology), followed by Western blot analysis with pan-RAS or RAS isoform–specific antibod-ies. Pulldown samples and whole-cell lysates were resolved on 4% to 12% Bis-Tris gels, and Western blotting was performed using antibod-ies against KRAS (Sigma) and pan-RAS (Cell Signaling Technology).\\nStructural Modeling\\nPublicly available crystal structures of KRASG12C in complex with \\nMRTX849 (PDB:6UT0), AMG 510 (PDB:6OIM), and ARS-1620 (PDB:5V9U) were downloaded from the RCSB Protein Data Bank (PDB; ref. 28). Structures were rendered in PyMol (The PyMOL Molecular Graphics System) and analyzed for hydrogen bonds and other molecular interactions between the KRAS\\nG12C inhibitors and \\nthe KRAS protein. Structures of Y96 amino acid mutation were gen-erated by Protein Mutagenesis Wizard implemented in PyMol, with one of the backbone-dependent rotamers manually selected.\\ncfDNA Extraction and ddPCR\\nWhole blood was collected by routine phlebotomy in two 10-mL \\nStreck tubes. Plasma was separated within 1 to 4 days of collection through two different centrifugation steps (the first at room temper -\\nature for 10 minutes at 1,600 × g and the second at 3,000 × g for the \\nsame time and temperature). Plasma was stored at −80°C until cfDNA extraction. cfDNA was extracted from plasma using the QIAamp  Circulating Nucleic Acid Kit (QIAGEN) with 60 minutes of protein-ase K incubation at 60°C. All other steps were performed according to the manufacturer’s instructions. For ddPCR experiments, DNA template (up to 10 \\nμL, with a total of 20 ng) was added to 12.5 μL \\nddPCR Supermix for Probes (Bio-Rad) and 1.25 μL custom primer/' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 7}\n",
      "page_content='μL, with a total of 20 ng) was added to 12.5 μL \\nddPCR Supermix for Probes (Bio-Rad) and 1.25 μL custom primer/\\nprobe mixture. This reaction mix was added to a DG8 cartridge together with 60 \\nμL Droplet Generation Oil for Probes (Bio-Rad) \\nand used for droplet generation. Droplets were then transferred to a 96-well plate (Eppendorf) and then thermal cycled with the following conditions: 5 minutes at 95°C, 40 cycles of 94°C for 30 seconds, 55°C (with a few grades of difference among assays) for 1 minute, followed by 98°C for 10 minutes (Ramp Rate 2°C/s). Droplets were analyzed with the QX200 Droplet Reader (Bio-Rad) for fluorescent measure-ment of FAM and HEX probes. Gating was performed based on posi-tive and negative controls, and mutant populations were identified. The ddPCR data were analyzed with QuantaSoft analysis software (Bio-Rad) to obtain fractional abundance of the mutant DNA alleles in the wild-type/normal background. The quantification of the target molecule was presented as the number of total copies (mutant plus wild-type) per sample in each reaction. Allelic fraction was calculated as follows: AF % = [N\\nmut/(N mut + N wt) * 100], where N mut is the num-\\nber of mutant alleles and Nwt is the number of wild-type alleles per \\nreaction. ddPCR analysis of normal control plasma DNA (from cell lines) and no DNA template controls was always included. Probe and primer sequences are available upon request.\\nAuthors’ Disclosures\\nN. Tanaka reports other support from Japanese Foundation for \\nCancer Research during the conduct of the study. J.J. Lin reports personal fees and other support from Genentech, personal fees from C4 Therapeutics, personal fees from Nuvalent, personal fees from Blueprint Medicines, personal fees and other support from Turning Point Therapeutics, personal fees and other sup-port from Elevation Oncology, personal fees from Pfizer, other support from Hengrui Therapeutics, other support from Novartis, other support from Neon Therapeutics, other support from Relay Therapeutics, other support from Bayer, and other support from Roche outside the submitted work. L.A. Kiedrowski reports other support from Guardant Health during the conduct of the study. D. Juric reports grants and personal fees from Novartis, grants and personal fees from Genentech, grants and personal fees from EMD Serono, grants and personal fees from Eisai, personal  \\nfees from Guardant, personal fees from Ipsen, personal fees from Petra Pharma, personal fees from Mapkure, personal fees from Vib-liome, personal fees from Relay Therapeutics, grants from Takeda, grants and personal fees from Syros, grants from Ribon Therapeu-tics, grants from Infinity Pharmaceuticals, grants from InventisBio,  \\nand grants from Amgen outside the submitted work. J.F. Gainor reports personal fees from Mirati and personal fees from Amgen dur -\\ning the conduct of the study, personal fees from Bristol-Myers Squibb, personal fees from Genentech/Roche, personal fees from Takeda, personal fees from Lilly, personal fees and other support from Blue-print, personal fees from Oncorus, personal fees from Regeneron, personal fees from Gilead, personal fees from AstraZeneca, personal fees from Pfizer, grants and personal fees from Novartis, personal  \\nfees and other support from Merck, personal fees from Agios, other support from Moderna, other support from Jounce, other support from Array, other support from Alexo outside the submitted work, and an immediate family member who is an employee with equity at Ironwood Pharmaceuticals. S.J. Klempner reports personal fees from BMS, personal fees from Merck, personal fees from Daiichi-Sankyo, personal fees from Astellas, personal fees from Pieris Oncology, personal fees from Natera, personal fees from Foundation Medicine, and other support from Turning Point Therapeutics outside the submitted work. L. Bar-Peled reports personal fees from Scorpion' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 7}\n",
      "page_content='Research. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 7}\n",
      "page_content='Clinical Acquired Resistance to KRASG12C InhibitionReSeARCH BRI ef\\n A UGUST  2021\\u2003 CANCER DISCOVERY \\u2002|\\u2002OF9 \\nTherapeutics outside the submitted work. A.N. Hata reports grants \\nfrom Amgen, grants from Eli Lilly, grants from Relay Therapeu-tics, grants from Roche/Genentech, grants from Novartis, grants from Blueprint Medicines, grants from Pfizer, and grants and per -\\nsonal fees from Nuvalent outside the submitted work. R.S. Heist reports personal fees from Novartis, personal fees from Daichii  \\nSankyo, personal fees from EMD Serono, personal fees from Tarveda, personal fees from Apollomics, grants from Novartis, grants from Corvus, grants from Incyte, grants from Genentech Roche, grants from Mirati, grants from Turning Point, grants from BMS, grants from  \\nDaichii Sankyo, grants from AbbVie, grants from Agios, grants from Exelixis, and grants from Lilly outside the submitted work. R.B. Corcoran reports personal fees from AbbVie, grants and per -\\nsonal fees from Asana Biosciences, personal fees from Astex Phar -\\nmaceuticals, grants and personal fees from AstraZeneca, personal fees and other support from Avidity Biosciences, personal fees and other support from C4 Therapeutics, personal fees from Chugai, personal fees from Elicio, other support from Erasca, personal fees from Fog Pharma, personal fees from Guardant Health, personal fees from Ipsen, personal fees and other support from  \\nKinnate Biopharma, personal fees from Mirati Therapeutics, personal  \\nfees from Natera, grants from Novartis, grants from Lilly, personal fees  \\nand other support from nRichDx, personal fees from Pfizer, personal fees from QIAGEN, personal fees and other support from Remix Therapeutics, personal fees and other support from Revolution Medicines, personal fees from Roivant, personal fees from Shionogi, personal fees from Tango Therapeutics, personal fees from Taiho, and personal fees from Zikani Therapeutics outside the submitted work; in addition, R.B. Corcoran has a patent for Intellectual Prop-erty pending. No disclosures were reported by the other authors.\\nAuthors’ Contributions\\nN. Tanaka: Conceptualization, data curation, formal analysis, \\ninvestigation, writing–original draft, writing–review and editing.  \\nJ.J. Lin: Conceptualization, data curation, formal analysis, inves-tigation, writing–original draft, writing–review and editing. C. Li:  \\nConceptualization, data curation, formal analysis, investigation, writing–  \\noriginal draft, writing–review and editing. M.B. Ryan: Conceptual-ization, data curation, formal analysis, investigation, writing–original  \\ndraft, writing–review and editing. J. Zhang: Conceptualization, for -\\nmal analysis, writing–original draft, writing–review and editing.  \\nL.A. Kiedrowski: Conceptualization, formal analysis, writing–review and editing. A.G. Michel: Formal analysis, investigation, writing–review and editing. M.U. Syed: Formal analysis, investigation, writing–  \\nreview and editing. K.A. Fella: Formal analysis, investigation,  \\nwriting–review and editing. M. Sakhi: Formal analysis, investigation, writing–review and editing. I. Baiev: Formal analysis, investiga-tion, writing–review and editing. D. Juric: Conceptualization, formal  \\nanalysis, investigation, writing–review and editing. J.F. Gainor: Con-ceptualization, formal analysis, investigation, writing–review and editing.  \\nS.J. Klempner: Conceptualization, formal analysis, investigation, writing–review and editing. J.K. Lennerz: Formal analysis, investiga-tion, writing–review and editing. G. Siravegna: Conceptualization, data curation, formal analysis, supervision, investigation, writing–original draft, writing–review and editing. L. Bar-Peled: Conceptu-alization, data curation, formal analysis, supervision, investigation, writing–review and editing. A.N. Hata: Conceptualization, resources, data curation, formal analysis, supervision, funding acquisition,' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 8}\n",
      "page_content='validation, investigation, writing–original draft, writing–review and editing. R.S. Heist: Conceptualization, resources, data curation, formal \\nanalysis, supervision, funding acquisition, validation, investigation, writing–original draft, writing–review and editing. R.B. Corcoran:   \\nConceptualization, resources, data curation, formal analysis, supervi-sion, funding acquisition, validation, investigation, writing–original draft, writing–review and editing.Acknowledgments\\nRM-018 was kindly provided by Revolution Medicines. This \\nresearch was supported in part by a Mark Foundation for Cancer Research EXTOL Project Grant (J.F. Gainor, A.N. Hata) and a Stand Up To Cancer—American Cancer Society Lung Cancer Dream Team Translational Research Grant (J.F. Gainor, A.N. Hata; grant number: SU2C-AACR-DT17–15). Stand Up To Cancer (SU2C) is a division of the Entertainment Industry Foundation. The indicated SU2C research grant is administered by the American Association for Cancer Research, a scientific partner of SU2C. J.J. Lin is partially supported by funding from NIH R01-CA164273. C. Li is supported by funding from NIH 1F32CA250231–01. N. Tanaka is partially supported by funding from US-JFCR Scientist Exchange Program. L. Bar-Peled is supported by AACR (19–20–45-BARP), Damon Runyon (DRR-62–20), and NIH/NCI (R00 CA215249).\\nReceived March 19, 2021; revised March 30, 2021; accepted April 1, \\n2021; published first April 6, 2021.\\nRefe ReNCeS\\n 1 . F ell JB, Fischer JP, Baer BR, Blake JF, Bouhana K, Briere DM, et al. \\nIdentification of the clinical development candidate MRTX849, a \\ncovalent KRAS(G12C) inhibitor for the treatment of cancer. J Med Chem 2020;63:6679–93.\\n 2.\\n Lanman BA\\n, Allen JR, Allen JG, Amegadzie AK, Ashton KS, Booker SK,  \\net al. Discovery of a covalent inhibitor of KRAS(G12C) (AMG 510) for the treatment of solid tumors. J Med Chem 2020;63:52–65.\\n 3.\\n Ostrem JM, P\\neters U, Sos ML, Wells JA, Shokat KM. K-Ras(G12C) \\ninhibitors allosterically control GTP affinity and effector interac-tions. Nature 2013;503:548–51.\\n 4.\\n Hallin J, Engstrom LD, Hargis L, Calinisan A\\n, Aranda R, Briere DM, \\net al. The KRAS(G12C) inhibitor MRTX849 provides insight toward therapeutic susceptibility of KRAS-mutant cancers in mouse models and patients. Cancer Discov 2020;10:54–71.\\n 5\\n. Hong DS, F\\nakih MG, Strickler JH, Desai J, Durm GA, Shapiro GI, et al.  \\nKRAS(G12C) inhibition with sotorasib in advanced solid tumors.  \\nN Engl J Med 2020;383:1207–17.\\n 6.\\n Li B, Sk\\noulidis F, Falchook G, Sacher A, Velcheti V, Dy G, et al. \\nPS01.07 registrational phase 2 trial of sotorasib in KRAS p.G12C mutant NSCLC: first disclosure of the codebreak 100 primary analy-sis. J Thorac Oncol 2021;16:S61.\\n 7\\n.\\n Jänne P\\nA, Rybkin II, Spira AI, Riely GJ, Papadopoulos KP, Sabari JK, \\net al. KRYSTAL-1: activity and safety of adagrasib (MRTX849) in advanced/metastatic non–small-cell lung cancer (NSCLC) harboring KRAS G12C mutation. Eur J Cancer 2020;138:S1–S2.\\n 8.\\n R\\nyan MB, Fece de la Cruz F, Phat S, Myers DT, Wong E, Shahzade HA, \\net al. Vertical pathway inhibition overcomes adaptive feedback resist-ance to KRAS(G12C) inhibition. Clin Cancer Res 2020;26:1633–43.\\n 9.\\n Misale \\nS, Fatherree JP, Cortez E, Li C, Bilton S, Timonina D, et al. KRAS \\nG12C NSCLC models are sensitive to direct targeting of KRAS in com-bination with PI3K inhibition. Clin Cancer Res 2019;25:796–807.\\n 10.\\n Sant\\nana-Codina N, Chandhoke AS, Yu Q, Malachowska B, Kuljanin \\nM, Gikandi A, et al. Defining and targeting adaptations to oncogenic KRAS(G12C) inhibition using quantitative temporal proteomics. Cell Rep 2020;30:4584–99.\\n 11\\n.\\n Solanki HS, W\\nelsh EA, Fang B, Izumi V, Darville L, Stone B, et al. Cell \\ntype-specific adaptive signaling responses to KRAS(G12C) inhibition. Clin Cancer Res 2021;27:2533–48.\\n 12.\\n Xue JY\\n, Zhao Y, Aronowitz J, Mai TT, Vides A, Qeriqi B, et al. Rapid \\nnon-uniform adaptation to conformation-specific KRAS(G12C) inhi-bition. Nature 2020;577:421–5.\\n 13.\\n F' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 8}\n",
      "page_content='12.\\n Xue JY\\n, Zhao Y, Aronowitz J, Mai TT, Vides A, Qeriqi B, et al. Rapid \\nnon-uniform adaptation to conformation-specific KRAS(G12C) inhi-bition. Nature 2020;577:421–5.\\n 13.\\n F\\nedele C, Li S, Teng KW, Foster CJR, Peng D, Ran H, et al. SHP2 inhi-\\nbition diminishes KRASG12C cycling and promotes tumor microen-vironment remodeling. J Exp Med 2021;218:e20201414.\\n 14.\\n Kinoshit\\na-Kikuta E, Kinoshita E, Ueda S, Ino Y, Kimura Y, Hirano H, \\net al. Increase in constitutively active MEK1 species by  introduction \\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 8}\n",
      "page_content='Tanaka et al.ReSeARCH BRI ef\\nOF10 \\u2002|\\u2002CANCER DISCOVERY \\u2003AUGUST  2021  AACRJournals.orgof MEK1 mutations identified in cancers. Biochim Biophys Acta  \\nProteins Proteom 2019;1867:62–70.\\n 15.\\n Gao Y\\n, Chang MT, McKay D, Na N, Zhou B, Yaeger R, et al. Allele-\\nspecific mechanisms of activation of MEK1 mutants determine their \\nproperties. Cancer Discov 2018;8:648–61.\\n 16.\\n Canon J, R\\nex K, Saiki AY, Mohr C, Cooke K, Bagal D, et al. The clini-\\ncal KRAS(G12C) inhibitor AMG 510 drives anti-tumour immunity. Nature 2019;575:217–23.\\n 17\\n.\\n Sondk\\na Z, Bamford S, Cole CG, Ward SA, Dunham I, Forbes SA. The \\nCOSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. Nat Rev Cancer 2018;18:696–705.\\n 1\\n8. Consor\\ntium APG. AACR Project GENIE: powering precision medi-\\ncine through an international consortium. Cancer Discov 2017;7:  \\n818–31.\\n 19.\\n F\\nell JB, Fischer JP, Baer BR, Ballard J, Blake JF, Bouhana K, et al. \\nDiscovery of tetrahydropyridopyrimidines as irreversible covalent inhibitors of KRAS-G12C with in vivo activity. ACS Med Chem Lett 2018;9:1230–4.\\n 20.\\n Janes MR, Zhang J, Li LS, Hansen R, P\\neters U, Guo X, et al. Targeting \\nKRAS mutant cancers with a covalent G12C-specific inhibitor. Cell 2018;172:578–89.\\n 21\\n.\\n Chen H, Smaill JB, Liu T\\n, Ding K, Lu X. Small-molecule inhibitors \\ndirectly targeting KRAS as anticancer therapeutics. J Med Chem 2020;63:14404–24. 22.  Zeng M, Lu J, Li L, F eru F, Quan C, Gero TW, et al. Potent and selec-\\ntive covalent quinazoline inhibitors of KRAS G12C. Cell Chem Biol 2017;24:1005–16.\\n 23.\\n Schulze CJ, Bermingham A\\n, Choy TJ, Cregg JJ, Kiss G, Marquez A, et al. \\nAbstract PR10: tri-complex inhibitors of the oncogenic, GTP-bound form of KRAS\\nG12C overcome RTK-mediated escape mechanisms and \\ndrive tumor regressions in vivo. Mol Cancer Ther 2019;18:PR10.\\n 2\\n4. N\\nichols R, Schulze C, Bermingham A, Choy T, Cregg J, Kiss G, et al. \\nA06 Tri-complex inhibitors of the oncogenic, GTP-bound form of KRASG12C overcome RTK-mediated escape mechanisms and drive tumor regressions in preclinical models of NSCLC. J Thorac Oncol 2020;15:S13–S4.\\n 25.\\n Zheng Z, Liebers M, Zhelyazk\\nova B, Cao Y, Panditi D, Lynch KD,  \\net al. Anchored multiplex PCR for targeted next-generation sequenc-ing. Nat Med 2014;20:1479–84.\\n 2\\n6. Cr\\nystal AS, Shaw AT, Sequist LV, Friboulet L, Niederst MJ, Lockerman  \\nEL, et al. Patient-derived models of acquired resistance can identify effective drug combinations for cancer. Science 2014;346:1480–6.\\n 27\\n.\\n Ahronian L\\nG, Sennott EM, Van Allen EM, Wagle N, Kwak EL, Faris \\nJE, et al. Clinical acquired resistance to RAF inhibitor combinations in BRAF-mutant colorectal cancer through MAPK pathway altera-tions. Cancer Discov 2015;5:358–67.\\n 28.\\n Berman HM, W\\nestbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, \\net al. The protein data bank. Nucleic Acids Res 2000;28:235–42.\\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 9}\n",
      "page_content='Published OnlineFirst April 6, 2021. Cancer Discov\\xa0\\xa0 Noritaka Tanaka, Jessica J. Lin, Chendi Li, et al. \\xa0 MAPK Reactivation− lterations Converging on RASa Novel KRAS Switch-II Pocket Mutation and Polyclonal Athrough  Inhibition G12C Clinical Acquired Resistance to KRAS\\n\\xa0 Updated version\\n\\xa0  10.1158/2159-8290.CD-21-0365doi:Access the most recent version of this article at:\\n\\xa0 MaterialSupplementary\\n\\xa0  http://cancerdiscovery.aacrjournals.org/content/suppl/2021/04/03/2159-8290.CD-21-0365.DC1Access the most recent supplemental material at:\\n\\xa0 \\xa0 \\n\\xa0 \\xa0 \\n\\xa0 \\xa0 \\n\\xa0 E-mail alerts  related to this article or journal. Sign up to receive free email-alerts\\n\\xa0 SubscriptionsReprints and \\n\\xa0 . pubs@aacr.org Department atTo order reprints of this article or to subscribe to the journal, contact the AACR Publications\\n\\xa0 Permissions\\n\\xa0 Rightslink site. Click on \"Request Permissions\" which will take you to the Copyright Clearance Center\\'s (CCC). http://cancerdiscovery.aacrjournals.org/content/early/2021/06/16/2159-8290.CD-21-0365To request permission to re-use all or part of this article, use this link\\nResearch. on July 28, 2021. © 2021 American Association for Cancer cancerdiscovery.aacrjournals.org Downloaded from Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 10}\n"
     ]
    }
   ],
   "source": [
    "from langchain_community.document_loaders import PyPDFLoader\n",
    "loader = PyPDFLoader(pdf_file)\n",
    "\n",
    "a = time.time()\n",
    "pages = loader.load_and_split()\n",
    "b = time.time()\n",
    "\n",
    "print(b - a)\n",
    "print(len(pages))\n",
    "\n",
    "for p in pages:\n",
    "  print(p)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3.132020950317383\n",
      "19\n",
      "page_content='Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nReseaRch BRief\\nClinical Acquired Resistance to KRASG12C\\nInhibition through a Novel KRAS Switch-II\\nPocket Mutation and Polyclonal Alterations\\nConverging on RAS–MAPK Reactivation\\nNoritaka Tanaka1, Jessica J. Lin1, Chendi Li1, Meagan B. Ryan1, Junbing Zhang1, Lesli A. Kiedrowski2,\\nAlexa G. Michel1, Mohammed U. Syed1, Katerina A. Fella1, Mustafa Sakhi1, Islam Baiev1, Dejan Juric1,\\nJustin F. Gainor1, Samuel J. Klempner1, Jochen K. Lennerz3, Giulia Siravegna1, Liron Bar-Peled1,\\nAaron N. Hata1, Rebecca S. Heist1, and Ryan B. Corcoran1\\naBstRact Mutant-selective KRASG12C inhibitors, such as MRTX849 (adagrasib) and AMG 510\\n(sotorasib), have demonstrated efficacy in KRASG12C-mutant cancers, including\\nnon–small cell lung cancer (NSCLC). However, mechanisms underlying clinical acquired resistance to\\nKRASG12C inhibitors remain undetermined. To begin to define the mechanistic spectrum of acquired\\nresistance, we describe a patient with KRASG12C NSCLC who developed polyclonal acquired resistance\\nto MRTX849 with the emergence of 10 heterogeneous resistance alterations in serial cell-free DNA\\nspanning four genes (KRAS, NRAS, BRAF, MAP2K1), all of which converge to reactivate RAS–MAPK\\nsignaling. Notably, a novel KRASY96D mutation affecting the switch-II pocket, to which MRTX849 and\\nother inactive-state inhibitors bind, was identified that interferes with key protein–drug interactions\\nand confers resistance to these inhibitors in engineered and patient-derived KRASG12C cancer models.\\nInterestingly, a novel, functionally distinct tricomplex KRASG12C active-state inhibitor RM-018 retained\\nthe ability to bind and inhibit KRASG12C/Y96D and could overcome resistance.\\nSIGNIfICANCe: In one of the first reports of clinical acquired resistance to KRASG12C inhibitors, our\\ndata suggest polyclonal RAS–MAPK reactivation as a central resistance mechanism. We also identify\\na novel KRAS switch-II pocket mutation that impairs binding and drives resistance to inactive-state\\ninhibitors but is surmountable by a functionally distinct KRASG12C inhibitor.\\niNtRODUctiON antitumor responses (1–3). The lead clinical compounds soto-\\nrasib (AMG 510) and adagrasib (MRTX849) have advanced\\nThe development of compounds that bind covalently to rapidly and demonstrated tolerability and single-agent\\ncysteine 12 in KRASG12C cancers has ushered in a new era in activity across KRASG12C-mutant cancers (4, 5). In patients\\nefforts to target KRAS directly. Biochemically, these agents with advanced non–small cell lung cancer (NSCLC) harbor-\\nlock KRAS in its inactive GDP-bound conformation, thereby ing KRASG12C (which comprises approximately 13% of all\\ninhibiting downstream signaling, leading to preclinical lung adenocarcinomas), AMG 510 and MRTX849 have\\n1Massachusetts General Hospital Cancer Center and Department of 724-3442; E-mail: ahata@mgh.harvard.edu; Rebecca S. Heist, Massachu-\\nMedicine, Harvard Medical School, Boston, Massachusetts. 2Guardant Health, setts General Hospital Cancer Center, 55 Fruit Street, Boston, MA 02114.\\nRedwood City, California. 3Department of Pathology, Massachusetts Phone: 617-724-4000; E-mail: rheist@partners.org; and Ryan B. Corcoran,\\nGeneral Hospital, Boston, Massachusetts. Massachusetts General Hospital Cancer Center, 149 13th Street, 7th Floor,\\nNote: Supplementary data for this article are available at Cancer Discovery Boston, MA 02129. Phone: 617-726-8599; E-mail: rbcorcoran@partners.org\\nOnline (http://cancerdiscovery.aacrjournals.org/). Cancer Discov 2021;11:1–10\\nN. Tanaka, J.J. Lin, and C. Li contributed equally to this work. doi: 10.1158/2159-8290.CD-21-0365\\nCorresponding Authors: Aaron N. Hata, Massachusetts General Hospital ©2021 American Association for Cancer Research\\nCancer Center, 149 13th Street, 7th Floor, Boston, MA 02129. Phone: 617-\\nAUGUST 2021 CANCER DISCOVERY | OF1\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 0, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='AUGUST 2021 CANCER DISCOVERY | OF1\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 0, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nReSeARCH BRIef Tanaka et al.\\ndemonstrated meaningful efficacy with objective response these mutations in heterogeneous subclonal populations.\\nrates of 37% and 45%, as well as disease control rates of 81% These included three activating NRAS mutations (NRASQ61L,\\nand 96%, respectively (6, 7). AMG 510 has recently received NRASQ61K, NRASQ61R), which can drive active RAS signaling\\nBreakthrough Therapy designation from the FDA for the in a KRAS-independent manner, and BRAFV600E, which can\\ntreatment of patients with advanced KRASG12C-mutant maintain MAPK signaling downstream of KRASG12C in the\\nNSCLC following at least one prior systemic therapy. Mul- presence of MRTX849 (Supplementary Fig. S1). Three MAP2K1\\ntiple ongoing trials seek to augment responses to KRASG12C mutations (MAP2K1K57N, MAP2K1Q56P, MAP2K1E102-I103del) pre-\\ninhibitors through combination strategies. viously demonstrated to be activating and known to be\\nPreclinical studies with MRTX849 and other KRASG12C involved in resistance to upstream MAPK pathway inhibitors\\ninhibitors have suggested several mechanisms of up-front (i.e., BRAF inhibitors) were also identified (14, 15).\\nresistance, including reactivation of ERK-dependent sig- In addition, three KRAS mutations emerged in the postpro-\\nnaling to bypass KRASG12C blockade (4). Prior work by our gression cfDNA. Two of these mutations are the known activat-\\ngroup and others has identified adaptive RAS pathway feed- ing mutations KRASG13D and KRASG12V, and mutant-selective\\nback reactivation as a key mechanism of primary resistance KRASG12C inhibitors have previously been shown to be inef-\\nto KRASG12C inhibition (8–13). However, the key mecha- fective against these mutations (4, 16). A deeper analysis of\\nnisms of clinical acquired resistance to KRASG12C inhibitors individual sequencing reads from cfDNA suggested that these\\nare currently unknown. Here, as an initial effort to charac- mutations seemed to occur in trans to the original KRASG12C\\nterize the clinical landscape of potential acquired resistance mutation (Supplementary Fig. S2A and S2B), likely arising\\nmechanisms to KRASG12C inhibitors, we present a patient in the remaining wild-type copy of KRAS, which appeared to\\nwith KRASG12C-mutant NSCLC who developed acquired be retained based on pretreatment tumor sequencing (Sup-\\nresistance to MRTX849, characterized by the emergence plementary Table S2). However, it is not possible from the\\nof 10 individual resistance alterations involving four RAS– cfDNA data to confirm that these mutations coexist in cells\\nMAPK genes. All of these resistance alterations converge that also harbor the original KRASG12C mutation. Notably, a\\nto reactivate RAS–MAPK signaling, implicating this as a single, well-supported family of sequencing reads from the\\npotential central mechanism of acquired resistance. We same original template molecule showed the concurrent pres-\\nalso identify a novel KRASY96D resistance mutation in the ence of both nucleotide changes corresponding to KRASG12C\\nswitch-II pocket of KRAS through serial cell-free DNA and KRASG12V in cis on the same strand, which would encode\\n(cfDNA) analysis. Through structural modeling and in vitro for a KRASG12F mutation. While it is not possible to confirm\\nfunctional studies, we find that KRASY96D confers resist- the presence of this mutation based on a single read family,\\nance to multiple KRASG12C inhibitors currently in clinical this finding raises the possibility that cis mutations resulting\\ndevelopment but identify a novel active-state KRASG12C in “loss” of the original KRASG12C mutation and conversion\\ninhibitor, RM-018, that is able to overcome KRASG12C/Y96D- to a different KRAS mutation might be another potential\\nmediated resistance. mechanism of resistance. Notably, all putative resistance\\nmutations identified are predicted to converge on reactiva-\\ntion of RAS–MAPK pathway signaling, suggesting that this' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='mediated resistance. mechanism of resistance. Notably, all putative resistance\\nmutations identified are predicted to converge on reactiva-\\ntion of RAS–MAPK pathway signaling, suggesting that this\\nResULts may represent a common primary mechanism of acquired\\nresistance to KRASG12C inhibitors (Fig. 1C).\\nHeterogeneous Acquired Resistance Alterations\\nInterestingly, the third KRAS mutation identified, KRASY96D,\\nConverge on RAS–MAPK Reactivation\\nrepresents a novel mutation that is not known to be activat-\\nA 67-year-old woman with metastatic KRASG12C-mutant ing. Notably, although KRAS is the most commonly mutated\\nNSCLC was treated on the dose-expansion cohort of the oncogene in human cancer, a search of two large tumor muta-\\nphase 1 study of MRTX849 (NCT03785249; Methods; fur- tional databases—COSMIC and GENIE, which collectively\\nther detailed in Supplementary Methods). Initial scans contain >450,000 molecularly characterized cancers (17,\\nshowed a 32% reduction in tumor size (by RECIST v1.1), 18)—did not reveal a single previously identified mutation at\\nbut after approximately 4 months of treatment, the patient the KRASY96 locus among >75,000 cases with documented\\ndeveloped progressive disease, and the patient discontinued KRAS mutations (Supplementary Table S3). However, the\\ntherapy at 5.5 months (Fig. 1A). To identify putative mecha- Y96 residue is associated with the switch-II pocket to which\\nnisms of acquired resistance to MRTX849 in this patient, we MRTX849 and other inactive-state KRASG12C inhibitors bind,\\nassessed cfDNA using a targeted next-generation sequencing suggesting that the previously undescribed Y96D mutation\\nassay (Guardant360; Guardant Health) and droplet digital may have a novel and specific role in driving resistance to\\nPCR (ddPCR). Upon development of acquired resistance, the KRASG12C inhibitors.\\noriginal KRASG12C and TP53F338fs variants present in pretreat-\\nStructural Modeling of KRASG12C/Y96D\\nment tumor and cfDNA were again detected in cfDNA but\\nwere accompanied by the emergence of 10 distinct muta- To understand the significance of the acquired KRASY96D\\ntions affecting RAS–MAPK components KRAS, NRAS, BRAF, mutation, we performed structural modeling of the G12C-\\nand MAP2K1 (which encodes the MEK1 protein) identified mutant and G12C/Y96D double-mutant KRAS proteins\\nacross cfDNA specimens obtained after disease progression bound to the KRASG12C inhibitors MRTX849, AMG 510, and\\n(Fig. 1B; Supplementary Table S1). The lower allele frequen- ARS-1620 (Fig. 2). These three inhibitors bind the GDP state\\ncies of these alterations relative to the truncal KRASG12C of KRASG12C and exploit a cryptic pocket formed by the central\\nand TP53 mutations are consistent with the emergence of β sheet of RAS and switch-II (first identified by Ostrem and\\nOF2 | CANCER DISCOVERY AUGUST 2021 AACRJournals.org\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nClinical Acquired Resistance to KRASG12C Inhibition ReSeARCH BRIef\\nA MRTX849\\nBaseline 6 weeks 4 months\\nB CC RRTTKK\\nTumor cfDNA\\nDays post-MRTX849\\nPre- Pre- discontinuation:\\nMRTX849MRTX849\\n0 9 51 Secondary NRAS Q61K\\nKRAS WT NRAS Q61R\\nTP53 F338fs 36.8% 0.22% 8.8% 10.1% 14.3% KRAS Y96D G12C RAS\\nNRAS Q61L\\nKRAS G12C 21.3% 0.12% 31.7% 47.1% 24.9%\\nCis*\\nKRAS G12V - - - - 0.09% KRAS G12F*\\nKRAS G13D - - - 0.13%† 0.04% A/B/ BRAF V600E\\nTrans CRAF\\nKRAS Y96D - - 0.4% 0.2% - KRAS G12V\\nNRAS Q61L - - - 0.2% - KRAS G13D\\nMAP2K1 K57N\\nNRAS Q61R - - - - 0.02%\\nMEK MAP2K1 Q56P\\nNRAS Q61K - - 0.6% 0.6% 0.9% MAP2K1 E102_I103 del\\nBRAF V600E - - 0.1% 0.1% 0.5%\\nMAP2K1 K57N - - 0.05%† - 0.3% ERK\\nMAP2K1 Q56P - - - - 0.1%\\nMAP2K1 E102_I103del - - - 0.12%† 0.2%\\nProliferation and Survival\\nfigure 1. Acquired resistance to KRASG12C inhibitor MRTX849 (adagrasib). A, Computed tomography images of the patient’s axillary lymph node\\nmetastasis at baseline, during response to MRTX849, and at progression on MRTX849. B, Variant allele fractions of mutations detected in the patient’s\\nserial plasma samples. †, indicates the mutations were detected by ddPCR but not by plasma next-generation sequencing. C, Alterations detected in post-\\nMRTX849 cfDNA include acquired mutations in KRAS as well as multiple components of the MAPK signaling cascade. *, KRASG12F represents a potential\\nresistance mechanism supported by limited sequencing reads, as shown in Supplementary Fig. S2.\\ncolleagues; ref. 3). To determine the effects of the amino acid ring of Y96, which is disrupted with the Y96D mutation. In\\nsubstitution at the Y96 locus, crystal structures of MRTX849, addition, by introducing a negatively charged amino acid, the\\nAMG 510, and ARS-1620 bound to KRASG12C were modeled Y96D mutation changes the hydrophobic nature of the bind-\\nfor interactions with the Y96 residue within the switch-II ing pocket for all three compounds to a substantially more\\npocket (16, 19–21). The hydroxyl group of Y96 forms a direct hydrophilic pocket, which may further destabilize binding.\\nhydrogen bond with the pyrimidine ring of MTRX849, which\\nfunctional Characterization of KRASY96D\\nis abolished with the Y96D mutation. Y96D also disrupts the\\nwater-mediated hydrogen bond between Y96 and a carboxyl To assess whether KRASY96D can mediate resistance to\\ngroup on AMG 510. Finally, although Y96 does not form a MRTX849 and other inactive-state KRASG12C inhibitors, we\\ndirect hydrogen bond with ARS-1620, it stabilizes the inter- expressed KRASG12C or the KRASG12C/Y96D double mutant\\naction with ARS-1620 through pi-stacking with the phenyl in NCI-H358 (KRASG12C-mutant NSCLC), MIA PaCa-2\\nAUGUST 2021 CANCER DISCOVERY | OF3\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 2, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='ReSeARCH BRIef Tanaka et al.\\nMRTX849 AMG 510 ARS-1620\\nfigure 2. Structural basis for resistance to KRASG12C inhibition conferred by KRASY96D. Shown are the modeled crystal structures of MRTX849 (6UT0),\\nAMG 510 (6OIM), and ARS-1620 (5V9U) bound to KRASG12C (top) and KRASG12C/Y96D (bottom), highlighting the loss of the hydrogen bonds between\\nMRTX849 or AMG 510 and the Y96 residue and the disruption of the switch-II pocket dynamics between ARS-1620 and KRASG12C/Y96D.\\nOF4 | CANCER DISCOVERY AUGUST 2021 AACRJournals.org\\nC21G\\nD69Y/C21G\\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\n(KRASG12C-mutant pancreatic ductal adenocarcinoma), and expressing cells treated with MRTX849, levels of active\\nBa/F3 cells, which lack endogenous KRASG12C but become GTP-bound KRAS were maintained in KRASG12C/Y96D-expressing\\noncogene dependent upon withdrawal of IL3. In cell via- cells (Fig. 3E; ref. 22). These results suggest that the KRASY96D\\nbility assays, relative to KRASG12C-expressing controls, cells mutation disrupts KRASG12C inhibitor binding, leading to\\nexpressing KRASG12C/Y96D showed marked resistance to sustained KRAS signaling and therapeutic resistance.\\nthree KRASG12C inhibitors, with IC shifts of >100-fold for\\nMRTX849 and AMG 510 and\\n∼20-fo5 l0\\nd for ARS-1620 (Fig. 3A;\\nThe Active State KRASG12C Inhibitor RM-018\\nOvercomes KRASG12C/Y96D\\nSupplementary Table S4).\\nConsistent with the effects on cell viability, RAS–MAPK As KRASG12C/Y96D conferred resistance to multiple KRASG12C\\npathway activity, as measured by levels of phosphorylated inhibitors currently in clinical development, suggestive of\\nERK (pERK) and pRSK, was sustained in KRASG12C/Y96D- shared vulnerability for this class of inhibitors, we sought\\nexpressing MIA PaCa-2 cells even at high concentrations to identify whether a structurally and functionally distinct\\nof MRTX849, relative to cells expressing KRASG12C alone KRASG12C inhibitor might retain potency against this resist-\\n(Fig. 3B). Similarly, in KRASG12C-mutant NSCLC cells in ance mutation. RM-018 is a novel KRASG12C inhibitor that\\nwhich PI3K signaling is driven by mutant KRAS, including binds specifically to the GTP-bound, active [“RAS(ON)”] state\\nan existing patient-derived model MGH1138-1, persistent of KRASG12C. RM-018 is a “tricomplex” KRAS inhibitor, which\\npERK and pAKT levels were observed with KRASG12C/Y96D exploits a highly abundant chaperone protein, cyclophilin A,\\nin the presence of MRTX849, relative to KRASG12C expres- to bind and inhibit KRASG12C, as previously described (Fig. 4A;\\nsion alone (Fig. 3C; Supplementary Fig. S3). KRASG12C/Y96D structure shown in Supplementary Fig. S5; refs. 23, 24).\\nalso drove marked resistance to MRTX849 in the patient- Briefly, upon entering the cell, RM-018 forms a “binary com-\\nderived MGH1138-1 model. Furthermore, in 293T cells, plex” with cyclophilin A. This binary complex can associate\\nwhich lack endogenous KRASG12C expression, MRTX849 with the active state of KRASG12C, aided by protein–protein\\nwas unable to inhibit pERK levels driven by KRASG12C/Y96D surface interactions between cyclophilin A and KRAS, and\\n(Fig. 3D). Because MRTX849 and other inactive-state forms a covalent bond with KRASG12C in a mutant-selec-\\nKRASG12C inhibitors bind covalently to KRASG12C, an elec- tive manner. This resultant “tricomplex” inhibits KRASG12C\\ntrophoretic mobility shift of drug-adducted KRASG12C can through binding of cyclophilin A, leading to steric occlu-\\nbe observed upon drug binding due to increased molecular sion and preventing the association of downstream effector\\nweight. However, this mobility shift was no longer observed proteins. Given the markedly different mechanism of action\\nwhen 293T cells expressing KRASG12C/Y96D were treated with of this class of inhibitor, we hypothesized that the inhibitory\\nMRTX849, suggesting that the Y96D mutation may abro- activity of RM-018 might be differentially affected by KRASY96D' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='MRTX849, suggesting that the Y96D mutation may abro- activity of RM-018 might be differentially affected by KRASY96D\\ngate inhibitor binding. Notably, KRASG12C/Y96D appeared to compared with inactive-state KRASG12C inhibitors.\\nhave higher basal activation than KRASG12C, as measured RM-018 demonstrated selectivity for KRASG12C-driven\\nby a higher proportion of the active GTP-bound form of cells, exhibiting low nanomolar potency in KRASG12C-mutant\\nKRAS, although activation still appeared to be partly depend- H358 cells while not impairing the viability of cells driven\\nent on upstream pathway input (Supplementary Fig. S4A by KRASG12D, BRAFV600E, or RTK-driven signaling through\\nand S4B). Finally, although a decrease in GTP-bound KRAS wild-type RAS (Fig. 4B). Interestingly, although KRASG12C/Y96D\\n(representing the active state) was observed in KRASG12C- expression led to marked IC shifts of >100-fold for\\n50\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='Clinical Acquired Resistance to KRASG12C Inhibition ReSeARCH BRIef\\nB\\nMIA PaCa-2\\nG12C G12C/Y96D\\nMRTX849\\n(nmol/L): 0 3 10 30 100 300 0 3 10 30 100 300\\npERK\\npRSK\\nGAPDH\\nE\\nfigure 3. Cellular characterization of KRASY96D in KRASG12C-mutant models. A, Cell viability assays performed with NCI-H358, MIA PaCa-2, and Ba/F3\\ncells infected with retrovirus packaging KRAS (G12C or G12C/Y96D). Cell lines were treated with indicated drugs for 72 hours and the viabilities were\\nmeasured with CellTiter-Glo. B, Western blot analysis was performed after treating MIA PaCa-2 cells stably expressing KRASG12C or KRASG12C/Y96D with\\nMRTX849 for 4 hours. C, MGH1138-1 cells expressing KRASG12C or KRASG12C/Y96D were treated with MRTX849 for 4 hours and subjected to Western blot\\nanalysis (left) and cell viability assay following 72 hours of treatment with the indicated concentrations of MRTX849 (right). D, Western blot analysis\\nof HEK293T cells transiently expressing KRAS mutants after treatment with MRTX849 for 4 hours. e, RAS-GTP pulldown was performed after treating\\nHEK293T stably expressing KRAS mutants with MRTX849.\\nAUGUST 2021 CANCER DISCOVERY | OF5\\nRAF-RBD\\nInput\\nA\\nMRTX849 AMG 510\\nD\\n293T\\nG12C G12C/Y96D\\nMRTX849\\n(nmol/L): 0 3 10 30 100 300 0 3 10 30 100 300\\nKRAS\\npERK\\npRSK\\nGAPDH\\nC 293T\\nMRTX849 G12C G12C/Y96D\\n(nmol/L): 0 3 10 30 100 300 0 3 10 30 100 300\\nKRAS-GTP\\nRAS-GTP\\nKRAS\\nRAS\\npERK\\npRSK\\nGAPDH\\n)%(\\nretiT\\nlleC\\nevitaleR\\nARS-1620\\nLog ARS-1620 (mol/L)\\n853H\\n2-aCaP\\nAIM\\nKRASG12C\\nKRASG12C/Y96D\\n3FaB\\n150\\n100\\n50\\n0\\nLog MRTX849 (mol/L) Log AMG 510 (mol/L)\\nLog AMG 510 (mol/L) Log ARS-1620 (mol/L)\\nLog MRTX849 (mol/L) Log AMG 510 (mol/L) Log ARS-1620 (mol/L)\\nMGH1138-1\\nG12C G12C/Y96D MRTX849\\n(nmol/L): 0 30 300 0 30 300\\npERK\\nERK\\npAKT\\nAKT\\nGAPDH\\n)%(\\nretiT\\nlleC\\nevitaleR\\n150\\n100\\n50\\n0\\n)%(\\nretiT\\nlleC\\nevitaleR\\n150 150\\n100 100\\n50 50\\n0 0\\n−11 −10 −9 −8 −7 −6 −11 −10 −9 −8 −7 −6 −10 −9 −8 −7 −6 −5\\n150 150\\n100 100\\n50 50\\n0 0\\n−11 −10 −9 −8 −7 −6 −11 −10 −9 −8 −7 −6 −10 −9 −8 −7 −6 −5\\nLog MRTX849 (mol/L)\\n150 150 150\\n100 100 100\\n50 50 50\\n0 0 0\\n−12 −11 −10 −9 −8 −7 −11 −10 −9 −8 −7 −6 −10 −9 −8 −7 −6 −5\\nKRASG12C\\n150 KRASG12C/Y96D\\n100\\n50\\n0\\n−9 −8 −7 −6 −5\\nLog MRTX849 (mol/L)\\n)%(\\nretiT\\nlleC\\nevitaleR\\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 4, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='ReSeARCH BRIef Tanaka et al.\\nA B\\nC\\nRM-018\\n150\\n100\\n50\\n0\\n−11 −10 −9 −8 −7 −6\\nfigure 4. Novel KRAS inhibitor RM-018 overcomes KRASG12C/Y96D. A, Mechanism of action of RM-018. B, RM-018 selectively inhibits cell viability in\\ncells harboring KRASG12C. C, Cell viability assays performed with NCI-H358, MIA PaCa-2, Ba/F3, and MGH1138-1 cells stably infected with KRASG12C or\\nKRASG12C/Y96D treated for 72 hours with RM-018. D and e, Western blot analysis performed in MIA PaCa-2 stably expressing KRASG12C or KRASG12C/Y96D (D)\\nand HEK293T cells transiently expressing KRAS mutants (e) after treatment of RM-018 for 4 hours. f, Western blot analysis of MGH1138-1 cells transiently\\nexpressing KRASG12C or KRASG12C/Y96D after treatment with RM-018 for 4 hours. G, HEK293T cells transiently expressing KRAS mutants were treated with\\nthe indicated drug at 100 nmol/L each for 4 hours and then subjected to Western blot analysis.\\nOF6 | CANCER DISCOVERY AUGUST 2021 AACRJournals.org\\n2-aCaP\\nAIM\\n3FaB\\nKRASG12C\\nKRASG12C/Y96D\\n−11 −10 −9 −8 −7 −6 −11 −10 −9 −8 −7 −6 −9 −8 −7 −6 −5\\nLog RM-018 (mol/L) Log RM-018 (mol/L) Log RM-018 (mol/L) Log RM-018 (mol/L)\\n)%(\\nretiT lleC\\nevitaleR\\n150\\n100\\n50\\n0\\n)%(\\nretiT lleC\\nevitaleR\\n150\\n100\\n50\\n0\\n)%(\\nretiT lleC\\nevitaleR\\n150\\n100\\n50\\n0\\n853H\\n)%(\\nretiT lleC\\nevitaleR\\nD E\\nMIA PaCa-2 293T\\nG12C G12C/Y96D G12C G12C/Y96D\\nRM-018 RM-018\\n(nmol/L): 0 1 3 10 30 100 0 1 3 10 30 100 (nmol/L): 0 1 3 10 30 100 0 1 3 10 30 100\\nKRAS KRAS\\npERK pERK\\npRSK\\npRSK\\nGAPDH GAPDH\\nG\\nKRAS\\npERK\\npRSK\\nGAPDH\\nOSMD\\n948XTRM 015\\nGMA\\n810-MR OSMD\\n948XTRM 015\\nGMA\\n810-MR\\nKRASG12C(ON)\\n150\\nInhibitor NCI-H358 (KRASG12C)\\n100 AsPC-1 (KRASG12D)\\nNCI-H1975 (EGFRT790M,L858R, KRASWT)\\n50 A375 (BRAFV600E)\\n0\\n−10 −9 −8 −7 −6\\nCYPA\\n293T\\nG12C G12C/Y96D\\n)%(\\nretiT\\nlleC\\nevitaleR\\nLog RM-018 (mol/L)\\nF\\nMGH1138-1\\nG12C G12C/Y96D\\nRM-018\\n(nmol/L): 0 10 30 100 300 1,000 0 10 30 100 300 1,000\\nKRAS\\npERK\\npRSK\\nGAPDH\\n1-8311\\nHGM\\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nMRTX849 and AMG 510 and ∼20-fold for ARS-1620 (Fig. 3A) inhibit pERK and pRSK levels with similar potency in the\\nrelative to KRASG12C expression alone, the efficacy of RM-018 presence of KRASG12C or KRASG12C/Y96D expression in MIA\\non cell viability was largely unaffected by KRASG12C/Y96D PaCa-2, 293T cells, and the patient-derived KRASG12C-mutant\\nexpression, with IC shifts of only ∼2-fold (Fig. 4C; Sup- NSCLC cell line MGH1138-1 (Fig. 4D–F). Inhibition of cell\\n50\\nplementary Table S4). In addition, RM-018 was able to viability by RM-018 was also unaffected by KRASG12C/Y96D\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 5, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nClinical Acquired Resistance to KRASG12C Inhibition ReSeARCH BRIef\\nexpression in the patient-derived MGH1138-1 model. Fur- resistance after an acquired KRAS resistance mutation\\nthermore, the KRAS mobility shift induced by covalent bind- emerges on the initial KRASG12C inhibitor, and they support\\ning of RM-018 was observed in both cell lines in the presence efforts toward rational design of next-generation inhibitors.\\nof either KRASG12C or KRASG12C/Y96D expression, suggesting Of note, however, the abundance of mutations down-\\nthat binding of RM-018 to KRAS is not abrogated by the stream of RAS suggests that MAPK reactivation alone may\\nKRASY96D mutation. Indeed, although a KRAS mobility shift be sufficient to drive resistance in at least some KRASG12C\\ndue to covalent drug binding was observed in 293T cells cancers. At the time of disease progression on MRTX849,\\nexpressing KRASG12C for MRTX849, AMG 510, and RM-018, this patient’s plasma biopsy specimen revealed a low allele\\nonly RM-018 exhibited this same mobility shift and was fraction of KRASY96D, particularly compared with the sig-\\nable to inhibit downstream signaling in the presence of the nificantly higher allele fraction of KRASG12C (0.4% vs. 31.7%,\\nKRASG12C/Y96D mutation (Fig. 4G). Taken together, these data respectively), suggestive of the presence of polyclonal resist-\\nsuggest that RM-018 retains the ability to bind and inhibit ance to MRTX849 (with KRASG12C/Y96D-harboring tumor cells\\nKRASG12C/Y96D and may represent a potential therapeutic representing a minor subclone). Concordant with this notion\\nstrategy to overcome this acquired resistance mechanism. and prior reports based on preclinical models, subsequent\\nposttreatment plasma samples in this case revealed numer-\\nous mutations affecting various nodes of the RAS–MAPK\\nDiscUssiON\\npathway (including NRAS, BRAF, and MAP2K1 mutations).\\nThe arrival of covalent KRASG12C-selective inhibitors in the Thus, rational combinations of KRASG12C inhibitors with\\nclinic and early signs of activity demonstrated by MRTX849 downstream MAPK pathway inhibitors may be needed to suc-\\nand AMG 510 have generated great enthusiasm (4, 5). How- cessfully prevent or overcome resistance. Although analysis\\never, our experiences across targeted therapies in lung cancer of additional patients is clearly needed to more fully define\\nand other cancers collectively demonstrate that acquired the spectrum of potential acquired resistance mechanisms\\nresistance to the KRASG12C inhibitors will represent an inevi- to KRASG12C inhibitors, this study begins to delineate some\\ntable challenge going forward. Although preclinical studies of the potential resistance alterations that may be observed\\nhave nominated putative mechanisms of up-front resistance, clinically. Ultimately, an iterative discovery process of identi-\\nincluding RAS–MAPK pathway reactivation (4, 8), mecha- fying mechanisms of resistance and validating in vitro with a\\nnisms of acquired resistance to MRTX849 or AMG 510 caus- detailed structural and molecular understanding should help\\ning disease relapse in patients remain unknown. advance the development of novel strategies for therapeutic\\nThe acquired resistance demonstrated in this patient is targeting of KRAS-mutant cancers.\\ninstructive in highlighting several points. First, 10 distinct\\nresistance alterations arose in this patient, all converging on\\nMethODs\\nthe reactivation of RAS–MAPK signaling, suggesting that this\\nmay be a central common mechanism of acquired resistance. Patient Treatment and Specimen Collection\\nRAS reactivation occurred by multiple different mechanisms,\\nThe patient was treated with MRTX849 dosed 600 mg twice\\nincluding (i) activation of another RAS isoform (in this case, daily on the phase I study (KRYSTAL-1) after providing written\\nNRAS); (ii) other KRAS activating mutations in trans (G13D, informed consent (ClinicalTrials.gov identifier: NCT03785249). She' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='NRAS); (ii) other KRAS activating mutations in trans (G13D, informed consent (ClinicalTrials.gov identifier: NCT03785249). She\\nG12V); (iii) potential loss of KRASG12C through a mutational had received two prior lines of therapy. All pre- and posttreat-\\nswitch to a different KRAS mutation in cis, although sup- ment biopsies and genotyping were performed in accordance with\\nported by limited sequencing reads; and (iv) a novel secondary the Massachusetts General Hospital (MGH) institutional review\\nalteration in KRAS (i.e., Y96D), which alters inhibitor binding. board–approved protocol and in accordance with the Declaration\\nStructural modeling predicted that this KRASY96D muta- of Helsinki. The pretreatment tumor specimen was analyzed using\\nthe MGH SNaPshot next-generation sequencing assay (25). All\\ntion disrupts critical hydrogen bonding between the Y96\\ncfDNA samples were sequenced using the commercially available\\nresidue of KRAS and MRTX849. Importantly, we found that\\nGuardant360 assay (Guardant Health). More detailed patient history\\nthis KRASY96D mutation conferred resistance not only to\\nis available in the Supplement.\\nMRTX849 but also to additional KRASG12C-selective inhibi-\\ntors in clinical development, AMG 510 and ARS-1620, high- Cell Lines and Reagents\\nlighting that mutations affecting the Y96 residue of KRAS\\nBa/F3 cells were obtained from the RIKEN BRC Cell Bank (RIKEN\\nmay represent a shared vulnerability for the currently avail-\\nBioResource Center). MGH1138-1 cells were generated from a\\nable KRASG12C inhibitors. Continued investigation into clini- patient with KRASG12C-mutant NSCLC using methods that have\\ncal mechanisms of resistance to KRASG12C inhibitors in larger been previously described (26). Prior to cell line generation, the\\ncohorts of patients will be required to define the spectrum patient provided written informed consent to participate in a Dana-\\nand frequency of KRASY96 mutations (as well as other on- Farber/Harvard Cancer Center institutional review board–approved\\ntarget mutations). Nonetheless, in light of our observations, protocol giving permission for research to be performed on their\\nit may be necessary to develop novel compounds that are able sample. The remaining cell lines were obtained from ATCC or the\\nCenter for Molecular Therapeutics at the MGH Cancer Center, which\\nto target KRASG12C/Y96D to overcome resistance to KRASG12C\\nroutinely performs cell line authentication testing by SNP and short-\\ninhibitors in clinic. RM-018 was identified as a novel KRASG12C\\ntandem repeat analysis. HEK293T cells were maintained in DMEM\\ninhibitor with a distinct mechanism of action for targeting\\nsupplemented with 10% FBS. MIA PaCa-2 and NCI-H358 cells were\\nKRAS, which was uniquely able to overcome KRASG12C/Y96D\\nmaintained in DMEM/F12 supplemented with 10% FBS. LU-65 and\\nacross multiple models. These results suggest that a novel, MGH1138-1 cells were maintained in RPMI supplemented with\\ndistinct KRAS inhibitor could theoretically be used to target 10% FBS. Ba/F3 cells were maintained in DMEM supplemented\\nAUGUST 2021 CANCER DISCOVERY | OF7\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nReSeARCH BRIef Tanaka et al.\\nwith 10% FBS and 10 ng/mL IL3. The KRAS (G12C or G12C/Y96D) Circulating Nucleic Acid Kit (QIAGEN) with 60 minutes of protein-\\ngene was inserted in pMXs-Puro Retroviral Expression Vector, which ase K incubation at 60°C. All other steps were performed according\\nwas purchased from Cell Biolabs. Retrovirus packaging mutated to the manufacturer’s instructions. For ddPCR experiments, DNA\\nKRAS genes were produced with HEK293T cells. After concentra- template (up to 10 μ L, with a total of 20 ng) was added to 12.5 μ L\\ntion of virus with Retro-Concentin Retro Concentration Reagent ddPCR Supermix for Probes (Bio-Rad) and 1.25 μ L custom primer/\\n(System Biosciences), MIA PaCa-2, NCI-H358, and Ba/F3 cells were probe mixture. This reaction mix was added to a DG8 cartridge\\ninfected with the virus packaging either the KRASG12C or KRASG12C/Y96D together with 60 μ L Droplet Generation Oil for Probes (Bio-Rad)\\ngene. After 48 hours of incubation, the cells were treated with puro- and used for droplet generation. Droplets were then transferred to a\\nmycin (1–2 μ g/mL) for another 48 hours. IL3 was withdrawn to 96-well plate (Eppendorf) and then thermal cycled with the following\\nselect for Ba/F3 cells dependent on mutant KRAS signaling after 48 conditions: 5 minutes at 95°C, 40 cycles of 94°C for 30 seconds, 55°C\\nhours of puromycin treatment. The remaining cells were maintained (with a few grades of difference among assays) for 1 minute, followed\\nin media supplemented with puromycin. For transient expression by 98°C for 10 minutes (Ramp Rate 2°C/s). Droplets were analyzed\\nexperiments, a day after seeding the cells, pMXs-Puro-KRASG12C or with the QX200 Droplet Reader (Bio-Rad) for fluorescent measure-\\npMXs-Puro-KRASG12C/Y96D vectors were induced with Lipofectamine ment of FAM and HEX probes. Gating was performed based on posi-\\n2000 Transfection Reagent (ThermoFisher Scientific) following the tive and negative controls, and mutant populations were identified.\\nmanufacturer’s protocol. After 16 to 24 hours of incubation, cells The ddPCR data were analyzed with QuantaSoft analysis software\\nwere treated with inhibitors for 4 hours. AMG 510 was purchased (Bio-Rad) to obtain fractional abundance of the mutant DNA alleles\\nfrom MedChemExpress. MRTX849 and ARS-1620 were purchased in the wild-type/normal background. The quantification of the target\\nfrom Selleck Chemicals. RM-018 was provided by Revolution Medi- molecule was presented as the number of total copies (mutant plus\\ncines, and details of the chemical synthesis can be found in Interna- wild-type) per sample in each reaction. Allelic fraction was calculated\\ntional Patent Application No. PCT/US2020/058841. as follows: AF % = [N /(N + N ) * 100], where N is the num-\\nmut mut wt mut\\nber of mutant alleles and N is the number of wild-type alleles per\\nwt\\nCell Viability Assays reaction. ddPCR analysis of normal control plasma DNA (from cell\\nCell lines were seeded in a 96-well plate at 2 to 10 × 103 cells/well lines) and no DNA template controls was always included. Probe and\\nprimer sequences are available upon request.\\ndepending on cell lines and after 24 hours treated with a serial dilu-\\ntion of drugs and incubated for 72 hours. Cell viability was measured\\nwith CellTiter-Glo (Promega). Authors’ Disclosures\\nN. Tanaka reports other support from Japanese Foundation for\\nWestern Blot Analysis\\nCancer Research during the conduct of the study. J.J. Lin reports\\nCell lines were treated with MRTX849, AMG 510, or RM-018 personal fees and other support from Genentech, personal fees\\nfor 4 hours and lysates were prepared as described previously (27). from C4 Therapeutics, personal fees from Nuvalent, personal\\nAll antibodies were diluted in 5% bovine serum albumin as fol- fees from Blueprint Medicines, personal fees and other support' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='All antibodies were diluted in 5% bovine serum albumin as fol- fees from Blueprint Medicines, personal fees and other support\\nlows: KRAS (Sigma), pERK (Thr202/Tyr204,1:1,000; Cell Signal- from Turning Point Therapeutics, personal fees and other sup-\\ning Technology), p44/42 MAPK (ERK1/2) (1:1,000; Cell Signaling port from Elevation Oncology, personal fees from Pfizer, other\\nTechnology), phospho-RSK1 (T359+S363, 1:1,000; Abcam), phospho- support from Hengrui Therapeutics, other support from Novartis,\\nAKT (Ser473, 1:1,000; Cell Signaling Technology), AKT (1:1,000; Cell other support from Neon Therapeutics, other support from\\nSignaling Technology), and GAPDH (1:1,000; MilliporeSigma). Relay Therapeutics, other support from Bayer, and other support\\nfrom Roche outside the submitted work. L.A. Kiedrowski reports\\nRAS-GTP Pulldown other support from Guardant Health during the conduct of the\\nAfter indicated inhibitor treatment, RAS activity was assessed by study. D. Juric reports grants and personal fees from Novartis,\\nGST-RAF-RBD pulldown (Cell Signaling Technology), followed by grants and personal fees from Genentech, grants and personal fees\\nWestern blot analysis with pan-RAS or RAS isoform–specific antibod- from EMD Serono, grants and personal fees from Eisai, personal\\nies. Pulldown samples and whole-cell lysates were resolved on 4% to fees from Guardant, personal fees from Ipsen, personal fees from\\n12% Bis-Tris gels, and Western blotting was performed using antibod- Petra Pharma, personal fees from Mapkure, personal fees from Vib-\\nies against KRAS (Sigma) and pan-RAS (Cell Signaling Technology). liome, personal fees from Relay Therapeutics, grants from Takeda,\\ngrants and personal fees from Syros, grants from Ribon Therapeu-\\ntics, grants from Infinity Pharmaceuticals, grants from InventisBio,\\nStructural Modeling\\nand grants from Amgen outside the submitted work. J.F. Gainor\\nPublicly available crystal structures of KRASG12C in complex with\\nreports personal fees from Mirati and personal fees from Amgen dur-\\nMRTX849 (PDB:6UT0), AMG 510 (PDB:6OIM), and ARS-1620\\ning the conduct of the study, personal fees from Bristol-Myers Squibb,\\n(PDB:5V9U) were downloaded from the RCSB Protein Data Bank\\npersonal fees from Genentech/Roche, personal fees from Takeda,\\n(PDB; ref. 28). Structures were rendered in PyMol (The PyMOL\\npersonal fees from Lilly, personal fees and other support from Blue-\\nMolecular Graphics System) and analyzed for hydrogen bonds and\\nprint, personal fees from Oncorus, personal fees from Regeneron,\\nother molecular interactions between the KRASG12C inhibitors and\\npersonal fees from Gilead, personal fees from AstraZeneca, personal\\nthe KRAS protein. Structures of Y96 amino acid mutation were gen-\\nfees from Pfizer, grants and personal fees from Novartis, personal\\nerated by Protein Mutagenesis Wizard implemented in PyMol, with\\nfees and other support from Merck, personal fees from Agios, other\\none of the backbone-dependent rotamers manually selected.\\nsupport from Moderna, other support from Jounce, other support\\nfrom Array, other support from Alexo outside the submitted work,\\ncfDNA Extraction and ddPCR\\nand an immediate family member who is an employee with equity at\\nWhole blood was collected by routine phlebotomy in two 10-mL Ironwood Pharmaceuticals. S.J. Klempner reports personal fees from\\nStreck tubes. Plasma was separated within 1 to 4 days of collection BMS, personal fees from Merck, personal fees from Daiichi-Sankyo,\\nthrough two different centrifugation steps (the first at room temper- personal fees from Astellas, personal fees from Pieris Oncology,\\nature for 10 minutes at 1,600 × g and the second at 3,000 × g for the personal fees from Natera, personal fees from Foundation Medicine,\\nsame time and temperature). Plasma was stored at −80°C until cfDNA and other support from Turning Point Therapeutics outside the' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='same time and temperature). Plasma was stored at −80°C until cfDNA and other support from Turning Point Therapeutics outside the\\nextraction. cfDNA was extracted from plasma using the QIAamp submitted work. L. Bar-Peled reports personal fees from Scorpion\\nOF8 | CANCER DISCOVERY AUGUST 2021 AACRJournals.org\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nClinical Acquired Resistance to KRASG12C Inhibition ReSeARCH BRIef\\nTherapeutics outside the submitted work. A.N. Hata reports grants Acknowledgments\\nfrom Amgen, grants from Eli Lilly, grants from Relay Therapeu-\\nRM-018 was kindly provided by Revolution Medicines. This\\ntics, grants from Roche/Genentech, grants from Novartis, grants\\nresearch was supported in part by a Mark Foundation for Cancer\\nfrom Blueprint Medicines, grants from Pfizer, and grants and per-\\nResearch EXTOL Project Grant (J.F. Gainor, A.N. Hata) and a Stand\\nsonal fees from Nuvalent outside the submitted work. R.S. Heist\\nUp To Cancer—American Cancer Society Lung Cancer Dream Team\\nreports personal fees from Novartis, personal fees from Daichii\\nTranslational Research Grant (J.F. Gainor, A.N. Hata; grant number:\\nSankyo, personal fees from EMD Serono, personal fees from Tarveda,\\nSU2C-AACR-DT17–15). Stand Up To Cancer (SU2C) is a division\\npersonal fees from Apollomics, grants from Novartis, grants from\\nof the Entertainment Industry Foundation. The indicated SU2C\\nCorvus, grants from Incyte, grants from Genentech Roche, grants\\nresearch grant is administered by the American Association for\\nfrom Mirati, grants from Turning Point, grants from BMS, grants from\\nCancer Research, a scientific partner of SU2C. J.J. Lin is partially\\nDaichii Sankyo, grants from AbbVie, grants from Agios, grants\\nsupported by funding from NIH R01-CA164273. C. Li is supported\\nfrom Exelixis, and grants from Lilly outside the submitted work.\\nby funding from NIH 1F32CA250231–01. N. Tanaka is partially\\nR.B. Corcoran reports personal fees from AbbVie, grants and per-\\nsupported by funding from US-JFCR Scientist Exchange Program. L.\\nsonal fees from Asana Biosciences, personal fees from Astex Phar-\\nBar-Peled is supported by AACR (19–20–45-BARP), Damon Runyon\\nmaceuticals, grants and personal fees from AstraZeneca, personal\\n(DRR-62–20), and NIH/NCI (R00 CA215249).\\nfees and other support from Avidity Biosciences, personal fees and\\nother support from C4 Therapeutics, personal fees from Chugai,\\npersonal fees from Elicio, other support from Erasca, personal Received March 19, 2021; revised March 30, 2021; accepted April 1,\\nfees from Fog Pharma, personal fees from Guardant Health, 2021; published first April 6, 2021.\\npersonal fees from Ipsen, personal fees and other support from\\nKinnate Biopharma, personal fees from Mirati Therapeutics, personal\\nfees from Natera, grants from Novartis, grants from Lilly, personal fees RefeReNCeS\\nand other support from nRichDx, personal fees from Pfizer, personal 1. Fell JB, Fischer JP, Baer BR, Blake JF, Bouhana K, Briere DM, et al.\\nfees from QIAGEN, personal fees and other support from Remix Identification of the clinical development candidate MRTX849, a\\nTherapeutics, personal fees and other support from Revolution covalent KRAS(G12C) inhibitor for the treatment of cancer. J Med\\nMedicines, personal fees from Roivant, personal fees from Shionogi, Chem 2020;63:6679–93.\\npersonal fees from Tango Therapeutics, personal fees from Taiho, 2. Lanman BA, Allen JR, Allen JG, Amegadzie AK, Ashton KS, Booker SK,\\nand personal fees from Zikani Therapeutics outside the submitted et al. Discovery of a covalent inhibitor of KRAS(G12C) (AMG 510) for\\nwork; in addition, R.B. Corcoran has a patent for Intellectual Prop- the treatment of solid tumors. J Med Chem 2020;63:52–65.\\nerty pending. No disclosures were reported by the other authors. 3. Ostrem JM, Peters U, Sos ML, Wells JA, Shokat KM. K-Ras(G12C)\\ninhibitors allosterically control GTP affinity and effector interac-\\ntions. Nature 2013;503:548–51.\\nAuthors’ Contributions 4. Hallin J, Engstrom LD, Hargis L, Calinisan A, Aranda R, Briere DM,\\nN. Tanaka: Conceptualization, data curation, formal analysis, et al. The KRAS(G12C) inhibitor MRTX849 provides insight toward\\ntherapeutic susceptibility of KRAS-mutant cancers in mouse models' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='N. Tanaka: Conceptualization, data curation, formal analysis, et al. The KRAS(G12C) inhibitor MRTX849 provides insight toward\\ntherapeutic susceptibility of KRAS-mutant cancers in mouse models\\ninvestigation, writing–original draft, writing–review and editing.\\nand patients. Cancer Discov 2020;10:54–71.\\nJ.J. Lin: Conceptualization, data curation, formal analysis, inves-\\n5. Hong DS, Fakih MG, Strickler JH, Desai J, Durm GA, Shapiro GI, et al.\\ntigation, writing–original draft, writing–review and editing. C. Li:\\nKRAS(G12C) inhibition with sotorasib in advanced solid tumors.\\nConceptualization, data curation, formal analysis, investigation, writing–\\nN Engl J Med 2020;383:1207–17.\\noriginal draft, writing–review and editing. M.B. Ryan: Conceptual-\\n6. Li B, Skoulidis F, Falchook G, Sacher A, Velcheti V, Dy G, et al.\\nization, data curation, formal analysis, investigation, writing–original\\nPS01.07 registrational phase 2 trial of sotorasib in KRAS p.G12C\\ndraft, writing–review and editing. J. Zhang: Conceptualization, for-\\nmutant NSCLC: first disclosure of the codebreak 100 primary analy-\\nmal analysis, writing–original draft, writing–review and editing. sis. J Thorac Oncol 2021;16:S61.\\nL.A. Kiedrowski: Conceptualization, formal analysis, writing–review 7. Jänne PA, Rybkin II, Spira AI, Riely GJ, Papadopoulos KP, Sabari JK,\\nand editing. A.G. Michel: Formal analysis, investigation, writing– et al. KRYSTAL-1: activity and safety of adagrasib (MRTX849) in\\nreview and editing. M.U. Syed: Formal analysis, investigation, writing– advanced/metastatic non–small-cell lung cancer (NSCLC) harboring\\nreview and editing. K.A. Fella: Formal analysis, investigation, KRAS G12C mutation. Eur J Cancer 2020;138:S1–S2.\\nwriting–review and editing. M. Sakhi: Formal analysis, investigation, 8. Ryan MB, Fece de la Cruz F, Phat S, Myers DT, Wong E, Shahzade HA,\\nwriting–review and editing. I. Baiev: Formal analysis, investiga- et al. Vertical pathway inhibition overcomes adaptive feedback resist-\\ntion, writing–review and editing. D. Juric: Conceptualization, formal ance to KRAS(G12C) inhibition. Clin Cancer Res 2020;26:1633–43.\\nanalysis, investigation, writing–review and editing. J.F. Gainor: Con- 9. Misale S, Fatherree JP, Cortez E, Li C, Bilton S, Timonina D, et al. KRAS\\nceptualization, formal analysis, investigation, writing–review and editing. G12C NSCLC models are sensitive to direct targeting of KRAS in com-\\nS.J. Klempner: Conceptualization, formal analysis, investigation, bination with PI3K inhibition. Clin Cancer Res 2019;25:796–807.\\nwriting–review and editing. J.K. Lennerz: Formal analysis, investiga- 10. Santana-Codina N, Chandhoke AS, Yu Q, Malachowska B, Kuljanin\\nM, Gikandi A, et al. Defining and targeting adaptations to oncogenic\\ntion, writing–review and editing. G. Siravegna: Conceptualization,\\nKRAS(G12C) inhibition using quantitative temporal proteomics. Cell\\ndata curation, formal analysis, supervision, investigation, writing–\\nRep 2020;30:4584–99.\\noriginal draft, writing–review and editing. L. Bar-Peled: Conceptu-\\n11. Solanki HS, Welsh EA, Fang B, Izumi V, Darville L, Stone B, et al. Cell\\nalization, data curation, formal analysis, supervision, investigation,\\ntype-specific adaptive signaling responses to KRAS(G12C) inhibition.\\nwriting–review and editing. A.N. Hata: Conceptualization, resources,\\nClin Cancer Res 2021;27:2533–48.\\ndata curation, formal analysis, supervision, funding acquisition,\\n12. Xue JY, Zhao Y, Aronowitz J, Mai TT, Vides A, Qeriqi B, et al. Rapid\\nvalidation, investigation, writing–original draft, writing–review and\\nnon-uniform adaptation to conformation-specific KRAS(G12C) inhi-\\nediting. R.S. Heist: Conceptualization, resources, data curation, formal bition. Nature 2020;577:421–5.\\nanalysis, supervision, funding acquisition, validation, investigation, 13. Fedele C, Li S, Teng KW, Foster CJR, Peng D, Ran H, et al. SHP2 inhi-' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='analysis, supervision, funding acquisition, validation, investigation, 13. Fedele C, Li S, Teng KW, Foster CJR, Peng D, Ran H, et al. SHP2 inhi-\\nwriting–original draft, writing–review and editing. R.B. Corcoran: bition diminishes KRASG12C cycling and promotes tumor microen-\\nConceptualization, resources, data curation, formal analysis, supervi- vironment remodeling. J Exp Med 2021;218:e20201414.\\nsion, funding acquisition, validation, investigation, writing–original 14. Kinoshita-Kikuta E, Kinoshita E, Ueda S, Ino Y, Kimura Y, Hirano H,\\ndraft, writing–review and editing. et al. Increase in constitutively active MEK1 species by introduction\\nAUGUST 2021 CANCER DISCOVERY | OF9\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nReSeARCH BRIef Tanaka et al.\\nof MEK1 mutations identified in cancers. Biochim Biophys Acta 22. Zeng M, Lu J, Li L, Feru F, Quan C, Gero TW, et al. Potent and selec-\\nProteins Proteom 2019;1867:62–70. tive covalent quinazoline inhibitors of KRAS G12C. Cell Chem Biol\\n15. Gao Y, Chang MT, McKay D, Na N, Zhou B, Yaeger R, et al. Allele- 2017;24:1005–16.\\nspecific mechanisms of activation of MEK1 mutants determine their 23. Schulze CJ, Bermingham A, Choy TJ, Cregg JJ, Kiss G, Marquez A, et al.\\nproperties. Cancer Discov 2018;8:648–61. Abstract PR10: tri-complex inhibitors of the oncogenic, GTP-bound\\n16. Canon J, Rex K, Saiki AY, Mohr C, Cooke K, Bagal D, et al. The clini- form of KRASG12C overcome RTK-mediated escape mechanisms and\\ncal KRAS(G12C) inhibitor AMG 510 drives anti-tumour immunity. drive tumor regressions in vivo. Mol Cancer Ther 2019;18:PR10.\\nNature 2019;575:217–23. 24. Nichols R, Schulze C, Bermingham A, Choy T, Cregg J, Kiss G, et al.\\n17. Sondka Z, Bamford S, Cole CG, Ward SA, Dunham I, Forbes SA. The A06 Tri-complex inhibitors of the oncogenic, GTP-bound form of\\nCOSMIC Cancer Gene Census: describing genetic dysfunction across KRASG12C overcome RTK-mediated escape mechanisms and drive\\nall human cancers. Nat Rev Cancer 2018;18:696–705. tumor regressions in preclinical models of NSCLC. J Thorac Oncol\\n18. Consortium APG. AACR Project GENIE: powering precision medi- 2020;15:S13–S4.\\ncine through an international consortium. Cancer Discov 2017;7: 25. Zheng Z, Liebers M, Zhelyazkova B, Cao Y, Panditi D, Lynch KD,\\n818–31. et al. Anchored multiplex PCR for targeted next-generation sequenc-\\n19. Fell JB, Fischer JP, Baer BR, Ballard J, Blake JF, Bouhana K, et al. ing. Nat Med 2014;20:1479–84.\\nDiscovery of tetrahydropyridopyrimidines as irreversible covalent 26. Crystal AS, Shaw AT, Sequist LV, Friboulet L, Niederst MJ, Lockerman\\ninhibitors of KRAS-G12C with in vivo activity. ACS Med Chem Lett EL, et al. Patient-derived models of acquired resistance can identify\\n2018;9:1230–4. effective drug combinations for cancer. Science 2014;346:1480–6.\\n20. Janes MR, Zhang J, Li LS, Hansen R, Peters U, Guo X, et al. Targeting 27. Ahronian LG, Sennott EM, Van Allen EM, Wagle N, Kwak EL, Faris\\nKRAS mutant cancers with a covalent G12C-specific inhibitor. Cell JE, et al. Clinical acquired resistance to RAF inhibitor combinations\\n2018;172:578–89. in BRAF-mutant colorectal cancer through MAPK pathway altera-\\n21. Chen H, Smaill JB, Liu T, Ding K, Lu X. Small-molecule inhibitors tions. Cancer Discov 2015;5:358–67.\\ndirectly targeting KRAS as anticancer therapeutics. J Med Chem 28. Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H,\\n2020;63:14404–24. et al. The protein data bank. Nucleic Acids Res 2000;28:235–42.\\nOF10 | CANCER DISCOVERY AUGUST 2021 AACRJournals.org\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 9, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n",
      "page_content='Published OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365\\nG12C\\nClinical Acquired Resistance to KRAS Inhibition through\\na Novel KRAS Switch-II Pocket Mutation and Polyclonal A\\n-\\nlterations Converging on RAS MAPK Reactivation\\nNoritaka Tanaka, Jessica J. Lin, Chendi Li, et al.\\nCancer Discov Published OnlineFirst April 6, 2021.\\nUpdated version Access the most recent version of this article at:\\ndoi:10.1158/2159-8290.CD-21-0365\\nSupplementary Access the most recent supplemental material at:\\nMaterial http://cancerdiscovery.aacrjournals.org/content/suppl/2021/04/03/2159-8290.CD-21-0365.DC1\\nE-mail alerts Sign up to receive free email-alerts related to this article or journal.\\nReprints and To order reprints of this article or to subscribe to the journal, contact the AACR Publications\\nSubscriptions Department atpubs@aacr.org.\\nPermissions To request permission to re-use all or part of this article, use this link\\nhttp://cancerdiscovery.aacrjournals.org/content/early/2021/06/16/2159-8290.CD-21-0365.\\nClick on \"Request Permissions\" which will take you to the Copyright Clearance Center\\'s (CCC)\\nRightslink site.\\nDownloaded from cancerdiscovery.aacrjournals.org on July 28, 2021. © 2021 American Association for Cancer\\nResearch.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Aaron/2021_Cancer Discovery_Hata.pdf', 'page': 10, 'total_pages': 11, 'CreationDate': \"D:20210616091024+05'30'\", 'Creator': 'Adobe InDesign CS6 (Windows)', 'ModDate': \"D:20210728110726-07'00'\", 'Producer': 'Adobe PDF Library 10.0.1', 'Title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'Trapped': 'False'}\n"
     ]
    }
   ],
   "source": [
    "from langchain_community.document_loaders import PDFPlumberLoader\n",
    "loader = PDFPlumberLoader(pdf_file)\n",
    "\n",
    "a = time.time()\n",
    "pages = loader.load_and_split()\n",
    "b = time.time()\n",
    "\n",
    "print(b - a)\n",
    "print(len(pages))\n",
    "\n",
    "for p in pages:\n",
    "  print(p)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.22659611701965332\n",
      "19\n",
      "page_content='AUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF1 \\nResearch Brief\\nClinical Acquired Resistance to KRASG12C \\nInhibition through a Novel KRAS Switch-II \\nPocket Mutation and Polyclonal Alterations \\nConverging on RAS–MAPK Reactivation  \\nNoritaka Tanaka1, Jessica J. Lin1, Chendi Li1, Meagan B. Ryan1, Junbing Zhang1, Lesli A. Kiedrowski2,  \\nAlexa G. Michel1, Mohammed U. Syed1, Katerina A. Fella1, Mustafa Sakhi1, Islam Baiev1, Dejan Juric1,  \\nJustin F. Gainor1, Samuel J. Klempner1, Jochen K. Lennerz3, Giulia Siravegna1, Liron Bar-Peled1,  \\nAaron N. Hata1, Rebecca S. Heist1, and Ryan B. Corcoran1\\nabstract\\nMutant-selective KRASG12C inhibitors, such as MRTX849 (adagrasib) and AMG 510 \\n(sotorasib), have demonstrated efficacy in KRASG12C-mutant cancers, including \\nnon–small cell lung cancer (NSCLC). However, mechanisms underlying clinical acquired resistance to \\nKRASG12C inhibitors remain undetermined. To begin to define the mechanistic spectrum of acquired \\nresistance, we describe a patient with KRASG12C NSCLC who developed polyclonal acquired resistance \\nto MRTX849 with the emergence of 10 heterogeneous resistance alterations in serial cell-free DNA \\nspanning four genes (KRAS, NRAS, BRAF, MAP2K1), all of which converge to reactivate RAS–MAPK \\nsignaling. Notably, a novel KRASY96D mutation affecting the switch-II pocket, to which MRTX849 and \\nother inactive-state inhibitors bind, was identified that interferes with key protein–drug interactions \\nand confers resistance to these inhibitors in engineered and patient-derived KRASG12C cancer models. \\nInterestingly, a novel, functionally distinct tricomplex KRASG12C active-state inhibitor RM-018 retained \\nthe ability to bind and inhibit KRASG12C/Y96D and could overcome resistance.\\nSignificance: In one of the first reports of clinical acquired resistance to KRASG12C inhibitors, our \\ndata suggest polyclonal RAS–MAPK reactivation as a central resistance mechanism. We also identify \\na novel KRAS switch-II pocket mutation that impairs binding and drives resistance to inactive-state \\ninhibitors but is surmountable by a functionally distinct KRASG12C inhibitor.\\n1Massachusetts General Hospital Cancer Center and Department of \\n \\nMedicine, Harvard Medical School, Boston, Massachusetts. 2Guardant Health, \\nRedwood City, California. 3Department of Pathology, Massachusetts \\n \\nGeneral Hospital, Boston, Massachusetts.\\nNote: Supplementary data for this article are available at Cancer Discovery \\nOnline (http://cancerdiscovery.aacrjournals.org/).\\nN. Tanaka, J.J. Lin, and C. Li contributed equally to this work.\\nCorresponding Authors: Aaron N. Hata, Massachusetts General Hospital \\nCancer Center, 149 13th Street,  7th Floor, Boston, MA 02129. Phone: 617-\\n724-3442; E-mail: ahata@mgh.harvard.edu; Rebecca S. Heist, Massachu\\xad\\nsetts General Hospital Cancer Center, 55 Fruit Street, Boston, MA 02114. \\nPhone: 617-724-4000; E-mail: rheist@partners.org; and Ryan B. Corcoran, \\n \\nMassachusetts General Hospital Cancer Center, 149 13th Street, 7th Floor, \\nBoston, MA 02129. Phone: 617-726-8599; E-mail: rbcorcoran@partners.org\\nCancer Discov 2021;11:1–10\\ndoi: 10.1158/2159-8290.CD-21-0365\\n©2021 American Association for Cancer Research\\nINTRODUCTION\\nThe development of compounds that bind covalently to \\ncysteine 12 in KRASG12C cancers has ushered in a new era in \\nefforts to target KRAS directly. Biochemically, these agents \\nlock KRAS in its inactive GDP-bound conformation, thereby \\ninhibiting downstream signaling, leading to preclinical \\n \\nantitumor responses (1–3). The lead clinical compounds soto-\\nrasib (AMG 510) and adagrasib (MRTX849) have advanced \\nrapidly and demonstrated tolerability and single-agent \\nactivity across KRASG12C-mutant cancers (4, 5). In patients \\nwith advanced non–small cell lung cancer (NSCLC) harbor-\\ning KRASG12C (which comprises approximately 13% of all \\n \\nlung adenocarcinomas), AMG 510 and MRTX849 have \\n \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 0, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='lung adenocarcinomas), AMG 510 and MRTX849 have \\n \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 0, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF2\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\ndemonstrated meaningful efficacy with objective response \\nrates of 37% and 45%, as well as disease control rates of 81% \\nand 96%, respectively (6, 7). AMG 510 has recently received \\nBreakthrough Therapy designation from the FDA for the \\ntreatment of patients with advanced KRASG12C-mutant \\nNSCLC following at least one prior systemic therapy. Mul-\\ntiple ongoing trials seek to augment responses to KRASG12C \\ninhibitors through combination strategies.\\nPreclinical studies with MRTX849 and other KRASG12C \\ninhibitors have suggested several mechanisms of up-front \\nresistance, including reactivation of ERK-dependent sig\\xad\\nnaling to bypass KRASG12C blockade (4). Prior work by our \\ngroup and others has identified adaptive RAS pathway feed-\\nback reactivation as a key mechanism of primary resistance \\nto KRASG12C inhibition (8–13). However, the key mecha-\\nnisms of clinical acquired resistance to KRASG12C inhibitors \\nare currently unknown. Here, as an initial effort to charac-\\nterize the clinical landscape of potential acquired resistance \\nmechanisms to KRASG12C inhibitors, we present a patient \\nwith KRASG12C-mutant NSCLC who developed acquired \\nresistance to MRTX849, characterized by the emergence \\nof 10 individual resistance alterations involving four RAS–\\nMAPK genes. All of these resistance alterations converge \\nto reactivate RAS–MAPK signaling, implicating this as a \\npotential central mechanism of acquired resistance. We \\nalso identify a novel KRASY96D resistance mutation in the \\nswitch-II pocket of KRAS through serial cell-free DNA \\n(cfDNA) analysis. Through structural modeling and in vitro \\nfunctional studies, we find that KRASY96D confers resist-\\nance to multiple KRASG12C inhibitors currently in clinical \\ndevelopment but identify a novel active-state KRASG12C \\ninhibitor, RM-018, that is able to overcome KRASG12C/Y96D-\\nmediated resistance.\\nRESULTS\\nHeterogeneous Acquired Resistance Alterations \\nConverge on RAS–MAPK Reactivation\\nA 67-year-old woman with metastatic KRASG12C-mutant \\nNSCLC was treated on the dose-expansion cohort of the \\nphase 1 study of MRTX849 (NCT03785249; Methods; fur-\\nther detailed in Supplementary Methods). Initial scans \\nshowed a 32% reduction in tumor size (by RECIST v1.1), \\nbut after approximately 4 months of treatment, the patient \\ndeveloped progressive disease, and the patient discontinued \\ntherapy at 5.5 months (Fig. 1A). To identify putative mecha-\\nnisms of acquired resistance to MRTX849 in this patient, we \\nassessed cfDNA using a targeted next-generation sequencing \\nassay (Guardant360; Guardant Health) and droplet digital \\nPCR (ddPCR). Upon development of acquired resistance, the \\noriginal KRASG12C and TP53F338fs variants present in pretreat-\\nment tumor and cfDNA were again detected in cfDNA but \\nwere accompanied by the emergence of 10 distinct muta-\\ntions affecting RAS–MAPK components KRAS, NRAS, BRAF, \\nand MAP2K1 (which encodes the MEK1 protein) identified \\nacross cfDNA specimens obtained after disease progression \\n(Fig. 1B; Supplementary Table S1). The lower allele frequen-\\ncies of these alterations relative to the truncal KRASG12C \\nand TP53 mutations are consistent with the emergence of \\nthese mutations in heterogeneous subclonal populations. \\nThese included three activating NRAS mutations (NRASQ61L, \\nNRASQ61K, NRASQ61R), which can drive active RAS signaling \\nin a KRAS-independent manner, and BRAFV600E, which can \\nmaintain MAPK signaling downstream of KRASG12C in the \\npresence of MRTX849 (Supplementary Fig. S1). Three MAP2K1  \\nmutations (MAP2K1K57N, MAP2K1Q56P, MAP2K1E102-I103del) pre\\xad\\nviously demonstrated to be activating and known to be \\ninvolved in resistance to upstream MAPK pathway inhibitors \\n(i.e., BRAF inhibitors) were also identified (14, 15).\\nIn addition, three KRAS mutations emerged in the postpro-\\ngression cfDNA. Two of these mutations are the known activat-' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='(i.e., BRAF inhibitors) were also identified (14, 15).\\nIn addition, three KRAS mutations emerged in the postpro-\\ngression cfDNA. Two of these mutations are the known activat-\\ning mutations KRASG13D and KRASG12V, and mutant-selective \\nKRASG12C inhibitors have previously been shown to be inef-\\nfective against these mutations (4, 16). A deeper analysis of \\nindividual sequencing reads from cfDNA suggested that these \\nmutations seemed to occur in trans to the original KRASG12C \\nmutation (Supplementary Fig. S2A and S2B), likely arising \\nin the remaining wild-type copy of KRAS, which appeared to \\nbe retained based on pretreatment tumor sequencing (Sup-\\nplementary Table S2). However, it is not possible from the \\ncfDNA data to confirm that these mutations coexist in cells \\nthat also harbor the original KRASG12C mutation. Notably, a \\nsingle, well-supported family of sequencing reads from the \\nsame original template molecule showed the concurrent pres-\\nence of both nucleotide changes corresponding to KRASG12C \\nand KRASG12V in cis on the same strand, which would encode \\nfor a KRASG12F mutation. While it is not possible to confirm \\nthe presence of this mutation based on a single read family, \\nthis finding raises the possibility that cis mutations resulting \\nin “loss” of the original KRASG12C mutation and conversion \\nto a different KRAS mutation might be another potential \\nmechanism of resistance. Notably, all putative resistance \\nmutations identified are predicted to converge on reactiva-\\ntion of RAS–MAPK pathway signaling, suggesting that this \\nmay represent a common primary mechanism of acquired \\nresistance to KRASG12C inhibitors (Fig. 1C).\\nInterestingly, the third KRAS mutation identified, KRASY96D, \\nrepresents a novel mutation that is not known to be activat-\\ning. Notably, although KRAS is the most commonly mutated \\noncogene in human cancer, a search of two large tumor muta-\\ntional databases—COSMIC and GENIE, which collectively \\ncontain >450,000 molecularly characterized cancers (17, \\n18)—did not reveal a single previously identified mutation at \\nthe KRASY96 locus among >75,000 cases with documented \\nKRAS mutations (Supplementary Table S3). However, the \\nY96 residue is associated with the switch-II pocket to which \\nMRTX849 and other inactive-state KRASG12C inhibitors bind, \\nsuggesting that the previously undescribed Y96D mutation \\nmay have a novel and specific role in driving resistance to \\nKRASG12C inhibitors.\\nStructural Modeling of KRASG12C/Y96D\\nTo understand the significance of the acquired KRASY96D \\nmutation, we performed structural modeling of the G12C-\\nmutant and G12C/Y96D double-mutant KRAS proteins \\nbound to the KRASG12C inhibitors MRTX849, AMG 510, and \\nARS-1620 (Fig. 2). These three inhibitors bind the GDP state \\nof KRASG12C and exploit a cryptic pocket formed by the central \\nβ sheet of RAS and switch-II (first identified by Ostrem and \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Clinical Acquired Resistance to KRASG12C Inhibition\\nRESEARCH BRIEF\\n\\t\\nAUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF3 \\nFigure 1.\\u2002Acquired resistance to KRASG12C inhibitor MRTX849 (adagrasib). A, Computed tomography images of the patient’s axillary lymph node \\nmetastasis at baseline, during response to MRTX849, and at progression on MRTX849. B, Variant allele fractions of mutations detected in the patient’s \\nserial plasma samples. †, indicates the mutations were detected by ddPCR but not by plasma next-generation sequencing. C, Alterations detected in post-\\nMRTX849 cfDNA include acquired mutations in KRAS as well as multiple components of the MAPK signaling cascade. *, KRASG12F represents a potential \\nresistance mechanism supported by limited sequencing reads, as shown in Supplementary Fig. S2.\\nA\\nBaseline \\n6 weeks \\n4 months \\nMRTX849 \\nPre-\\nMRTX849\\nPre-\\nMRTX849\\nDays post-MRTX849\\ndiscontinuation:\\n0\\n9\\n51\\nTP53 F338fs\\n36.8%\\n0.22%\\n8.8%\\n10.1%\\n14.3%\\nKRAS G12C\\n21.3%\\n0.12%\\n31.7%\\n47.1%\\n24.9%\\nKRAS G12V\\n-\\n-\\n-\\n-\\n0.09%\\nKRAS G13D\\n-\\n-\\n-\\n0.13%†\\n0.04%\\nKRAS Y96D\\n-\\n-\\n0.4%\\n0.2%\\n-\\nNRAS Q61L\\n-\\n-\\n-\\n0.2%\\n-\\nNRAS Q61R\\n-\\n-\\n-\\n-\\n0.02%\\nNRAS Q61K\\n-\\n-\\n0.6%\\n0.6%\\n0.9%\\nBRAF V600E\\n-\\n-\\n0.1%\\n0.1%\\n0.5%\\nMAP2K1 K57N\\n-\\n-\\n0.05%†\\n-\\n0.3%\\nMAP2K1 Q56P\\n-\\n-\\n-\\n-\\n0.1%\\nMAP2K1 E102_I103del\\n-\\n-\\n-\\n0.12%†\\n0.2%\\ncfDNA\\nTumor \\nB\\nC\\nA/B/ \\nCRAF \\nKRAS \\n G12C \\nMEK \\nRTK \\nProliferation and Survival  \\nC\\nRTK \\nWT \\nRAS \\nERK \\nNRAS Q61K \\nNRAS Q61R \\nNRAS Q61L \\nKRAS G12V \\nKRAS G13D \\nKRAS G12F* \\nKRAS Y96D \\nBRAF V600E \\nMAP2K1 K57N \\nMAP2K1 Q56P \\nMAP2K1 E102_I103 del \\nSecondary \\nCis* \\nTrans \\ncolleagues; ref. 3). To determine the effects of the amino acid \\nsubstitution at the Y96 locus, crystal structures of MRTX849, \\nAMG 510, and ARS-1620 bound to KRASG12C were modeled \\nfor interactions with the Y96 residue within the switch-II \\npocket (16, 19–21). The hydroxyl group of Y96 forms a direct \\nhydrogen bond with the pyrimidine ring of MTRX849, which \\nis abolished with the Y96D mutation. Y96D also disrupts the \\nwater-mediated hydrogen bond between Y96 and a carboxyl \\ngroup on AMG 510. Finally, although Y96 does not form a \\ndirect hydrogen bond with ARS-1620, it stabilizes the inter-\\naction with ARS-1620 through pi-stacking with the phenyl \\nring of Y96, which is disrupted with the Y96D mutation. In \\naddition, by introducing a negatively charged amino acid, the \\nY96D mutation changes the hydrophobic nature of the bind-\\ning pocket for all three compounds to a substantially more \\nhydrophilic pocket, which may further destabilize binding.\\nFunctional Characterization of KRASY96D\\nTo assess whether KRASY96D can mediate resistance to \\nMRTX849 and other inactive-state KRASG12C inhibitors, we \\nexpressed KRASG12C or the KRASG12C/Y96D double mutant \\nin NCI-H358 (KRASG12C-mutant NSCLC), MIA PaCa-2 \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 2, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF4\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\nFigure 2.\\u2002Structural basis for resistance to KRASG12C inhibition conferred by KRASY96D. Shown are the modeled crystal structures of MRTX849 (6UT0), \\nAMG 510 (6OIM), and ARS-1620 (5V9U) bound to KRASG12C (top) and KRASG12C/Y96D (bottom), highlighting the loss of the hydrogen bonds between \\nMRTX849 or AMG 510 and the Y96 residue and the disruption of the switch-II pocket dynamics between ARS-1620 and KRASG12C/Y96D.\\nARS-1620\\nMRTX849\\nAMG 510\\nG12C\\nG12C/Y96D\\n(KRASG12C-mutant pancreatic ductal adenocarcinoma), and \\nBa/F3 cells, which lack endogenous KRASG12C but become \\noncogene dependent upon withdrawal of IL3. In cell via-\\nbility assays, relative to KRASG12C-expressing controls, cells \\nexpressing KRASG12C/Y96D showed marked resistance to \\nthree KRASG12C inhibitors, with IC50 shifts of >100-fold for \\nMRTX849 and AMG 510 and ∼20-fold for ARS-1620 (Fig. 3A; \\nSupplementary Table S4).\\nConsistent with the effects on cell viability, RAS–MAPK \\npathway activity, as measured by levels of phosphorylated \\n \\nERK (pERK) and pRSK, was sustained in KRASG12C/Y96D-\\nexpressing MIA PaCa-2 cells even at high concentrations \\nof MRTX849, relative to cells expressing KRASG12C alone \\n(Fig. 3B). Similarly, in KRASG12C-mutant NSCLC cells in \\nwhich PI3K signaling is driven by mutant KRAS, including \\nan existing patient-derived model MGH1138-1, persistent \\npERK and pAKT levels were observed with KRASG12C/Y96D \\nin the presence of MRTX849, relative to KRASG12C expres-\\nsion alone (Fig. 3C; Supplementary Fig. S3). KRASG12C/Y96D \\nalso drove marked resistance to MRTX849 in the patient-\\nderived MGH1138-1 model. Furthermore, in 293T cells, \\nwhich lack endogenous KRASG12C expression, MRTX849 \\nwas unable to inhibit pERK levels driven by KRASG12C/Y96D  \\n(Fig. 3D). Because MRTX849 and other inactive-state \\nKRASG12C inhibitors bind covalently to KRASG12C, an elec-\\ntrophoretic mobility shift of drug-adducted KRASG12C can \\nbe observed upon drug binding due to increased molecular \\nweight. However, this mobility shift was no longer observed \\nwhen 293T cells expressing KRASG12C/Y96D were treated with \\nMRTX849, suggesting that the Y96D mutation may abro-\\ngate inhibitor binding. Notably, KRASG12C/Y96D appeared to \\nhave higher basal activation than KRASG12C, as measured \\nby a higher proportion of the active GTP-bound form of \\nKRAS, although activation still appeared to be partly depend-\\nent on upstream pathway input (Supplementary Fig. S4A \\nand S4B). Finally, although a decrease in GTP-bound KRAS \\n(representing the active state) was observed in KRASG12C-\\nexpressing cells treated with MRTX849, levels of active \\n \\nGTP-bound KRAS were maintained in KRASG12C/Y96D-expressing \\ncells (Fig. 3E; ref. 22). These results suggest that the KRASY96D \\nmutation disrupts KRASG12C inhibitor binding, leading to \\nsustained KRAS signaling and therapeutic resistance.\\nThe Active State KRASG12C Inhibitor RM-018 \\nOvercomes KRASG12C/Y96D\\nAs KRASG12C/Y96D conferred resistance to multiple KRASG12C \\ninhibitors currently in clinical development, suggestive of \\nshared vulnerability for this class of inhibitors, we sought \\nto identify whether a structurally and functionally distinct \\nKRASG12C inhibitor might retain potency against this resist-\\nance mutation. RM-018 is a novel KRASG12C inhibitor that \\nbinds specifically to the GTP-bound, active [“RAS(ON)”] state \\nof KRASG12C. RM-018 is a “tricomplex” KRAS inhibitor, which \\nexploits a highly abundant chaperone protein, cyclophilin A, \\nto bind and inhibit KRASG12C, as previously described (Fig. 4A; \\n \\nstructure shown in Supplementary Fig. S5; refs. 23, 24). \\nBriefly, upon entering the cell, RM-018 forms a “binary com-\\nplex” with cyclophilin A. This binary complex can associate \\nwith the active state of KRASG12C, aided by protein–protein \\nsurface interactions between cyclophilin A and KRAS, and \\nforms a covalent bond with KRASG12C in a mutant-selec-' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='with the active state of KRASG12C, aided by protein–protein \\nsurface interactions between cyclophilin A and KRAS, and \\nforms a covalent bond with KRASG12C in a mutant-selec-\\ntive manner. This resultant “tricomplex” inhibits KRASG12C \\nthrough binding of cyclophilin A, leading to steric occlu-\\nsion and preventing the association of downstream effector \\nproteins. Given the markedly different mechanism of action \\nof this class of inhibitor, we hypothesized that the inhibitory \\nactivity of RM-018 might be differentially affected by KRASY96D \\ncompared with inactive-state KRASG12C inhibitors.\\nRM-018 demonstrated selectivity for KRASG12C-driven \\ncells, exhibiting low nanomolar potency in KRASG12C-mutant \\nH358 cells while not impairing the viability of cells driven \\nby KRASG12D, BRAFV600E, or RTK-driven signaling through \\nwild-type RAS (Fig. 4B). Interestingly, although KRASG12C/Y96D \\n \\nexpression led to marked IC50 shifts of >100-fold for \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Clinical Acquired Resistance to KRASG12C Inhibition\\nRESEARCH BRIEF\\n\\t\\nAUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF5 \\nFigure 3.\\u2002Cellular characterization of KRASY96D in KRASG12C-mutant models. A, Cell viability assays performed with NCI-H358, MIA PaCa-2, and Ba/F3 \\ncells infected with retrovirus packaging KRAS (G12C or G12C/Y96D). Cell lines were treated with indicated drugs for 72 hours and the viabilities were \\nmeasured with CellTiter-Glo. B, Western blot analysis was performed after treating MIA PaCa-2 cells stably expressing KRASG12C or KRASG12C/Y96D with \\nMRTX849 for 4 hours. C, MGH1138-1 cells expressing KRASG12C or KRASG12C/Y96D were treated with MRTX849 for 4 hours and subjected to Western blot \\nanalysis (left) and cell viability assay following 72 hours of treatment with the indicated concentrations of MRTX849 (right). D, Western blot analysis \\nof HEK293T cells transiently expressing KRAS mutants after treatment with MRTX849 for 4 hours. E, RAS-GTP pulldown was performed after treating \\nHEK293T stably expressing KRAS mutants with MRTX849.\\nB\\n0 \\n3 10 30 100 300 0 \\n3 10 30 100 300 \\nG12C \\nG12C/Y96D \\nMRTX849\\n(nmol/L):\\nMIA PaCa-2 \\npERK\\npRSK\\nGAPDH\\nE\\nRAF-RBD \\nInput \\npERK\\npRSK\\nKRAS\\nKRAS-GTP\\nGAPDH\\nRAS\\nRAS-GTP\\nMRTX849\\n(nmol/L):\\nG12C \\n0 \\n3 10 \\n100 \\n30 \\n300 \\nG12C/Y96D \\n0 \\n3 10 \\n100 \\n30 \\n300 \\n293T \\nKRAS\\npERK\\npRSK\\n0 \\n3 10 30 100 300 0 \\n3 10 30 100 300 \\nMRTX849\\n(nmol/L):\\nG12C \\nG12C/Y96D \\nGAPDH\\nD\\nA\\n293T \\nC\\nAMG 510 \\nMRTX849 \\nRelative Cell Titer (%) \\nLog ARS-1620 (mol/L) \\nARS-1620 \\nH358 \\nMIA PaCa-2 \\nKRASG12C\\nKRASG12C/Y96D \\nBaF3 \\nLog ARS-1620 (mol/L) \\nLog AMG 510 (mol/L) \\nLog MRTX849 (mol/L) \\nLog ARS-1620 (mol/L) \\nLog MRTX849 (mol/L) \\nLog AMG 510 (mol/L) \\nLog AMG 510 (mol/L) \\npERK\\nERK \\nMRTX849\\n(nmol/L):\\nGAPDH\\npAKT\\nAKT\\n0 \\n30 300 0 30 300 \\nG12C \\nG12C/Y96D \\nMGH1138-1 \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\nLog MRTX849 (mol/L) \\n−11\\n−12\\n−11\\n−10\\n−9\\n−8\\n−7\\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\n−10\\n−9\\n−8\\n−7\\n−6\\n−5\\n−10\\n−9\\n−8\\n−7\\n−6\\n−11\\n−10\\n−10\\n−9\\n−8\\n−7\\n−6\\n−5\\n−9\\n−8\\n−7\\n−6\\n−11\\n−10\\n−10\\n−9\\n−8\\n−7\\n−6\\n−5\\n−9\\n−8\\n−7\\n−6\\nKRASG12C \\nKRASG12C/Y96D \\n−9\\n−8\\n−7\\n−6\\n−5\\n0\\n50\\n100\\n150\\nLog MRTX849 (mol/L)\\nRelative Cell Titer (%)\\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 4, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF6\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\nFigure 4.\\u2002Novel KRAS inhibitor RM-018 overcomes KRASG12C/Y96D. A, Mechanism of action of RM-018. B, RM-018 selectively inhibits cell viability in \\ncells harboring KRASG12C. C, Cell viability assays performed with NCI-H358, MIA PaCa-2, Ba/F3, and MGH1138-1 cells stably infected with KRASG12C or \\nKRASG12C/Y96D treated for 72 hours with RM-018. D and E, Western blot analysis performed in MIA PaCa-2 stably expressing KRASG12C or KRASG12C/Y96D (D) \\nand HEK293T cells transiently expressing KRAS mutants (E) after treatment of RM-018 for 4 hours. F, Western blot analysis of MGH1138-1 cells transiently \\nexpressing KRASG12C or KRASG12C/Y96D after treatment with RM-018 for 4 hours. G, HEK293T cells transiently expressing KRAS mutants were treated with \\nthe indicated drug at 100 nmol/L each for 4 hours and then subjected to Western blot analysis.\\n150\\n100\\n50\\n0\\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\nA\\nB\\nC\\nRM-018 \\nMIA PaCa-2 \\nBaF3 \\nKRASG12C \\nKRASG12C/Y96D \\nLog RM-018 (mol/L) \\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\nLog RM-018 (mol/L) \\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\nLog RM-018 (mol/L) \\n−9\\n−8\\n−7\\n−6\\n−5\\nLog RM-018 (mol/L) \\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\nH358 \\nD\\nMIA PaCa-2 \\n0 \\n1 \\n3 \\n10 30 100 0 \\n1 \\n3 \\n10 30 100 \\nG12C \\nG12C/Y96D \\nRM-018\\n(nmol/L):\\npERK\\npRSK\\nGAPDH\\nKRAS\\nG\\nE\\n0 \\n3 \\n10 30 100 \\n0 \\n3 10 \\n30 100 \\nRM-018\\n (nmol/L):\\nG12C \\nG12C/Y96D \\n1 \\n1 \\npERK\\npRSK\\nGAPDH\\nKRAS\\n293T \\nKRAS\\npERK\\npRSK\\nGAPDH\\nDMSO \\nMRTX849 \\nAMG 510 \\nRM-018 \\nDMSO \\nMRTX849 \\nAMG 510 \\nRM-018 \\nG12C \\nG12C/Y96D \\n293T \\n150\\nInhibitor\\nCYPA\\nKRASG12C(ON)\\n100\\n50\\n0\\n−10\\n−9\\n−8\\n−7\\n−6\\nNCI-H358 (KRASG12C) \\nAsPC-1 (KRASG12D) \\nNCI-H1975 (EGFRT790M,L858R, KRASWT) \\nA375 (BRAFV600E) \\nRelative Cell Titer (%) \\nLog RM-018 (mol/L) \\nMGH1138-1 \\nKRAS\\npERK\\nGAPDH\\npRSK\\n0 \\n10 \\n30 100 300 1,000 0 10 30 100 300 1,000 \\nG12C \\nG12C/Y96D \\nRM-018\\n(nmol/L):\\nF \\nMGH 1138-1 \\nMRTX849 and AMG 510 and ∼20-fold for ARS-1620 (Fig. 3A) \\nrelative to KRASG12C expression alone, the efficacy of RM-018 \\non cell viability was largely unaffected by KRASG12C/Y96D  \\nexpression, with IC50 shifts of only ∼2-fold (Fig. 4C; Sup-\\nplementary Table S4). In addition, RM-018 was able to \\ninhibit pERK and pRSK levels with similar potency in the \\npresence of KRASG12C or KRASG12C/Y96D expression in MIA \\nPaCa-2, 293T cells, and the patient-derived KRASG12C-mutant \\nNSCLC cell line MGH1138-1 (Fig. 4D–F). Inhibition of cell \\nviability by RM-018 was also unaffected by KRASG12C/Y96D  \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 5, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Clinical Acquired Resistance to KRASG12C Inhibition\\nRESEARCH BRIEF\\n\\t\\nAUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF7 \\nexpression in the patient-derived MGH1138-1 model. Fur-\\nthermore, the KRAS mobility shift induced by covalent bind-\\ning of RM-018 was observed in both cell lines in the presence \\nof either KRASG12C or KRASG12C/Y96D expression, suggesting \\nthat binding of RM-018 to KRAS is not abrogated by the \\nKRASY96D mutation. Indeed, although a KRAS mobility shift \\ndue to covalent drug binding was observed in 293T cells \\nexpressing KRASG12C for MRTX849, AMG 510, and RM-018, \\nonly RM-018 exhibited this same mobility shift and was \\nable to inhibit downstream signaling in the presence of the \\nKRASG12C/Y96D mutation (Fig. 4G). Taken together, these data \\nsuggest that RM-018 retains the ability to bind and inhibit \\nKRASG12C/Y96D and may represent a potential therapeutic \\nstrategy to overcome this acquired resistance mechanism.\\nDISCUSSION\\nThe arrival of covalent KRASG12C-selective inhibitors in the \\nclinic and early signs of activity demonstrated by MRTX849 \\nand AMG 510 have generated great enthusiasm (4, 5). How-\\never, our experiences across targeted therapies in lung cancer \\nand other cancers collectively demonstrate that acquired \\nresistance to the KRASG12C inhibitors will represent an inevi-\\ntable challenge going forward. Although preclinical studies \\nhave nominated putative mechanisms of up-front resistance, \\nincluding RAS–MAPK pathway reactivation (4, 8), mecha-\\nnisms of acquired resistance to MRTX849 or AMG 510 caus-\\ning disease relapse in patients remain unknown.\\nThe acquired resistance demonstrated in this patient is \\ninstructive in highlighting several points. First, 10 distinct \\nresistance alterations arose in this patient, all converging on \\nthe reactivation of RAS–MAPK signaling, suggesting that this \\nmay be a central common mechanism of acquired resistance. \\nRAS reactivation occurred by multiple different mechanisms, \\nincluding (i) activation of another RAS isoform (in this case, \\nNRAS); (ii) other KRAS activating mutations in trans (G13D, \\nG12V); (iii) potential loss of KRASG12C through a mutational \\nswitch to a different KRAS mutation in cis, although sup-\\nported by limited sequencing reads; and (iv) a novel secondary \\nalteration in KRAS (i.e., Y96D), which alters inhibitor binding.\\nStructural modeling predicted that this KRASY96D muta-\\ntion disrupts critical hydrogen bonding between the Y96 \\nresidue of KRAS and MRTX849. Importantly, we found that \\nthis KRASY96D mutation conferred resistance not only to \\nMRTX849 but also to additional KRASG12C-selective inhibi-\\ntors in clinical development, AMG 510 and ARS-1620, high-\\nlighting that mutations affecting the Y96 residue of KRAS \\nmay represent a shared vulnerability for the currently avail-\\nable KRASG12C inhibitors. Continued investigation into clini-\\ncal mechanisms of resistance to KRASG12C inhibitors in larger \\ncohorts of patients will be required to define the spectrum \\nand frequency of KRASY96 mutations (as well as other on-\\ntarget mutations). Nonetheless, in light of our observations, \\nit may be necessary to develop novel compounds that are able \\nto target KRASG12C/Y96D to overcome resistance to KRASG12C \\ninhibitors in clinic. RM-018 was identified as a novel KRASG12C \\ninhibitor with a distinct mechanism of action for targeting \\nKRAS, which was uniquely able to overcome KRASG12C/Y96D \\nacross multiple models. These results suggest that a novel, \\ndistinct KRAS inhibitor could theoretically be used to target \\n \\nresistance after an acquired KRAS resistance mutation \\nemerges on the initial KRASG12C inhibitor, and they support \\nefforts toward rational design of next-generation inhibitors.\\nOf note, however, the abundance of mutations down-\\nstream of RAS suggests that MAPK reactivation alone may \\nbe sufficient to drive resistance in at least some KRASG12C \\ncancers. At the time of disease progression on MRTX849, \\nthis patient’s plasma biopsy specimen revealed a low allele' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='be sufficient to drive resistance in at least some KRASG12C \\ncancers. At the time of disease progression on MRTX849, \\nthis patient’s plasma biopsy specimen revealed a low allele \\nfraction of KRASY96D, particularly compared with the sig-\\nnificantly higher allele fraction of KRASG12C (0.4% vs. 31.7%, \\nrespectively), suggestive of the presence of polyclonal resist-\\nance to MRTX849 (with KRASG12C/Y96D-harboring tumor cells \\nrepresenting a minor subclone). Concordant with this notion \\nand prior reports based on preclinical models, subsequent \\nposttreatment plasma samples in this case revealed numer-\\nous mutations affecting various nodes of the RAS–MAPK \\npathway (including NRAS, BRAF, and MAP2K1 mutations). \\nThus, rational combinations of KRASG12C inhibitors with \\ndownstream MAPK pathway inhibitors may be needed to suc-\\ncessfully prevent or overcome resistance. Although analysis \\nof additional patients is clearly needed to more fully define \\nthe spectrum of potential acquired resistance mechanisms \\nto KRASG12C inhibitors, this study begins to delineate some \\nof the potential resistance alterations that may be observed \\nclinically. Ultimately, an iterative discovery process of identi-\\nfying mechanisms of resistance and validating in vitro with a \\ndetailed structural and molecular understanding should help \\nadvance the development of novel strategies for therapeutic \\ntargeting of KRAS-mutant cancers.\\nMETHODS\\nPatient Treatment and Specimen Collection\\nThe patient was treated with MRTX849 dosed 600 mg twice \\ndaily on the phase I study (KRYSTAL-1) after providing written \\ninformed consent (ClinicalTrials.gov identifier: NCT03785249). She \\nhad received two prior lines of therapy. All pre- and posttreat-\\nment biopsies and genotyping were performed in accordance with \\nthe Massachusetts General Hospital (MGH) institutional review \\nboard–approved protocol and in accordance with the Declaration \\nof Helsinki. The pretreatment tumor specimen was analyzed using \\nthe MGH SNaPshot next-generation sequencing assay (25). All \\ncfDNA samples were sequenced using the commercially available \\nGuardant360 assay (Guardant Health). More detailed patient history \\nis available in the Supplement.\\nCell Lines and Reagents\\nBa/F3 cells were obtained from the RIKEN BRC Cell Bank (RIKEN \\nBioResource Center). MGH1138-1 cells were generated from a \\npatient with KRASG12C-mutant NSCLC using methods that have \\nbeen previously described (26). Prior to cell line generation, the \\npatient provided written informed consent to participate in a Dana-\\nFarber/Harvard Cancer Center institutional review board–approved \\nprotocol giving permission for research to be performed on their \\nsample. The remaining cell lines were obtained from ATCC or the \\nCenter for Molecular Therapeutics at the MGH Cancer Center, which \\nroutinely performs cell line authentication testing by SNP and short-\\ntandem repeat analysis. HEK293T cells were maintained in DMEM \\nsupplemented with 10% FBS. MIA PaCa-2 and NCI-H358 cells were \\nmaintained in DMEM/F12 supplemented with 10% FBS. LU-65 and \\nMGH1138-1 cells were maintained in RPMI supplemented with \\n10% FBS. Ba/F3 cells were maintained in DMEM supplemented \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF8\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\nwith 10% FBS and 10 ng/mL IL3. The KRAS (G12C or G12C/Y96D) \\ngene was inserted in pMXs-Puro Retroviral Expression Vector, which \\nwas purchased from Cell Biolabs. Retrovirus packaging mutated \\nKRAS genes were produced with HEK293T cells. After concentra-\\ntion of virus with Retro-Concentin Retro Concentration Reagent \\n \\n(System Biosciences), MIA PaCa-2, NCI-H358, and Ba/F3 cells were \\ninfected with the virus packaging either the KRASG12C or KRASG12C/Y96D \\ngene. After 48 hours of incubation, the cells were treated with puro-\\nmycin (1–2 μg/mL) for another 48 hours. IL3 was withdrawn to \\nselect for Ba/F3 cells dependent on mutant KRAS signaling after 48 \\nhours of puromycin treatment. The remaining cells were maintained \\nin media supplemented with puromycin. For transient expression \\nexperiments, a day after seeding the cells, pMXs-Puro-KRASG12C or \\npMXs-Puro-KRASG12C/Y96D vectors were induced with Lipofectamine \\n2000 Transfection Reagent (ThermoFisher Scientific) following the \\nmanufacturer’s protocol. After 16 to 24 hours of incubation, cells \\nwere treated with inhibitors for 4 hours. AMG 510 was purchased \\nfrom MedChemExpress. MRTX849 and ARS-1620 were purchased \\nfrom Selleck Chemicals. RM-018 was provided by Revolution Medi-\\ncines, and details of the chemical synthesis can be found in Interna-\\ntional Patent Application No. PCT/US2020/058841.\\nCell Viability Assays\\nCell lines were seeded in a 96-well plate at 2 to 10 × 103 cells/well \\ndepending on cell lines and after 24 hours treated with a serial dilu-\\ntion of drugs and incubated for 72 hours. Cell viability was measured \\nwith CellTiter-Glo (Promega).\\nWestern Blot Analysis\\nCell lines were treated with MRTX849, AMG 510, or RM-018 \\nfor 4 hours and lysates were prepared as described previously (27). \\nAll antibodies were diluted in 5% bovine serum albumin as fol-\\nlows: KRAS (Sigma), pERK (Thr202/Tyr204,1:1,000; Cell Signal-\\ning Technology), p44/42 MAPK (ERK1/2) (1:1,000; Cell Signaling \\n \\nTechnology), phospho-RSK1 (T359+S363, 1:1,000; Abcam), phospho- \\nAKT (Ser473, 1:1,000; Cell Signaling Technology), AKT (1:1,000; Cell \\nSignaling Technology), and GAPDH (1:1,000; MilliporeSigma).\\nRAS-GTP Pulldown\\nAfter indicated inhibitor treatment, RAS activity was assessed by \\nGST-RAF-RBD pulldown (Cell Signaling Technology), followed by \\nWestern blot analysis with pan-RAS or RAS isoform–specific antibod-\\nies. Pulldown samples and whole-cell lysates were resolved on 4% to \\n12% Bis-Tris gels, and Western blotting was performed using antibod-\\nies against KRAS (Sigma) and pan-RAS (Cell Signaling Technology).\\nStructural Modeling\\nPublicly available crystal structures of KRASG12C in complex with \\nMRTX849 (PDB:6UT0), AMG 510 (PDB:6OIM), and ARS-1620 \\n(PDB:5V9U) were downloaded from the RCSB Protein Data Bank \\n(PDB; ref. 28). Structures were rendered in PyMol (The PyMOL \\nMolecular Graphics System) and analyzed for hydrogen bonds and \\nother molecular interactions between the KRASG12C inhibitors and \\nthe KRAS protein. Structures of Y96 amino acid mutation were gen-\\nerated by Protein Mutagenesis Wizard implemented in PyMol, with \\none of the backbone-dependent rotamers manually selected.\\ncfDNA Extraction and ddPCR\\nWhole blood was collected by routine phlebotomy in two 10-mL \\nStreck tubes. Plasma was separated within 1 to 4 days of collection \\nthrough two different centrifugation steps (the first at room temper-\\nature for 10 minutes at 1,600 × g and the second at 3,000 × g for the \\nsame time and temperature). Plasma was stored at −80°C until cfDNA \\nextraction. cfDNA was extracted from plasma using the QIAamp \\n \\nCirculating Nucleic Acid Kit (QIAGEN) with 60 minutes of protein-\\nase K incubation at 60°C. All other steps were performed according \\nto the manufacturer’s instructions. For ddPCR experiments, DNA \\ntemplate (up to 10 μL, with a total of 20 ng) was added to 12.5 μL' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='ase K incubation at 60°C. All other steps were performed according \\nto the manufacturer’s instructions. For ddPCR experiments, DNA \\ntemplate (up to 10 μL, with a total of 20 ng) was added to 12.5 μL \\nddPCR Supermix for Probes (Bio-Rad) and 1.25 μL custom primer/\\nprobe mixture. This reaction mix was added to a DG8 cartridge \\ntogether with 60 μL Droplet Generation Oil for Probes (Bio-Rad) \\nand used for droplet generation. Droplets were then transferred to a \\n96-well plate (Eppendorf) and then thermal cycled with the following \\nconditions: 5 minutes at 95°C, 40 cycles of 94°C for 30 seconds, 55°C \\n(with a few grades of difference among assays) for 1 minute, followed \\nby 98°C for 10 minutes (Ramp Rate 2°C/s). Droplets were analyzed \\nwith the QX200 Droplet Reader (Bio-Rad) for fluorescent measure-\\nment of FAM and HEX probes. Gating was performed based on posi-\\ntive and negative controls, and mutant populations were identified. \\nThe ddPCR data were analyzed with QuantaSoft analysis software \\n(Bio-Rad) to obtain fractional abundance of the mutant DNA alleles \\nin the wild-type/normal background. The quantification of the target \\nmolecule was presented as the number of total copies (mutant plus \\nwild-type) per sample in each reaction. Allelic fraction was calculated \\nas follows: AF % = [Nmut/(Nmut + Nwt) * 100], where Nmut is the num-\\nber of mutant alleles and Nwt is the number of wild-type alleles per \\nreaction. ddPCR analysis of normal control plasma DNA (from cell \\nlines) and no DNA template controls was always included. Probe and \\nprimer sequences are available upon request.\\nAuthors’ Disclosures\\nN. Tanaka reports other support from Japanese Foundation for \\nCancer Research during the conduct of the study. J.J. Lin reports \\npersonal fees and other support from Genentech, personal fees \\nfrom C4 Therapeutics, personal fees from Nuvalent, personal \\nfees from Blueprint Medicines, personal fees and other support \\nfrom Turning Point Therapeutics, personal fees and other sup-\\nport from Elevation Oncology, personal fees from Pfizer, other \\nsupport from Hengrui Therapeutics, other support from Novartis, \\nother support from Neon Therapeutics, other support from \\nRelay Therapeutics, other support from Bayer, and other support \\nfrom Roche outside the submitted work. L.A. Kiedrowski reports \\nother support from Guardant Health during the conduct of the \\nstudy. D. Juric reports grants and personal fees from Novartis, \\ngrants and personal fees from Genentech, grants and personal fees \\nfrom EMD Serono, grants and personal fees from Eisai, personal \\n \\nfees from Guardant, personal fees from Ipsen, personal fees from \\nPetra Pharma, personal fees from Mapkure, personal fees from Vib-\\nliome, personal fees from Relay Therapeutics, grants from Takeda, \\ngrants and personal fees from Syros, grants from Ribon Therapeu-\\ntics, grants from Infinity Pharmaceuticals, grants from InventisBio, \\n \\nand grants from Amgen outside the submitted work. J.F. Gainor \\nreports personal fees from Mirati and personal fees from Amgen dur-\\ning the conduct of the study, personal fees from Bristol-Myers Squibb, \\npersonal fees from Genentech/Roche, personal fees from Takeda, \\npersonal fees from Lilly, personal fees and other support from Blue-\\nprint, personal fees from Oncorus, personal fees from Regeneron, \\npersonal fees from Gilead, personal fees from AstraZeneca, personal \\nfees from Pfizer, grants and personal fees from Novartis, personal \\n \\nfees and other support from Merck, personal fees from Agios, other \\nsupport from Moderna, other support from Jounce, other support \\nfrom Array, other support from Alexo outside the submitted work, \\nand an immediate family member who is an employee with equity at \\nIronwood Pharmaceuticals. S.J. Klempner reports personal fees from \\nBMS, personal fees from Merck, personal fees from Daiichi-Sankyo, \\npersonal fees from Astellas, personal fees from Pieris Oncology, \\npersonal fees from Natera, personal fees from Foundation Medicine,' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='personal fees from Astellas, personal fees from Pieris Oncology, \\npersonal fees from Natera, personal fees from Foundation Medicine, \\nand other support from Turning Point Therapeutics outside the \\nsubmitted work. L. Bar-Peled reports personal fees from Scorpion \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Clinical Acquired Resistance to KRASG12C Inhibition\\nRESEARCH BRIEF\\n\\t\\nAUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF9 \\nTherapeutics outside the submitted work. A.N. Hata reports grants \\nfrom Amgen, grants from Eli Lilly, grants from Relay Therapeu-\\ntics, grants from Roche/Genentech, grants from Novartis, grants \\nfrom Blueprint Medicines, grants from Pfizer, and grants and per-\\nsonal fees from Nuvalent outside the submitted work. R.S. Heist \\nreports personal fees from Novartis, personal fees from Daichii \\n \\nSankyo, personal fees from EMD Serono, personal fees from Tarveda, \\npersonal fees from Apollomics, grants from Novartis, grants from \\nCorvus, grants from Incyte, grants from Genentech Roche, grants \\nfrom Mirati, grants from Turning Point, grants from BMS, grants from \\n \\nDaichii Sankyo, grants from AbbVie, grants from Agios, grants \\nfrom Exelixis, and grants from Lilly outside the submitted work. \\nR.B. Corcoran reports personal fees from AbbVie, grants and per-\\nsonal fees from Asana Biosciences, personal fees from Astex Phar-\\nmaceuticals, grants and personal fees from AstraZeneca, personal \\nfees and other support from Avidity Biosciences, personal fees and \\nother support from C4 Therapeutics, personal fees from Chugai, \\npersonal fees from Elicio, other support from Erasca, personal \\nfees from Fog Pharma, personal fees from Guardant Health, \\npersonal fees from Ipsen, personal fees and other support from \\n \\nKinnate Biopharma, personal fees from Mirati Therapeutics, personal \\n \\nfees from Natera, grants from Novartis, grants from Lilly, personal fees \\n \\nand other support from nRichDx, personal fees from Pfizer, personal \\nfees from QIAGEN, personal fees and other support from Remix \\nTherapeutics, personal fees and other support from Revolution \\nMedicines, personal fees from Roivant, personal fees from Shionogi, \\npersonal fees from Tango Therapeutics, personal fees from Taiho, \\nand personal fees from Zikani Therapeutics outside the submitted \\nwork; in addition, R.B. Corcoran has a patent for Intellectual Prop-\\nerty pending. No disclosures were reported by the other authors.\\nAuthors’ Contributions\\nN. Tanaka: Conceptualization, data curation, formal analysis, \\ninvestigation, writing–original draft, writing–review and editing. \\n \\nJ.J. Lin: Conceptualization, data curation, formal analysis, inves-\\ntigation, writing–original draft, writing–review and editing. C. Li: \\nConceptualization, data curation, formal analysis, investigation, writing– \\noriginal draft, writing–review and editing. M.B. Ryan: Conceptual-\\nization, data curation, formal analysis, investigation, writing–original \\n \\ndraft, writing–review and editing. J. Zhang: Conceptualization, for-\\nmal analysis, writing–original draft, writing–review and editing. \\n \\nL.A. Kiedrowski: Conceptualization, formal analysis, writing–review \\nand editing. A.G. Michel: Formal analysis, investigation, writing–\\nreview and editing. M.U. Syed: Formal analysis, investigation, writing– \\nreview and editing. K.A. Fella: Formal analysis, investigation, \\n \\nwriting–review and editing. M. Sakhi: Formal analysis, investigation, \\nwriting–review and editing. I. Baiev: Formal analysis, investiga-\\ntion, writing–review and editing. D. Juric: Conceptualization, formal \\n \\nanalysis, investigation, writing–review and editing. J.F. Gainor: Con-\\nceptualization, formal analysis, investigation, writing–review and editing. \\n \\nS.J. Klempner: Conceptualization, formal analysis, investigation, \\nwriting–review and editing. J.K. Lennerz: Formal analysis, investiga-\\ntion, writing–review and editing. G. Siravegna: Conceptualization, \\ndata curation, formal analysis, supervision, investigation, writing–\\noriginal draft, writing–review and editing. L. Bar-Peled: Conceptu-\\nalization, data curation, formal analysis, supervision, investigation, \\nwriting–review and editing. A.N. Hata: Conceptualization, resources, \\ndata curation, formal analysis, supervision, funding acquisition,' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='writing–review and editing. A.N. Hata: Conceptualization, resources, \\ndata curation, formal analysis, supervision, funding acquisition, \\n \\nvalidation, investigation, writing–original draft, writing–review and \\nediting. R.S. Heist: Conceptualization, resources, data curation, formal \\nanalysis, supervision, funding acquisition, validation, investigation, \\nwriting–original draft, writing–review and editing. R.B. Corcoran:  \\nConceptualization, resources, data curation, formal analysis, supervi-\\nsion, funding acquisition, validation, investigation, writing–original \\ndraft, writing–review and editing.\\nAcknowledgments\\nRM-018 was kindly provided by Revolution Medicines. This \\nresearch was supported in part by a Mark Foundation for Cancer \\nResearch EXTOL Project Grant (J.F. Gainor, A.N. Hata) and a Stand \\nUp To Cancer—American Cancer Society Lung Cancer Dream Team \\nTranslational Research Grant (J.F. Gainor, A.N. Hata; grant number: \\nSU2C-AACR-DT17–15). Stand Up To Cancer (SU2C) is a division \\nof the Entertainment Industry Foundation. The indicated SU2C \\nresearch grant is administered by the American Association for \\nCancer Research, a scientific partner of SU2C. J.J. Lin is partially \\nsupported by funding from NIH R01-CA164273. C. Li is supported \\nby funding from NIH 1F32CA250231–01. N. Tanaka is partially \\nsupported by funding from US-JFCR Scientist Exchange Program. L. \\nBar-Peled is supported by AACR (19–20–45-BARP), Damon Runyon \\n(DRR-62–20), and NIH/NCI (R00 CA215249).\\nReceived March 19, 2021; revised March 30, 2021; accepted April 1, \\n2021; published first April 6, 2021.\\nReferences\\n\\t 1.\\t Fell JB, Fischer JP, Baer BR, Blake JF, Bouhana K, Briere DM, et al. \\nIdentification of the clinical development candidate MRTX849, a \\ncovalent KRAS(G12C) inhibitor for the treatment of cancer. J Med \\nChem 2020;63:6679–93.\\n\\t 2.\\t Lanman BA, Allen JR, Allen JG, Amegadzie AK, Ashton KS, Booker SK, \\n \\net al. Discovery of a covalent inhibitor of KRAS(G12C) (AMG 510) for \\nthe treatment of solid tumors. J Med Chem 2020;63:52–65.\\n\\t 3.\\t Ostrem JM, Peters U, Sos ML, Wells JA, Shokat KM. K-Ras(G12C) \\ninhibitors allosterically control GTP affinity and effector interac-\\ntions. Nature 2013;503:548–51.\\n\\t 4.\\t Hallin J, Engstrom LD, Hargis L, Calinisan A, Aranda R, Briere DM, \\net al. The KRAS(G12C) inhibitor MRTX849 provides insight toward \\ntherapeutic susceptibility of KRAS-mutant cancers in mouse models \\nand patients. Cancer Discov 2020;10:54–71.\\n\\t 5.\\t Hong DS, Fakih MG, Strickler JH, Desai J, Durm GA, Shapiro GI, et al. \\n \\nKRAS(G12C) inhibition with sotorasib in advanced solid tumors. \\n \\nN Engl J Med 2020;383:1207–17.\\n\\t 6.\\t Li B, Skoulidis F, Falchook G, Sacher A, Velcheti V, Dy G, et al. \\nPS01.07 registrational phase 2 trial of sotorasib in KRAS p.G12C \\nmutant NSCLC: first disclosure of the codebreak 100 primary analy-\\nsis. J Thorac Oncol 2021;16:S61.\\n\\t 7.\\t Jänne PA, Rybkin II, Spira AI, Riely GJ, Papadopoulos KP, Sabari JK, \\net al. KRYSTAL-1: activity and safety of adagrasib (MRTX849) in \\nadvanced/metastatic non–small-cell lung cancer (NSCLC) harboring \\nKRAS G12C mutation. Eur J Cancer 2020;138:S1–S2.\\n\\t 8.\\t Ryan MB, Fece de la Cruz F, Phat S, Myers DT, Wong E, Shahzade HA, \\net al. Vertical pathway inhibition overcomes adaptive feedback resist-\\nance to KRAS(G12C) inhibition. Clin Cancer Res 2020;26:1633–43.\\n\\t 9.\\t Misale S, Fatherree JP, Cortez E, Li C, Bilton S, Timonina D, et al. KRAS \\nG12C NSCLC models are sensitive to direct targeting of KRAS in com-\\nbination with PI3K inhibition. Clin Cancer Res 2019;25:796–807.\\n\\t\\n10.\\t Santana-Codina N, Chandhoke AS, Yu Q, Malachowska B, Kuljanin \\nM, Gikandi A, et al. Defining and targeting adaptations to oncogenic \\nKRAS(G12C) inhibition using quantitative temporal proteomics. Cell \\nRep 2020;30:4584–99.\\n\\t\\n11.\\t Solanki HS, Welsh EA, Fang B, Izumi V, Darville L, Stone B, et al. Cell \\ntype-specific adaptive signaling responses to KRAS(G12C) inhibition. \\nClin Cancer Res 2021;27:2533–48.' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='11.\\t Solanki HS, Welsh EA, Fang B, Izumi V, Darville L, Stone B, et al. Cell \\ntype-specific adaptive signaling responses to KRAS(G12C) inhibition. \\nClin Cancer Res 2021;27:2533–48.\\n\\t\\n12.\\t Xue JY, Zhao Y, Aronowitz J, Mai TT, Vides A, Qeriqi B, et al. Rapid \\nnon-uniform adaptation to conformation-specific KRAS(G12C) inhi-\\nbition. Nature 2020;577:421–5.\\n\\t\\n13.\\t Fedele C, Li S, Teng KW, Foster CJR, Peng D, Ran H, et al. SHP2 inhi-\\nbition diminishes KRASG12C cycling and promotes tumor microen-\\nvironment remodeling. J Exp Med 2021;218:e20201414.\\n\\t\\n14.\\t Kinoshita-Kikuta E, Kinoshita E, Ueda S, Ino Y, Kimura Y, Hirano H, \\net al. Increase in constitutively active MEK1 species by \\xad\\nintroduction \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF10\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\nof MEK1 mutations identified in cancers. Biochim Biophys Acta \\n \\nProteins Proteom 2019;1867:62–70.\\n\\t\\n15.\\t Gao Y, Chang MT, McKay D, Na N, Zhou B, Yaeger R, et al. Allele-\\nspecific mechanisms of activation of MEK1 mutants determine their \\nproperties. Cancer Discov 2018;8:648–61.\\n\\t\\n16.\\t Canon J, Rex K, Saiki AY, Mohr C, Cooke K, Bagal D, et al. The clini-\\ncal KRAS(G12C) inhibitor AMG 510 drives anti-tumour immunity. \\nNature 2019;575:217–23.\\n\\t\\n17.\\t Sondka Z, Bamford S, Cole CG, Ward SA, Dunham I, Forbes SA. The \\nCOSMIC Cancer Gene Census: describing genetic dysfunction across \\nall human cancers. Nat Rev Cancer 2018;18:696–705.\\n\\t\\n18.\\t Consortium APG. AACR Project GENIE: powering precision medi-\\ncine through an international consortium. Cancer Discov 2017;7: \\n818–31.\\n\\t\\n19.\\t Fell JB, Fischer JP, Baer BR, Ballard J, Blake JF, Bouhana K, et al. \\nDiscovery of tetrahydropyridopyrimidines as irreversible covalent \\ninhibitors of KRAS-G12C with in vivo activity. ACS Med Chem Lett \\n2018;9:1230–4.\\n\\t\\n20.\\t Janes MR, Zhang J, Li LS, Hansen R, Peters U, Guo X, et al. Targeting \\nKRAS mutant cancers with a covalent G12C-specific inhibitor. Cell \\n2018;172:578–89.\\n\\t\\n21.\\t Chen H, Smaill JB, Liu T, Ding K, Lu X. Small-molecule inhibitors \\ndirectly targeting KRAS as anticancer therapeutics. J Med Chem \\n2020;63:14404–24.\\n\\t\\n22.\\t Zeng M, Lu J, Li L, Feru F, Quan C, Gero TW, et al. Potent and selec-\\ntive covalent quinazoline inhibitors of KRAS G12C. Cell Chem Biol \\n2017;24:1005–16.\\n\\t\\n23.\\t Schulze CJ, Bermingham A, Choy TJ, Cregg JJ, Kiss G, Marquez A, et al. \\nAbstract PR10: tri-complex inhibitors of the oncogenic, GTP-bound \\nform of KRASG12C overcome RTK-mediated escape mechanisms and \\ndrive tumor regressions in vivo. Mol Cancer Ther 2019;18:PR10.\\n\\t\\n24.\\t Nichols R, Schulze C, Bermingham A, Choy T, Cregg J, Kiss G, et al. \\nA06 Tri-complex inhibitors of the oncogenic, GTP-bound form of \\nKRASG12C overcome RTK-mediated escape mechanisms and drive \\ntumor regressions in preclinical models of NSCLC. J Thorac Oncol \\n2020;15:S13–S4.\\n\\t\\n25.\\t Zheng Z, Liebers M, Zhelyazkova B, Cao Y, Panditi D, Lynch KD, \\n \\net al. Anchored multiplex PCR for targeted next-generation sequenc-\\ning. Nat Med 2014;20:1479–84.\\n\\t\\n26.\\t Crystal AS, Shaw AT, Sequist LV, Friboulet L, Niederst MJ, Lockerman \\n \\nEL, et al. Patient-derived models of acquired resistance can identify \\neffective drug combinations for cancer. Science 2014;346:1480–6.\\n\\t\\n27.\\t Ahronian LG, Sennott EM, Van Allen EM, Wagle N, Kwak EL, Faris \\nJE, et al. Clinical acquired resistance to RAF inhibitor combinations \\nin BRAF-mutant colorectal cancer through MAPK pathway altera-\\ntions. Cancer Discov 2015;5:358–67.\\n\\t\\n28.\\t Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, \\net al. The protein data bank. Nucleic Acids Res 2000;28:235–42.\\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 9, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n",
      "page_content='Published OnlineFirst April 6, 2021.\\nCancer Discov \\n  \\nNoritaka Tanaka, Jessica J. Lin, Chendi Li, et al. \\n  \\nMAPK Reactivation\\n−\\nlterations Converging on RAS\\na Novel KRAS Switch-II Pocket Mutation and Polyclonal A\\nthrough\\n Inhibition \\nG12C\\nClinical Acquired Resistance to KRAS\\n  \\nUpdated version\\n  \\n \\n10.1158/2159-8290.CD-21-0365\\ndoi:\\nAccess the most recent version of this article at:\\n  \\nMaterial\\nSupplementary\\n  \\n \\nhttp://cancerdiscovery.aacrjournals.org/content/suppl/2021/04/03/2159-8290.CD-21-0365.DC1\\nAccess the most recent supplemental material at:\\n  \\n  \\n  \\n  \\n  \\n  \\n  \\nE-mail alerts\\n related to this article or journal.\\nSign up to receive free email-alerts\\n  \\nSubscriptions\\nReprints and \\n  \\n.\\npubs@aacr.org\\nDepartment at\\nTo order reprints of this article or to subscribe to the journal, contact the AACR Publications\\n  \\nPermissions\\n  \\nRightslink site. \\nClick on \"Request Permissions\" which will take you to the Copyright Clearance Center\\'s (CCC)\\n.\\nhttp://cancerdiscovery.aacrjournals.org/content/early/2021/06/16/2159-8290.CD-21-0365\\nTo request permission to re-use all or part of this article, use this link\\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365' metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 10, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}\n"
     ]
    }
   ],
   "source": [
    "from langchain_community.document_loaders import PyMuPDFLoader\n",
    "loader = PyMuPDFLoader(pdf_file)\n",
    "\n",
    "a = time.time()\n",
    "pages = loader.load_and_split()\n",
    "b = time.time()\n",
    "\n",
    "print(b - a)\n",
    "print(len(pages))\n",
    "\n",
    "for p in pages:\n",
    "  print(p)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CHARS: 3996, WORDS: 506\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 240, WORDS: 28\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3942, WORDS: 521\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3079, WORDS: 432\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 2924, WORDS: 364\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3964, WORDS: 532\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 1135, WORDS: 145\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 2425, WORDS: 280\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 2725, WORDS: 353\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3987, WORDS: 554\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3371, WORDS: 449\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3941, WORDS: 552\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3981, WORDS: 592\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 452, WORDS: 56\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3929, WORDS: 484\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3959, WORDS: 554\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 884, WORDS: 120\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 3066, WORDS: 429\n",
      "--------------------------------------------------------------------------------\n",
      "CHARS: 1307, WORDS: 162\n",
      "--------------------------------------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "for p in pages:\n",
    "  pg = p.page_content\n",
    "  print(f\"CHARS: {len(pg)}, WORDS: {len(pg.split(\" \"))}\")\n",
    "  print(\"-\" * 80)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As an alterniative to the langchain pdf loaders / splitters, we consider parsers designed for scientific papers: https://github.com/titipata/scipdf_parser"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "None\n"
     ]
    }
   ],
   "source": [
    "import scipdf\n",
    "article_dict = scipdf.parse_pdf_to_dict(pdf_file)\n",
    "print(article_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Splitters\n",
    "Default splitters are based on page. We'll try to split by section / subsection."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "expected string or bytes-like object, got 'Document'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[37], line 8\u001b[0m\n\u001b[1;32m      1\u001b[0m pdf \u001b[38;5;241m=\u001b[39m loader\u001b[38;5;241m.\u001b[39mload()\n\u001b[1;32m      3\u001b[0m splitter \u001b[38;5;241m=\u001b[39m RecursiveCharacterTextSplitter(\n\u001b[1;32m      4\u001b[0m   chunk_size \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m1000\u001b[39m,\n\u001b[1;32m      5\u001b[0m   chunk_overlap \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m50\u001b[39m\n\u001b[1;32m      6\u001b[0m )\n\u001b[0;32m----> 8\u001b[0m pages \u001b[38;5;241m=\u001b[39m \u001b[43msplitter\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mcreate_documents\u001b[49m\u001b[43m(\u001b[49m\u001b[43mpdf\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/github/RAG-Chatbot/.venv/lib/python3.12/site-packages/langchain_text_splitters/base.py:78\u001b[0m, in \u001b[0;36mTextSplitter.create_documents\u001b[0;34m(self, texts, metadatas)\u001b[0m\n\u001b[1;32m     76\u001b[0m index \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m0\u001b[39m\n\u001b[1;32m     77\u001b[0m previous_chunk_len \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m0\u001b[39m\n\u001b[0;32m---> 78\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m chunk \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msplit_text\u001b[49m\u001b[43m(\u001b[49m\u001b[43mtext\u001b[49m\u001b[43m)\u001b[49m:\n\u001b[1;32m     79\u001b[0m     metadata \u001b[38;5;241m=\u001b[39m copy\u001b[38;5;241m.\u001b[39mdeepcopy(_metadatas[i])\n\u001b[1;32m     80\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_add_start_index:\n",
      "File \u001b[0;32m~/github/RAG-Chatbot/.venv/lib/python3.12/site-packages/langchain_text_splitters/character.py:110\u001b[0m, in \u001b[0;36mRecursiveCharacterTextSplitter.split_text\u001b[0;34m(self, text)\u001b[0m\n\u001b[1;32m    109\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21msplit_text\u001b[39m(\u001b[38;5;28mself\u001b[39m, text: \u001b[38;5;28mstr\u001b[39m) \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m>\u001b[39m List[\u001b[38;5;28mstr\u001b[39m]:\n\u001b[0;32m--> 110\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_split_text\u001b[49m\u001b[43m(\u001b[49m\u001b[43mtext\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_separators\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/github/RAG-Chatbot/.venv/lib/python3.12/site-packages/langchain_text_splitters/character.py:80\u001b[0m, in \u001b[0;36mRecursiveCharacterTextSplitter._split_text\u001b[0;34m(self, text, separators)\u001b[0m\n\u001b[1;32m     78\u001b[0m     separator \u001b[38;5;241m=\u001b[39m _s\n\u001b[1;32m     79\u001b[0m     \u001b[38;5;28;01mbreak\u001b[39;00m\n\u001b[0;32m---> 80\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[43mre\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msearch\u001b[49m\u001b[43m(\u001b[49m\u001b[43m_separator\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mtext\u001b[49m\u001b[43m)\u001b[49m:\n\u001b[1;32m     81\u001b[0m     separator \u001b[38;5;241m=\u001b[39m _s\n\u001b[1;32m     82\u001b[0m     new_separators \u001b[38;5;241m=\u001b[39m separators[i \u001b[38;5;241m+\u001b[39m \u001b[38;5;241m1\u001b[39m :]\n",
      "File \u001b[0;32m/usr/local/Cellar/python@3.12/3.12.2_1/Frameworks/Python.framework/Versions/3.12/lib/python3.12/re/__init__.py:177\u001b[0m, in \u001b[0;36msearch\u001b[0;34m(pattern, string, flags)\u001b[0m\n\u001b[1;32m    174\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21msearch\u001b[39m(pattern, string, flags\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m0\u001b[39m):\n\u001b[1;32m    175\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"Scan through string looking for a match to the pattern, returning\u001b[39;00m\n\u001b[1;32m    176\u001b[0m \u001b[38;5;124;03m    a Match object, or None if no match was found.\"\"\"\u001b[39;00m\n\u001b[0;32m--> 177\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43m_compile\u001b[49m\u001b[43m(\u001b[49m\u001b[43mpattern\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mflags\u001b[49m\u001b[43m)\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msearch\u001b[49m\u001b[43m(\u001b[49m\u001b[43mstring\u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[0;31mTypeError\u001b[0m: expected string or bytes-like object, got 'Document'"
     ]
    }
   ],
   "source": [
    "pdf = loader.load()\n",
    "\n",
    "splitter = RecursiveCharacterTextSplitter(\n",
    "  chunk_size = 1000,\n",
    "  chunk_overlap = 50\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(page_content='AUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF1 \\nResearch Brief\\nClinical Acquired Resistance to KRASG12C \\nInhibition through a Novel KRAS Switch-II \\nPocket Mutation and Polyclonal Alterations \\nConverging on RAS–MAPK Reactivation  \\nNoritaka Tanaka1, Jessica J. Lin1, Chendi Li1, Meagan B. Ryan1, Junbing Zhang1, Lesli A. Kiedrowski2,  \\nAlexa G. Michel1, Mohammed U. Syed1, Katerina A. Fella1, Mustafa Sakhi1, Islam Baiev1, Dejan Juric1,  \\nJustin F. Gainor1, Samuel J. Klempner1, Jochen K. Lennerz3, Giulia Siravegna1, Liron Bar-Peled1,  \\nAaron N. Hata1, Rebecca S. Heist1, and Ryan B. Corcoran1\\nabstract\\nMutant-selective KRASG12C inhibitors, such as MRTX849 (adagrasib) and AMG 510 \\n(sotorasib), have demonstrated efficacy in KRASG12C-mutant cancers, including \\nnon–small cell lung cancer (NSCLC). However, mechanisms underlying clinical acquired resistance to \\nKRASG12C inhibitors remain undetermined. To begin to define the mechanistic spectrum of acquired', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 0, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='resistance, we describe a patient with KRASG12C NSCLC who developed polyclonal acquired resistance \\nto MRTX849 with the emergence of 10 heterogeneous resistance alterations in serial cell-free DNA \\nspanning four genes (KRAS, NRAS, BRAF, MAP2K1), all of which converge to reactivate RAS–MAPK \\nsignaling. Notably, a novel KRASY96D mutation affecting the switch-II pocket, to which MRTX849 and \\nother inactive-state inhibitors bind, was identified that interferes with key protein–drug interactions \\nand confers resistance to these inhibitors in engineered and patient-derived KRASG12C cancer models. \\nInterestingly, a novel, functionally distinct tricomplex KRASG12C active-state inhibitor RM-018 retained \\nthe ability to bind and inhibit KRASG12C/Y96D and could overcome resistance.\\nSignificance: In one of the first reports of clinical acquired resistance to KRASG12C inhibitors, our \\ndata suggest polyclonal RAS–MAPK reactivation as a central resistance mechanism. We also identify', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 0, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='a novel KRAS switch-II pocket mutation that impairs binding and drives resistance to inactive-state \\ninhibitors but is surmountable by a functionally distinct KRASG12C inhibitor.\\n1Massachusetts General Hospital Cancer Center and Department of \\n \\nMedicine, Harvard Medical School, Boston, Massachusetts. 2Guardant Health, \\nRedwood City, California. 3Department of Pathology, Massachusetts \\n \\nGeneral Hospital, Boston, Massachusetts.\\nNote: Supplementary data for this article are available at Cancer Discovery \\nOnline (http://cancerdiscovery.aacrjournals.org/).\\nN. Tanaka, J.J. Lin, and C. Li contributed equally to this work.\\nCorresponding Authors: Aaron N. Hata, Massachusetts General Hospital \\nCancer Center, 149 13th Street,  7th Floor, Boston, MA 02129. Phone: 617-\\n724-3442; E-mail: ahata@mgh.harvard.edu; Rebecca S. Heist, Massachu\\xad\\nsetts General Hospital Cancer Center, 55 Fruit Street, Boston, MA 02114. \\nPhone: 617-724-4000; E-mail: rheist@partners.org; and Ryan B. Corcoran,', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 0, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Massachusetts General Hospital Cancer Center, 149 13th Street, 7th Floor, \\nBoston, MA 02129. Phone: 617-726-8599; E-mail: rbcorcoran@partners.org\\nCancer Discov 2021;11:1–10\\ndoi: 10.1158/2159-8290.CD-21-0365\\n©2021 American Association for Cancer Research\\nINTRODUCTION\\nThe development of compounds that bind covalently to \\ncysteine 12 in KRASG12C cancers has ushered in a new era in \\nefforts to target KRAS directly. Biochemically, these agents \\nlock KRAS in its inactive GDP-bound conformation, thereby \\ninhibiting downstream signaling, leading to preclinical \\n \\nantitumor responses (1–3). The lead clinical compounds soto-\\nrasib (AMG 510) and adagrasib (MRTX849) have advanced \\nrapidly and demonstrated tolerability and single-agent \\nactivity across KRASG12C-mutant cancers (4, 5). In patients \\nwith advanced non–small cell lung cancer (NSCLC) harbor-\\ning KRASG12C (which comprises approximately 13% of all \\n \\nlung adenocarcinomas), AMG 510 and MRTX849 have \\n \\nResearch.', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 0, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Research. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 0, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF2\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\ndemonstrated meaningful efficacy with objective response \\nrates of 37% and 45%, as well as disease control rates of 81% \\nand 96%, respectively (6, 7). AMG 510 has recently received \\nBreakthrough Therapy designation from the FDA for the \\ntreatment of patients with advanced KRASG12C-mutant \\nNSCLC following at least one prior systemic therapy. Mul-\\ntiple ongoing trials seek to augment responses to KRASG12C \\ninhibitors through combination strategies.\\nPreclinical studies with MRTX849 and other KRASG12C \\ninhibitors have suggested several mechanisms of up-front \\nresistance, including reactivation of ERK-dependent sig\\xad\\nnaling to bypass KRASG12C blockade (4). Prior work by our \\ngroup and others has identified adaptive RAS pathway feed-\\nback reactivation as a key mechanism of primary resistance \\nto KRASG12C inhibition (8–13). However, the key mecha-\\nnisms of clinical acquired resistance to KRASG12C inhibitors', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='are currently unknown. Here, as an initial effort to charac-\\nterize the clinical landscape of potential acquired resistance \\nmechanisms to KRASG12C inhibitors, we present a patient \\nwith KRASG12C-mutant NSCLC who developed acquired \\nresistance to MRTX849, characterized by the emergence \\nof 10 individual resistance alterations involving four RAS–\\nMAPK genes. All of these resistance alterations converge \\nto reactivate RAS–MAPK signaling, implicating this as a \\npotential central mechanism of acquired resistance. We \\nalso identify a novel KRASY96D resistance mutation in the \\nswitch-II pocket of KRAS through serial cell-free DNA \\n(cfDNA) analysis. Through structural modeling and in vitro \\nfunctional studies, we find that KRASY96D confers resist-\\nance to multiple KRASG12C inhibitors currently in clinical \\ndevelopment but identify a novel active-state KRASG12C \\ninhibitor, RM-018, that is able to overcome KRASG12C/Y96D-\\nmediated resistance.\\nRESULTS', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='mediated resistance.\\nRESULTS\\nHeterogeneous Acquired Resistance Alterations \\nConverge on RAS–MAPK Reactivation\\nA 67-year-old woman with metastatic KRASG12C-mutant \\nNSCLC was treated on the dose-expansion cohort of the \\nphase 1 study of MRTX849 (NCT03785249; Methods; fur-\\nther detailed in Supplementary Methods). Initial scans \\nshowed a 32% reduction in tumor size (by RECIST v1.1), \\nbut after approximately 4 months of treatment, the patient \\ndeveloped progressive disease, and the patient discontinued \\ntherapy at 5.5 months (Fig. 1A). To identify putative mecha-\\nnisms of acquired resistance to MRTX849 in this patient, we \\nassessed cfDNA using a targeted next-generation sequencing \\nassay (Guardant360; Guardant Health) and droplet digital \\nPCR (ddPCR). Upon development of acquired resistance, the \\noriginal KRASG12C and TP53F338fs variants present in pretreat-\\nment tumor and cfDNA were again detected in cfDNA but \\nwere accompanied by the emergence of 10 distinct muta-', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='tions affecting RAS–MAPK components KRAS, NRAS, BRAF, \\nand MAP2K1 (which encodes the MEK1 protein) identified \\nacross cfDNA specimens obtained after disease progression \\n(Fig. 1B; Supplementary Table S1). The lower allele frequen-\\ncies of these alterations relative to the truncal KRASG12C \\nand TP53 mutations are consistent with the emergence of \\nthese mutations in heterogeneous subclonal populations. \\nThese included three activating NRAS mutations (NRASQ61L, \\nNRASQ61K, NRASQ61R), which can drive active RAS signaling \\nin a KRAS-independent manner, and BRAFV600E, which can \\nmaintain MAPK signaling downstream of KRASG12C in the \\npresence of MRTX849 (Supplementary Fig. S1). Three MAP2K1  \\nmutations (MAP2K1K57N, MAP2K1Q56P, MAP2K1E102-I103del) pre\\xad\\nviously demonstrated to be activating and known to be \\ninvolved in resistance to upstream MAPK pathway inhibitors \\n(i.e., BRAF inhibitors) were also identified (14, 15).\\nIn addition, three KRAS mutations emerged in the postpro-', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='gression cfDNA. Two of these mutations are the known activat-\\ning mutations KRASG13D and KRASG12V, and mutant-selective \\nKRASG12C inhibitors have previously been shown to be inef-\\nfective against these mutations (4, 16). A deeper analysis of \\nindividual sequencing reads from cfDNA suggested that these \\nmutations seemed to occur in trans to the original KRASG12C \\nmutation (Supplementary Fig. S2A and S2B), likely arising \\nin the remaining wild-type copy of KRAS, which appeared to \\nbe retained based on pretreatment tumor sequencing (Sup-\\nplementary Table S2). However, it is not possible from the \\ncfDNA data to confirm that these mutations coexist in cells \\nthat also harbor the original KRASG12C mutation. Notably, a \\nsingle, well-supported family of sequencing reads from the \\nsame original template molecule showed the concurrent pres-\\nence of both nucleotide changes corresponding to KRASG12C \\nand KRASG12V in cis on the same strand, which would encode', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='for a KRASG12F mutation. While it is not possible to confirm \\nthe presence of this mutation based on a single read family, \\nthis finding raises the possibility that cis mutations resulting \\nin “loss” of the original KRASG12C mutation and conversion \\nto a different KRAS mutation might be another potential \\nmechanism of resistance. Notably, all putative resistance \\nmutations identified are predicted to converge on reactiva-\\ntion of RAS–MAPK pathway signaling, suggesting that this \\nmay represent a common primary mechanism of acquired \\nresistance to KRASG12C inhibitors (Fig. 1C).\\nInterestingly, the third KRAS mutation identified, KRASY96D, \\nrepresents a novel mutation that is not known to be activat-\\ning. Notably, although KRAS is the most commonly mutated \\noncogene in human cancer, a search of two large tumor muta-\\ntional databases—COSMIC and GENIE, which collectively \\ncontain >450,000 molecularly characterized cancers (17, \\n18)—did not reveal a single previously identified mutation at', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='the KRASY96 locus among >75,000 cases with documented \\nKRAS mutations (Supplementary Table S3). However, the \\nY96 residue is associated with the switch-II pocket to which \\nMRTX849 and other inactive-state KRASG12C inhibitors bind, \\nsuggesting that the previously undescribed Y96D mutation \\nmay have a novel and specific role in driving resistance to \\nKRASG12C inhibitors.\\nStructural Modeling of KRASG12C/Y96D\\nTo understand the significance of the acquired KRASY96D \\nmutation, we performed structural modeling of the G12C-\\nmutant and G12C/Y96D double-mutant KRAS proteins \\nbound to the KRASG12C inhibitors MRTX849, AMG 510, and \\nARS-1620 (Fig. 2). These three inhibitors bind the GDP state \\nof KRASG12C and exploit a cryptic pocket formed by the central \\nβ sheet of RAS and switch-II (first identified by Ostrem and \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Downloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 1, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Clinical Acquired Resistance to KRASG12C Inhibition\\nRESEARCH BRIEF\\n\\t\\nAUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF3 \\nFigure 1.\\u2002Acquired resistance to KRASG12C inhibitor MRTX849 (adagrasib). A, Computed tomography images of the patient’s axillary lymph node \\nmetastasis at baseline, during response to MRTX849, and at progression on MRTX849. B, Variant allele fractions of mutations detected in the patient’s \\nserial plasma samples. †, indicates the mutations were detected by ddPCR but not by plasma next-generation sequencing. C, Alterations detected in post-\\nMRTX849 cfDNA include acquired mutations in KRAS as well as multiple components of the MAPK signaling cascade. *, KRASG12F represents a potential \\nresistance mechanism supported by limited sequencing reads, as shown in Supplementary Fig. S2.\\nA\\nBaseline \\n6 weeks \\n4 months \\nMRTX849 \\nPre-\\nMRTX849\\nPre-\\nMRTX849\\nDays post-MRTX849\\ndiscontinuation:\\n0\\n9\\n51\\nTP53 F338fs\\n36.8%\\n0.22%\\n8.8%\\n10.1%\\n14.3%\\nKRAS G12C\\n21.3%\\n0.12%\\n31.7%\\n47.1%\\n24.9%\\nKRAS G12V\\n-\\n-\\n-\\n-', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 2, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='21.3%\\n0.12%\\n31.7%\\n47.1%\\n24.9%\\nKRAS G12V\\n-\\n-\\n-\\n-\\n0.09%\\nKRAS G13D\\n-\\n-\\n-\\n0.13%†\\n0.04%\\nKRAS Y96D\\n-\\n-\\n0.4%\\n0.2%\\n-\\nNRAS Q61L\\n-\\n-\\n-\\n0.2%\\n-\\nNRAS Q61R\\n-\\n-\\n-\\n-\\n0.02%\\nNRAS Q61K\\n-\\n-\\n0.6%\\n0.6%\\n0.9%\\nBRAF V600E\\n-\\n-\\n0.1%\\n0.1%\\n0.5%\\nMAP2K1 K57N\\n-\\n-\\n0.05%†\\n-\\n0.3%\\nMAP2K1 Q56P\\n-\\n-\\n-\\n-\\n0.1%\\nMAP2K1 E102_I103del\\n-\\n-\\n-\\n0.12%†\\n0.2%\\ncfDNA\\nTumor \\nB\\nC\\nA/B/ \\nCRAF \\nKRAS \\n G12C \\nMEK \\nRTK \\nProliferation and Survival  \\nC\\nRTK \\nWT \\nRAS \\nERK \\nNRAS Q61K \\nNRAS Q61R \\nNRAS Q61L \\nKRAS G12V \\nKRAS G13D \\nKRAS G12F* \\nKRAS Y96D \\nBRAF V600E \\nMAP2K1 K57N \\nMAP2K1 Q56P \\nMAP2K1 E102_I103 del \\nSecondary \\nCis* \\nTrans \\ncolleagues; ref. 3). To determine the effects of the amino acid \\nsubstitution at the Y96 locus, crystal structures of MRTX849, \\nAMG 510, and ARS-1620 bound to KRASG12C were modeled \\nfor interactions with the Y96 residue within the switch-II \\npocket (16, 19–21). The hydroxyl group of Y96 forms a direct \\nhydrogen bond with the pyrimidine ring of MTRX849, which \\nis abolished with the Y96D mutation. Y96D also disrupts the', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 2, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='water-mediated hydrogen bond between Y96 and a carboxyl \\ngroup on AMG 510. Finally, although Y96 does not form a \\ndirect hydrogen bond with ARS-1620, it stabilizes the inter-\\naction with ARS-1620 through pi-stacking with the phenyl \\nring of Y96, which is disrupted with the Y96D mutation. In \\naddition, by introducing a negatively charged amino acid, the \\nY96D mutation changes the hydrophobic nature of the bind-\\ning pocket for all three compounds to a substantially more \\nhydrophilic pocket, which may further destabilize binding.\\nFunctional Characterization of KRASY96D\\nTo assess whether KRASY96D can mediate resistance to \\nMRTX849 and other inactive-state KRASG12C inhibitors, we \\nexpressed KRASG12C or the KRASG12C/Y96D double mutant \\nin NCI-H358 (KRASG12C-mutant NSCLC), MIA PaCa-2 \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 2, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF4\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\nFigure 2.\\u2002Structural basis for resistance to KRASG12C inhibition conferred by KRASY96D. Shown are the modeled crystal structures of MRTX849 (6UT0), \\nAMG 510 (6OIM), and ARS-1620 (5V9U) bound to KRASG12C (top) and KRASG12C/Y96D (bottom), highlighting the loss of the hydrogen bonds between \\nMRTX849 or AMG 510 and the Y96 residue and the disruption of the switch-II pocket dynamics between ARS-1620 and KRASG12C/Y96D.\\nARS-1620\\nMRTX849\\nAMG 510\\nG12C\\nG12C/Y96D\\n(KRASG12C-mutant pancreatic ductal adenocarcinoma), and \\nBa/F3 cells, which lack endogenous KRASG12C but become \\noncogene dependent upon withdrawal of IL3. In cell via-\\nbility assays, relative to KRASG12C-expressing controls, cells \\nexpressing KRASG12C/Y96D showed marked resistance to \\nthree KRASG12C inhibitors, with IC50 shifts of >100-fold for \\nMRTX849 and AMG 510 and ∼20-fold for ARS-1620 (Fig. 3A; \\nSupplementary Table S4).', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Supplementary Table S4).\\nConsistent with the effects on cell viability, RAS–MAPK \\npathway activity, as measured by levels of phosphorylated \\n \\nERK (pERK) and pRSK, was sustained in KRASG12C/Y96D-\\nexpressing MIA PaCa-2 cells even at high concentrations \\nof MRTX849, relative to cells expressing KRASG12C alone \\n(Fig. 3B). Similarly, in KRASG12C-mutant NSCLC cells in \\nwhich PI3K signaling is driven by mutant KRAS, including \\nan existing patient-derived model MGH1138-1, persistent \\npERK and pAKT levels were observed with KRASG12C/Y96D \\nin the presence of MRTX849, relative to KRASG12C expres-\\nsion alone (Fig. 3C; Supplementary Fig. S3). KRASG12C/Y96D \\nalso drove marked resistance to MRTX849 in the patient-\\nderived MGH1138-1 model. Furthermore, in 293T cells, \\nwhich lack endogenous KRASG12C expression, MRTX849 \\nwas unable to inhibit pERK levels driven by KRASG12C/Y96D  \\n(Fig. 3D). Because MRTX849 and other inactive-state \\nKRASG12C inhibitors bind covalently to KRASG12C, an elec-', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='trophoretic mobility shift of drug-adducted KRASG12C can \\nbe observed upon drug binding due to increased molecular \\nweight. However, this mobility shift was no longer observed \\nwhen 293T cells expressing KRASG12C/Y96D were treated with \\nMRTX849, suggesting that the Y96D mutation may abro-\\ngate inhibitor binding. Notably, KRASG12C/Y96D appeared to \\nhave higher basal activation than KRASG12C, as measured \\nby a higher proportion of the active GTP-bound form of \\nKRAS, although activation still appeared to be partly depend-\\nent on upstream pathway input (Supplementary Fig. S4A \\nand S4B). Finally, although a decrease in GTP-bound KRAS \\n(representing the active state) was observed in KRASG12C-\\nexpressing cells treated with MRTX849, levels of active \\n \\nGTP-bound KRAS were maintained in KRASG12C/Y96D-expressing \\ncells (Fig. 3E; ref. 22). These results suggest that the KRASY96D \\nmutation disrupts KRASG12C inhibitor binding, leading to \\nsustained KRAS signaling and therapeutic resistance.', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='The Active State KRASG12C Inhibitor RM-018 \\nOvercomes KRASG12C/Y96D\\nAs KRASG12C/Y96D conferred resistance to multiple KRASG12C \\ninhibitors currently in clinical development, suggestive of \\nshared vulnerability for this class of inhibitors, we sought \\nto identify whether a structurally and functionally distinct \\nKRASG12C inhibitor might retain potency against this resist-\\nance mutation. RM-018 is a novel KRASG12C inhibitor that \\nbinds specifically to the GTP-bound, active [“RAS(ON)”] state \\nof KRASG12C. RM-018 is a “tricomplex” KRAS inhibitor, which \\nexploits a highly abundant chaperone protein, cyclophilin A, \\nto bind and inhibit KRASG12C, as previously described (Fig. 4A; \\n \\nstructure shown in Supplementary Fig. S5; refs. 23, 24). \\nBriefly, upon entering the cell, RM-018 forms a “binary com-\\nplex” with cyclophilin A. This binary complex can associate \\nwith the active state of KRASG12C, aided by protein–protein \\nsurface interactions between cyclophilin A and KRAS, and', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='forms a covalent bond with KRASG12C in a mutant-selec-\\ntive manner. This resultant “tricomplex” inhibits KRASG12C \\nthrough binding of cyclophilin A, leading to steric occlu-\\nsion and preventing the association of downstream effector \\nproteins. Given the markedly different mechanism of action \\nof this class of inhibitor, we hypothesized that the inhibitory \\nactivity of RM-018 might be differentially affected by KRASY96D \\ncompared with inactive-state KRASG12C inhibitors.\\nRM-018 demonstrated selectivity for KRASG12C-driven \\ncells, exhibiting low nanomolar potency in KRASG12C-mutant \\nH358 cells while not impairing the viability of cells driven \\nby KRASG12D, BRAFV600E, or RTK-driven signaling through \\nwild-type RAS (Fig. 4B). Interestingly, although KRASG12C/Y96D \\n \\nexpression led to marked IC50 shifts of >100-fold for \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Downloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 3, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Clinical Acquired Resistance to KRASG12C Inhibition\\nRESEARCH BRIEF\\n\\t\\nAUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF5 \\nFigure 3.\\u2002Cellular characterization of KRASY96D in KRASG12C-mutant models. A, Cell viability assays performed with NCI-H358, MIA PaCa-2, and Ba/F3 \\ncells infected with retrovirus packaging KRAS (G12C or G12C/Y96D). Cell lines were treated with indicated drugs for 72 hours and the viabilities were \\nmeasured with CellTiter-Glo. B, Western blot analysis was performed after treating MIA PaCa-2 cells stably expressing KRASG12C or KRASG12C/Y96D with \\nMRTX849 for 4 hours. C, MGH1138-1 cells expressing KRASG12C or KRASG12C/Y96D were treated with MRTX849 for 4 hours and subjected to Western blot \\nanalysis (left) and cell viability assay following 72 hours of treatment with the indicated concentrations of MRTX849 (right). D, Western blot analysis \\nof HEK293T cells transiently expressing KRAS mutants after treatment with MRTX849 for 4 hours. E, RAS-GTP pulldown was performed after treating', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 4, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='HEK293T stably expressing KRAS mutants with MRTX849.\\nB\\n0 \\n3 10 30 100 300 0 \\n3 10 30 100 300 \\nG12C \\nG12C/Y96D \\nMRTX849\\n(nmol/L):\\nMIA PaCa-2 \\npERK\\npRSK\\nGAPDH\\nE\\nRAF-RBD \\nInput \\npERK\\npRSK\\nKRAS\\nKRAS-GTP\\nGAPDH\\nRAS\\nRAS-GTP\\nMRTX849\\n(nmol/L):\\nG12C \\n0 \\n3 10 \\n100 \\n30 \\n300 \\nG12C/Y96D \\n0 \\n3 10 \\n100 \\n30 \\n300 \\n293T \\nKRAS\\npERK\\npRSK\\n0 \\n3 10 30 100 300 0 \\n3 10 30 100 300 \\nMRTX849\\n(nmol/L):\\nG12C \\nG12C/Y96D \\nGAPDH\\nD\\nA\\n293T \\nC\\nAMG 510 \\nMRTX849 \\nRelative Cell Titer (%) \\nLog ARS-1620 (mol/L) \\nARS-1620 \\nH358 \\nMIA PaCa-2 \\nKRASG12C\\nKRASG12C/Y96D \\nBaF3 \\nLog ARS-1620 (mol/L) \\nLog AMG 510 (mol/L) \\nLog MRTX849 (mol/L) \\nLog ARS-1620 (mol/L) \\nLog MRTX849 (mol/L) \\nLog AMG 510 (mol/L) \\nLog AMG 510 (mol/L) \\npERK\\nERK \\nMRTX849\\n(nmol/L):\\nGAPDH\\npAKT\\nAKT\\n0 \\n30 300 0 30 300 \\nG12C \\nG12C/Y96D \\nMGH1138-1 \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n150\\n100\\n50\\n0\\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\nLog MRTX849 (mol/L) \\n−11\\n−12\\n−11\\n−10', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 4, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='−9\\n−8\\n−7\\n−6\\nLog MRTX849 (mol/L) \\n−11\\n−12\\n−11\\n−10\\n−9\\n−8\\n−7\\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\n−10\\n−9\\n−8\\n−7\\n−6\\n−5\\n−10\\n−9\\n−8\\n−7\\n−6\\n−11\\n−10\\n−10\\n−9\\n−8\\n−7\\n−6\\n−5\\n−9\\n−8\\n−7\\n−6\\n−11\\n−10\\n−10\\n−9\\n−8\\n−7\\n−6\\n−5\\n−9\\n−8\\n−7\\n−6\\nKRASG12C \\nKRASG12C/Y96D \\n−9\\n−8\\n−7\\n−6\\n−5\\n0\\n50\\n100\\n150\\nLog MRTX849 (mol/L)\\nRelative Cell Titer (%)\\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 4, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF6\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\nFigure 4.\\u2002Novel KRAS inhibitor RM-018 overcomes KRASG12C/Y96D. A, Mechanism of action of RM-018. B, RM-018 selectively inhibits cell viability in \\ncells harboring KRASG12C. C, Cell viability assays performed with NCI-H358, MIA PaCa-2, Ba/F3, and MGH1138-1 cells stably infected with KRASG12C or \\nKRASG12C/Y96D treated for 72 hours with RM-018. D and E, Western blot analysis performed in MIA PaCa-2 stably expressing KRASG12C or KRASG12C/Y96D (D) \\nand HEK293T cells transiently expressing KRAS mutants (E) after treatment of RM-018 for 4 hours. F, Western blot analysis of MGH1138-1 cells transiently \\nexpressing KRASG12C or KRASG12C/Y96D after treatment with RM-018 for 4 hours. G, HEK293T cells transiently expressing KRAS mutants were treated with \\nthe indicated drug at 100 nmol/L each for 4 hours and then subjected to Western blot analysis.\\n150\\n100\\n50\\n0\\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\nA\\nB\\nC\\nRM-018 \\nMIA PaCa-2 \\nBaF3', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 5, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='−10\\n−9\\n−8\\n−7\\n−6\\nA\\nB\\nC\\nRM-018 \\nMIA PaCa-2 \\nBaF3 \\nKRASG12C \\nKRASG12C/Y96D \\nLog RM-018 (mol/L) \\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\nLog RM-018 (mol/L) \\n−11\\n−10\\n−9\\n−8\\n−7\\n−6\\nLog RM-018 (mol/L) \\n−9\\n−8\\n−7\\n−6\\n−5\\nLog RM-018 (mol/L) \\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\n150\\n100\\n50\\n0\\nRelative Cell Titer (%) \\nH358 \\nD\\nMIA PaCa-2 \\n0 \\n1 \\n3 \\n10 30 100 0 \\n1 \\n3 \\n10 30 100 \\nG12C \\nG12C/Y96D \\nRM-018\\n(nmol/L):\\npERK\\npRSK\\nGAPDH\\nKRAS\\nG\\nE\\n0 \\n3 \\n10 30 100 \\n0 \\n3 10 \\n30 100 \\nRM-018\\n (nmol/L):\\nG12C \\nG12C/Y96D \\n1 \\n1 \\npERK\\npRSK\\nGAPDH\\nKRAS\\n293T \\nKRAS\\npERK\\npRSK\\nGAPDH\\nDMSO \\nMRTX849 \\nAMG 510 \\nRM-018 \\nDMSO \\nMRTX849 \\nAMG 510 \\nRM-018 \\nG12C \\nG12C/Y96D \\n293T \\n150\\nInhibitor\\nCYPA\\nKRASG12C(ON)\\n100\\n50\\n0\\n−10\\n−9\\n−8\\n−7\\n−6\\nNCI-H358 (KRASG12C) \\nAsPC-1 (KRASG12D) \\nNCI-H1975 (EGFRT790M,L858R, KRASWT) \\nA375 (BRAFV600E) \\nRelative Cell Titer (%) \\nLog RM-018 (mol/L) \\nMGH1138-1 \\nKRAS\\npERK\\nGAPDH\\npRSK\\n0 \\n10 \\n30 100 300 1,000 0 10 30 100 300 1,000 \\nG12C \\nG12C/Y96D \\nRM-018\\n(nmol/L):\\nF \\nMGH 1138-1', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 5, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='G12C \\nG12C/Y96D \\nRM-018\\n(nmol/L):\\nF \\nMGH 1138-1 \\nMRTX849 and AMG 510 and ∼20-fold for ARS-1620 (Fig. 3A) \\nrelative to KRASG12C expression alone, the efficacy of RM-018 \\non cell viability was largely unaffected by KRASG12C/Y96D  \\nexpression, with IC50 shifts of only ∼2-fold (Fig. 4C; Sup-\\nplementary Table S4). In addition, RM-018 was able to \\ninhibit pERK and pRSK levels with similar potency in the \\npresence of KRASG12C or KRASG12C/Y96D expression in MIA \\nPaCa-2, 293T cells, and the patient-derived KRASG12C-mutant \\nNSCLC cell line MGH1138-1 (Fig. 4D–F). Inhibition of cell \\nviability by RM-018 was also unaffected by KRASG12C/Y96D  \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 5, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Clinical Acquired Resistance to KRASG12C Inhibition\\nRESEARCH BRIEF\\n\\t\\nAUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF7 \\nexpression in the patient-derived MGH1138-1 model. Fur-\\nthermore, the KRAS mobility shift induced by covalent bind-\\ning of RM-018 was observed in both cell lines in the presence \\nof either KRASG12C or KRASG12C/Y96D expression, suggesting \\nthat binding of RM-018 to KRAS is not abrogated by the \\nKRASY96D mutation. Indeed, although a KRAS mobility shift \\ndue to covalent drug binding was observed in 293T cells \\nexpressing KRASG12C for MRTX849, AMG 510, and RM-018, \\nonly RM-018 exhibited this same mobility shift and was \\nable to inhibit downstream signaling in the presence of the \\nKRASG12C/Y96D mutation (Fig. 4G). Taken together, these data \\nsuggest that RM-018 retains the ability to bind and inhibit \\nKRASG12C/Y96D and may represent a potential therapeutic \\nstrategy to overcome this acquired resistance mechanism.\\nDISCUSSION\\nThe arrival of covalent KRASG12C-selective inhibitors in the', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='clinic and early signs of activity demonstrated by MRTX849 \\nand AMG 510 have generated great enthusiasm (4, 5). How-\\never, our experiences across targeted therapies in lung cancer \\nand other cancers collectively demonstrate that acquired \\nresistance to the KRASG12C inhibitors will represent an inevi-\\ntable challenge going forward. Although preclinical studies \\nhave nominated putative mechanisms of up-front resistance, \\nincluding RAS–MAPK pathway reactivation (4, 8), mecha-\\nnisms of acquired resistance to MRTX849 or AMG 510 caus-\\ning disease relapse in patients remain unknown.\\nThe acquired resistance demonstrated in this patient is \\ninstructive in highlighting several points. First, 10 distinct \\nresistance alterations arose in this patient, all converging on \\nthe reactivation of RAS–MAPK signaling, suggesting that this \\nmay be a central common mechanism of acquired resistance. \\nRAS reactivation occurred by multiple different mechanisms,', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='including (i) activation of another RAS isoform (in this case, \\nNRAS); (ii) other KRAS activating mutations in trans (G13D, \\nG12V); (iii) potential loss of KRASG12C through a mutational \\nswitch to a different KRAS mutation in cis, although sup-\\nported by limited sequencing reads; and (iv) a novel secondary \\nalteration in KRAS (i.e., Y96D), which alters inhibitor binding.\\nStructural modeling predicted that this KRASY96D muta-\\ntion disrupts critical hydrogen bonding between the Y96 \\nresidue of KRAS and MRTX849. Importantly, we found that \\nthis KRASY96D mutation conferred resistance not only to \\nMRTX849 but also to additional KRASG12C-selective inhibi-\\ntors in clinical development, AMG 510 and ARS-1620, high-\\nlighting that mutations affecting the Y96 residue of KRAS \\nmay represent a shared vulnerability for the currently avail-\\nable KRASG12C inhibitors. Continued investigation into clini-\\ncal mechanisms of resistance to KRASG12C inhibitors in larger', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='cohorts of patients will be required to define the spectrum \\nand frequency of KRASY96 mutations (as well as other on-\\ntarget mutations). Nonetheless, in light of our observations, \\nit may be necessary to develop novel compounds that are able \\nto target KRASG12C/Y96D to overcome resistance to KRASG12C \\ninhibitors in clinic. RM-018 was identified as a novel KRASG12C \\ninhibitor with a distinct mechanism of action for targeting \\nKRAS, which was uniquely able to overcome KRASG12C/Y96D \\nacross multiple models. These results suggest that a novel, \\ndistinct KRAS inhibitor could theoretically be used to target \\n \\nresistance after an acquired KRAS resistance mutation \\nemerges on the initial KRASG12C inhibitor, and they support \\nefforts toward rational design of next-generation inhibitors.\\nOf note, however, the abundance of mutations down-\\nstream of RAS suggests that MAPK reactivation alone may \\nbe sufficient to drive resistance in at least some KRASG12C', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='cancers. At the time of disease progression on MRTX849, \\nthis patient’s plasma biopsy specimen revealed a low allele \\nfraction of KRASY96D, particularly compared with the sig-\\nnificantly higher allele fraction of KRASG12C (0.4% vs. 31.7%, \\nrespectively), suggestive of the presence of polyclonal resist-\\nance to MRTX849 (with KRASG12C/Y96D-harboring tumor cells \\nrepresenting a minor subclone). Concordant with this notion \\nand prior reports based on preclinical models, subsequent \\nposttreatment plasma samples in this case revealed numer-\\nous mutations affecting various nodes of the RAS–MAPK \\npathway (including NRAS, BRAF, and MAP2K1 mutations). \\nThus, rational combinations of KRASG12C inhibitors with \\ndownstream MAPK pathway inhibitors may be needed to suc-\\ncessfully prevent or overcome resistance. Although analysis \\nof additional patients is clearly needed to more fully define \\nthe spectrum of potential acquired resistance mechanisms', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='to KRASG12C inhibitors, this study begins to delineate some \\nof the potential resistance alterations that may be observed \\nclinically. Ultimately, an iterative discovery process of identi-\\nfying mechanisms of resistance and validating in vitro with a \\ndetailed structural and molecular understanding should help \\nadvance the development of novel strategies for therapeutic \\ntargeting of KRAS-mutant cancers.\\nMETHODS\\nPatient Treatment and Specimen Collection\\nThe patient was treated with MRTX849 dosed 600 mg twice \\ndaily on the phase I study (KRYSTAL-1) after providing written \\ninformed consent (ClinicalTrials.gov identifier: NCT03785249). She \\nhad received two prior lines of therapy. All pre- and posttreat-\\nment biopsies and genotyping were performed in accordance with \\nthe Massachusetts General Hospital (MGH) institutional review \\nboard–approved protocol and in accordance with the Declaration \\nof Helsinki. The pretreatment tumor specimen was analyzed using', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='the MGH SNaPshot next-generation sequencing assay (25). All \\ncfDNA samples were sequenced using the commercially available \\nGuardant360 assay (Guardant Health). More detailed patient history \\nis available in the Supplement.\\nCell Lines and Reagents\\nBa/F3 cells were obtained from the RIKEN BRC Cell Bank (RIKEN \\nBioResource Center). MGH1138-1 cells were generated from a \\npatient with KRASG12C-mutant NSCLC using methods that have \\nbeen previously described (26). Prior to cell line generation, the \\npatient provided written informed consent to participate in a Dana-\\nFarber/Harvard Cancer Center institutional review board–approved \\nprotocol giving permission for research to be performed on their \\nsample. The remaining cell lines were obtained from ATCC or the \\nCenter for Molecular Therapeutics at the MGH Cancer Center, which \\nroutinely performs cell line authentication testing by SNP and short-\\ntandem repeat analysis. HEK293T cells were maintained in DMEM', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='supplemented with 10% FBS. MIA PaCa-2 and NCI-H358 cells were \\nmaintained in DMEM/F12 supplemented with 10% FBS. LU-65 and \\nMGH1138-1 cells were maintained in RPMI supplemented with \\n10% FBS. Ba/F3 cells were maintained in DMEM supplemented \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 6, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF8\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\nwith 10% FBS and 10 ng/mL IL3. The KRAS (G12C or G12C/Y96D) \\ngene was inserted in pMXs-Puro Retroviral Expression Vector, which \\nwas purchased from Cell Biolabs. Retrovirus packaging mutated \\nKRAS genes were produced with HEK293T cells. After concentra-\\ntion of virus with Retro-Concentin Retro Concentration Reagent \\n \\n(System Biosciences), MIA PaCa-2, NCI-H358, and Ba/F3 cells were \\ninfected with the virus packaging either the KRASG12C or KRASG12C/Y96D \\ngene. After 48 hours of incubation, the cells were treated with puro-\\nmycin (1–2 μg/mL) for another 48 hours. IL3 was withdrawn to \\nselect for Ba/F3 cells dependent on mutant KRAS signaling after 48 \\nhours of puromycin treatment. The remaining cells were maintained \\nin media supplemented with puromycin. For transient expression \\nexperiments, a day after seeding the cells, pMXs-Puro-KRASG12C or', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='pMXs-Puro-KRASG12C/Y96D vectors were induced with Lipofectamine \\n2000 Transfection Reagent (ThermoFisher Scientific) following the \\nmanufacturer’s protocol. After 16 to 24 hours of incubation, cells \\nwere treated with inhibitors for 4 hours. AMG 510 was purchased \\nfrom MedChemExpress. MRTX849 and ARS-1620 were purchased \\nfrom Selleck Chemicals. RM-018 was provided by Revolution Medi-\\ncines, and details of the chemical synthesis can be found in Interna-\\ntional Patent Application No. PCT/US2020/058841.\\nCell Viability Assays\\nCell lines were seeded in a 96-well plate at 2 to 10 × 103 cells/well \\ndepending on cell lines and after 24 hours treated with a serial dilu-\\ntion of drugs and incubated for 72 hours. Cell viability was measured \\nwith CellTiter-Glo (Promega).\\nWestern Blot Analysis\\nCell lines were treated with MRTX849, AMG 510, or RM-018 \\nfor 4 hours and lysates were prepared as described previously (27). \\nAll antibodies were diluted in 5% bovine serum albumin as fol-', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='lows: KRAS (Sigma), pERK (Thr202/Tyr204,1:1,000; Cell Signal-\\ning Technology), p44/42 MAPK (ERK1/2) (1:1,000; Cell Signaling \\n \\nTechnology), phospho-RSK1 (T359+S363, 1:1,000; Abcam), phospho- \\nAKT (Ser473, 1:1,000; Cell Signaling Technology), AKT (1:1,000; Cell \\nSignaling Technology), and GAPDH (1:1,000; MilliporeSigma).\\nRAS-GTP Pulldown\\nAfter indicated inhibitor treatment, RAS activity was assessed by \\nGST-RAF-RBD pulldown (Cell Signaling Technology), followed by \\nWestern blot analysis with pan-RAS or RAS isoform–specific antibod-\\nies. Pulldown samples and whole-cell lysates were resolved on 4% to \\n12% Bis-Tris gels, and Western blotting was performed using antibod-\\nies against KRAS (Sigma) and pan-RAS (Cell Signaling Technology).\\nStructural Modeling\\nPublicly available crystal structures of KRASG12C in complex with \\nMRTX849 (PDB:6UT0), AMG 510 (PDB:6OIM), and ARS-1620 \\n(PDB:5V9U) were downloaded from the RCSB Protein Data Bank', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='(PDB; ref. 28). Structures were rendered in PyMol (The PyMOL \\nMolecular Graphics System) and analyzed for hydrogen bonds and \\nother molecular interactions between the KRASG12C inhibitors and \\nthe KRAS protein. Structures of Y96 amino acid mutation were gen-\\nerated by Protein Mutagenesis Wizard implemented in PyMol, with \\none of the backbone-dependent rotamers manually selected.\\ncfDNA Extraction and ddPCR\\nWhole blood was collected by routine phlebotomy in two 10-mL \\nStreck tubes. Plasma was separated within 1 to 4 days of collection \\nthrough two different centrifugation steps (the first at room temper-\\nature for 10 minutes at 1,600 × g and the second at 3,000 × g for the \\nsame time and temperature). Plasma was stored at −80°C until cfDNA \\nextraction. cfDNA was extracted from plasma using the QIAamp \\n \\nCirculating Nucleic Acid Kit (QIAGEN) with 60 minutes of protein-\\nase K incubation at 60°C. All other steps were performed according', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='to the manufacturer’s instructions. For ddPCR experiments, DNA \\ntemplate (up to 10 μL, with a total of 20 ng) was added to 12.5 μL \\nddPCR Supermix for Probes (Bio-Rad) and 1.25 μL custom primer/\\nprobe mixture. This reaction mix was added to a DG8 cartridge \\ntogether with 60 μL Droplet Generation Oil for Probes (Bio-Rad) \\nand used for droplet generation. Droplets were then transferred to a \\n96-well plate (Eppendorf) and then thermal cycled with the following \\nconditions: 5 minutes at 95°C, 40 cycles of 94°C for 30 seconds, 55°C \\n(with a few grades of difference among assays) for 1 minute, followed \\nby 98°C for 10 minutes (Ramp Rate 2°C/s). Droplets were analyzed \\nwith the QX200 Droplet Reader (Bio-Rad) for fluorescent measure-\\nment of FAM and HEX probes. Gating was performed based on posi-\\ntive and negative controls, and mutant populations were identified. \\nThe ddPCR data were analyzed with QuantaSoft analysis software', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='(Bio-Rad) to obtain fractional abundance of the mutant DNA alleles \\nin the wild-type/normal background. The quantification of the target \\nmolecule was presented as the number of total copies (mutant plus \\nwild-type) per sample in each reaction. Allelic fraction was calculated \\nas follows: AF % = [Nmut/(Nmut + Nwt) * 100], where Nmut is the num-\\nber of mutant alleles and Nwt is the number of wild-type alleles per \\nreaction. ddPCR analysis of normal control plasma DNA (from cell \\nlines) and no DNA template controls was always included. Probe and \\nprimer sequences are available upon request.\\nAuthors’ Disclosures\\nN. Tanaka reports other support from Japanese Foundation for \\nCancer Research during the conduct of the study. J.J. Lin reports \\npersonal fees and other support from Genentech, personal fees \\nfrom C4 Therapeutics, personal fees from Nuvalent, personal \\nfees from Blueprint Medicines, personal fees and other support \\nfrom Turning Point Therapeutics, personal fees and other sup-', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='port from Elevation Oncology, personal fees from Pfizer, other \\nsupport from Hengrui Therapeutics, other support from Novartis, \\nother support from Neon Therapeutics, other support from \\nRelay Therapeutics, other support from Bayer, and other support \\nfrom Roche outside the submitted work. L.A. Kiedrowski reports \\nother support from Guardant Health during the conduct of the \\nstudy. D. Juric reports grants and personal fees from Novartis, \\ngrants and personal fees from Genentech, grants and personal fees \\nfrom EMD Serono, grants and personal fees from Eisai, personal \\n \\nfees from Guardant, personal fees from Ipsen, personal fees from \\nPetra Pharma, personal fees from Mapkure, personal fees from Vib-\\nliome, personal fees from Relay Therapeutics, grants from Takeda, \\ngrants and personal fees from Syros, grants from Ribon Therapeu-\\ntics, grants from Infinity Pharmaceuticals, grants from InventisBio, \\n \\nand grants from Amgen outside the submitted work. J.F. Gainor', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='reports personal fees from Mirati and personal fees from Amgen dur-\\ning the conduct of the study, personal fees from Bristol-Myers Squibb, \\npersonal fees from Genentech/Roche, personal fees from Takeda, \\npersonal fees from Lilly, personal fees and other support from Blue-\\nprint, personal fees from Oncorus, personal fees from Regeneron, \\npersonal fees from Gilead, personal fees from AstraZeneca, personal \\nfees from Pfizer, grants and personal fees from Novartis, personal \\n \\nfees and other support from Merck, personal fees from Agios, other \\nsupport from Moderna, other support from Jounce, other support \\nfrom Array, other support from Alexo outside the submitted work, \\nand an immediate family member who is an employee with equity at \\nIronwood Pharmaceuticals. S.J. Klempner reports personal fees from \\nBMS, personal fees from Merck, personal fees from Daiichi-Sankyo, \\npersonal fees from Astellas, personal fees from Pieris Oncology,', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='personal fees from Natera, personal fees from Foundation Medicine, \\nand other support from Turning Point Therapeutics outside the \\nsubmitted work. L. Bar-Peled reports personal fees from Scorpion \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 7, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Clinical Acquired Resistance to KRASG12C Inhibition\\nRESEARCH BRIEF\\n\\t\\nAUGUST  2021\\u2003CANCER DISCOVERY\\u2002|\\u2002OF9 \\nTherapeutics outside the submitted work. A.N. Hata reports grants \\nfrom Amgen, grants from Eli Lilly, grants from Relay Therapeu-\\ntics, grants from Roche/Genentech, grants from Novartis, grants \\nfrom Blueprint Medicines, grants from Pfizer, and grants and per-\\nsonal fees from Nuvalent outside the submitted work. R.S. Heist \\nreports personal fees from Novartis, personal fees from Daichii \\n \\nSankyo, personal fees from EMD Serono, personal fees from Tarveda, \\npersonal fees from Apollomics, grants from Novartis, grants from \\nCorvus, grants from Incyte, grants from Genentech Roche, grants \\nfrom Mirati, grants from Turning Point, grants from BMS, grants from \\n \\nDaichii Sankyo, grants from AbbVie, grants from Agios, grants \\nfrom Exelixis, and grants from Lilly outside the submitted work. \\nR.B. Corcoran reports personal fees from AbbVie, grants and per-', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='sonal fees from Asana Biosciences, personal fees from Astex Phar-\\nmaceuticals, grants and personal fees from AstraZeneca, personal \\nfees and other support from Avidity Biosciences, personal fees and \\nother support from C4 Therapeutics, personal fees from Chugai, \\npersonal fees from Elicio, other support from Erasca, personal \\nfees from Fog Pharma, personal fees from Guardant Health, \\npersonal fees from Ipsen, personal fees and other support from \\n \\nKinnate Biopharma, personal fees from Mirati Therapeutics, personal \\n \\nfees from Natera, grants from Novartis, grants from Lilly, personal fees \\n \\nand other support from nRichDx, personal fees from Pfizer, personal \\nfees from QIAGEN, personal fees and other support from Remix \\nTherapeutics, personal fees and other support from Revolution \\nMedicines, personal fees from Roivant, personal fees from Shionogi, \\npersonal fees from Tango Therapeutics, personal fees from Taiho, \\nand personal fees from Zikani Therapeutics outside the submitted', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='work; in addition, R.B. Corcoran has a patent for Intellectual Prop-\\nerty pending. No disclosures were reported by the other authors.\\nAuthors’ Contributions\\nN. Tanaka: Conceptualization, data curation, formal analysis, \\ninvestigation, writing–original draft, writing–review and editing. \\n \\nJ.J. Lin: Conceptualization, data curation, formal analysis, inves-\\ntigation, writing–original draft, writing–review and editing. C. Li: \\nConceptualization, data curation, formal analysis, investigation, writing– \\noriginal draft, writing–review and editing. M.B. Ryan: Conceptual-\\nization, data curation, formal analysis, investigation, writing–original \\n \\ndraft, writing–review and editing. J. Zhang: Conceptualization, for-\\nmal analysis, writing–original draft, writing–review and editing. \\n \\nL.A. Kiedrowski: Conceptualization, formal analysis, writing–review \\nand editing. A.G. Michel: Formal analysis, investigation, writing–\\nreview and editing. M.U. Syed: Formal analysis, investigation, writing–', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='review and editing. K.A. Fella: Formal analysis, investigation, \\n \\nwriting–review and editing. M. Sakhi: Formal analysis, investigation, \\nwriting–review and editing. I. Baiev: Formal analysis, investiga-\\ntion, writing–review and editing. D. Juric: Conceptualization, formal \\n \\nanalysis, investigation, writing–review and editing. J.F. Gainor: Con-\\nceptualization, formal analysis, investigation, writing–review and editing. \\n \\nS.J. Klempner: Conceptualization, formal analysis, investigation, \\nwriting–review and editing. J.K. Lennerz: Formal analysis, investiga-\\ntion, writing–review and editing. G. Siravegna: Conceptualization, \\ndata curation, formal analysis, supervision, investigation, writing–\\noriginal draft, writing–review and editing. L. Bar-Peled: Conceptu-\\nalization, data curation, formal analysis, supervision, investigation, \\nwriting–review and editing. A.N. Hata: Conceptualization, resources, \\ndata curation, formal analysis, supervision, funding acquisition,', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='validation, investigation, writing–original draft, writing–review and \\nediting. R.S. Heist: Conceptualization, resources, data curation, formal \\nanalysis, supervision, funding acquisition, validation, investigation, \\nwriting–original draft, writing–review and editing. R.B. Corcoran:  \\nConceptualization, resources, data curation, formal analysis, supervi-\\nsion, funding acquisition, validation, investigation, writing–original \\ndraft, writing–review and editing.\\nAcknowledgments\\nRM-018 was kindly provided by Revolution Medicines. This \\nresearch was supported in part by a Mark Foundation for Cancer \\nResearch EXTOL Project Grant (J.F. Gainor, A.N. Hata) and a Stand \\nUp To Cancer—American Cancer Society Lung Cancer Dream Team \\nTranslational Research Grant (J.F. Gainor, A.N. Hata; grant number: \\nSU2C-AACR-DT17–15). Stand Up To Cancer (SU2C) is a division \\nof the Entertainment Industry Foundation. The indicated SU2C \\nresearch grant is administered by the American Association for', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Cancer Research, a scientific partner of SU2C. J.J. Lin is partially \\nsupported by funding from NIH R01-CA164273. C. Li is supported \\nby funding from NIH 1F32CA250231–01. N. Tanaka is partially \\nsupported by funding from US-JFCR Scientist Exchange Program. L. \\nBar-Peled is supported by AACR (19–20–45-BARP), Damon Runyon \\n(DRR-62–20), and NIH/NCI (R00 CA215249).\\nReceived March 19, 2021; revised March 30, 2021; accepted April 1, \\n2021; published first April 6, 2021.\\nReferences\\n\\t 1.\\t Fell JB, Fischer JP, Baer BR, Blake JF, Bouhana K, Briere DM, et al. \\nIdentification of the clinical development candidate MRTX849, a \\ncovalent KRAS(G12C) inhibitor for the treatment of cancer. J Med \\nChem 2020;63:6679–93.\\n\\t 2.\\t Lanman BA, Allen JR, Allen JG, Amegadzie AK, Ashton KS, Booker SK, \\n \\net al. Discovery of a covalent inhibitor of KRAS(G12C) (AMG 510) for \\nthe treatment of solid tumors. J Med Chem 2020;63:52–65.\\n\\t 3.\\t Ostrem JM, Peters U, Sos ML, Wells JA, Shokat KM. K-Ras(G12C)', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='inhibitors allosterically control GTP affinity and effector interac-\\ntions. Nature 2013;503:548–51.\\n\\t 4.\\t Hallin J, Engstrom LD, Hargis L, Calinisan A, Aranda R, Briere DM, \\net al. The KRAS(G12C) inhibitor MRTX849 provides insight toward \\ntherapeutic susceptibility of KRAS-mutant cancers in mouse models \\nand patients. Cancer Discov 2020;10:54–71.\\n\\t 5.\\t Hong DS, Fakih MG, Strickler JH, Desai J, Durm GA, Shapiro GI, et al. \\n \\nKRAS(G12C) inhibition with sotorasib in advanced solid tumors. \\n \\nN Engl J Med 2020;383:1207–17.\\n\\t 6.\\t Li B, Skoulidis F, Falchook G, Sacher A, Velcheti V, Dy G, et al. \\nPS01.07 registrational phase 2 trial of sotorasib in KRAS p.G12C \\nmutant NSCLC: first disclosure of the codebreak 100 primary analy-\\nsis. J Thorac Oncol 2021;16:S61.\\n\\t 7.\\t Jänne PA, Rybkin II, Spira AI, Riely GJ, Papadopoulos KP, Sabari JK, \\net al. KRYSTAL-1: activity and safety of adagrasib (MRTX849) in \\nadvanced/metastatic non–small-cell lung cancer (NSCLC) harboring', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='KRAS G12C mutation. Eur J Cancer 2020;138:S1–S2.\\n\\t 8.\\t Ryan MB, Fece de la Cruz F, Phat S, Myers DT, Wong E, Shahzade HA, \\net al. Vertical pathway inhibition overcomes adaptive feedback resist-\\nance to KRAS(G12C) inhibition. Clin Cancer Res 2020;26:1633–43.\\n\\t 9.\\t Misale S, Fatherree JP, Cortez E, Li C, Bilton S, Timonina D, et al. KRAS \\nG12C NSCLC models are sensitive to direct targeting of KRAS in com-\\nbination with PI3K inhibition. Clin Cancer Res 2019;25:796–807.\\n\\t\\n10.\\t Santana-Codina N, Chandhoke AS, Yu Q, Malachowska B, Kuljanin \\nM, Gikandi A, et al. Defining and targeting adaptations to oncogenic \\nKRAS(G12C) inhibition using quantitative temporal proteomics. Cell \\nRep 2020;30:4584–99.\\n\\t\\n11.\\t Solanki HS, Welsh EA, Fang B, Izumi V, Darville L, Stone B, et al. Cell \\ntype-specific adaptive signaling responses to KRAS(G12C) inhibition. \\nClin Cancer Res 2021;27:2533–48.\\n\\t\\n12.\\t Xue JY, Zhao Y, Aronowitz J, Mai TT, Vides A, Qeriqi B, et al. Rapid', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='non-uniform adaptation to conformation-specific KRAS(G12C) inhi-\\nbition. Nature 2020;577:421–5.\\n\\t\\n13.\\t Fedele C, Li S, Teng KW, Foster CJR, Peng D, Ran H, et al. SHP2 inhi-\\nbition diminishes KRASG12C cycling and promotes tumor microen-\\nvironment remodeling. J Exp Med 2021;218:e20201414.\\n\\t\\n14.\\t Kinoshita-Kikuta E, Kinoshita E, Ueda S, Ino Y, Kimura Y, Hirano H, \\net al. Increase in constitutively active MEK1 species by \\xad\\nintroduction \\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 8, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Tanaka et al.\\nRESEARCH BRIEF\\nOF10\\u2002|\\u2002CANCER DISCOVERY\\u2003AUGUST  2021\\t\\nAACRJournals.org\\nof MEK1 mutations identified in cancers. Biochim Biophys Acta \\n \\nProteins Proteom 2019;1867:62–70.\\n\\t\\n15.\\t Gao Y, Chang MT, McKay D, Na N, Zhou B, Yaeger R, et al. Allele-\\nspecific mechanisms of activation of MEK1 mutants determine their \\nproperties. Cancer Discov 2018;8:648–61.\\n\\t\\n16.\\t Canon J, Rex K, Saiki AY, Mohr C, Cooke K, Bagal D, et al. The clini-\\ncal KRAS(G12C) inhibitor AMG 510 drives anti-tumour immunity. \\nNature 2019;575:217–23.\\n\\t\\n17.\\t Sondka Z, Bamford S, Cole CG, Ward SA, Dunham I, Forbes SA. The \\nCOSMIC Cancer Gene Census: describing genetic dysfunction across \\nall human cancers. Nat Rev Cancer 2018;18:696–705.\\n\\t\\n18.\\t Consortium APG. AACR Project GENIE: powering precision medi-\\ncine through an international consortium. Cancer Discov 2017;7: \\n818–31.\\n\\t\\n19.\\t Fell JB, Fischer JP, Baer BR, Ballard J, Blake JF, Bouhana K, et al. \\nDiscovery of tetrahydropyridopyrimidines as irreversible covalent', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 9, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='inhibitors of KRAS-G12C with in vivo activity. ACS Med Chem Lett \\n2018;9:1230–4.\\n\\t\\n20.\\t Janes MR, Zhang J, Li LS, Hansen R, Peters U, Guo X, et al. Targeting \\nKRAS mutant cancers with a covalent G12C-specific inhibitor. Cell \\n2018;172:578–89.\\n\\t\\n21.\\t Chen H, Smaill JB, Liu T, Ding K, Lu X. Small-molecule inhibitors \\ndirectly targeting KRAS as anticancer therapeutics. J Med Chem \\n2020;63:14404–24.\\n\\t\\n22.\\t Zeng M, Lu J, Li L, Feru F, Quan C, Gero TW, et al. Potent and selec-\\ntive covalent quinazoline inhibitors of KRAS G12C. Cell Chem Biol \\n2017;24:1005–16.\\n\\t\\n23.\\t Schulze CJ, Bermingham A, Choy TJ, Cregg JJ, Kiss G, Marquez A, et al. \\nAbstract PR10: tri-complex inhibitors of the oncogenic, GTP-bound \\nform of KRASG12C overcome RTK-mediated escape mechanisms and \\ndrive tumor regressions in vivo. Mol Cancer Ther 2019;18:PR10.\\n\\t\\n24.\\t Nichols R, Schulze C, Bermingham A, Choy T, Cregg J, Kiss G, et al. \\nA06 Tri-complex inhibitors of the oncogenic, GTP-bound form of', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 9, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='KRASG12C overcome RTK-mediated escape mechanisms and drive \\ntumor regressions in preclinical models of NSCLC. J Thorac Oncol \\n2020;15:S13–S4.\\n\\t\\n25.\\t Zheng Z, Liebers M, Zhelyazkova B, Cao Y, Panditi D, Lynch KD, \\n \\net al. Anchored multiplex PCR for targeted next-generation sequenc-\\ning. Nat Med 2014;20:1479–84.\\n\\t\\n26.\\t Crystal AS, Shaw AT, Sequist LV, Friboulet L, Niederst MJ, Lockerman \\n \\nEL, et al. Patient-derived models of acquired resistance can identify \\neffective drug combinations for cancer. Science 2014;346:1480–6.\\n\\t\\n27.\\t Ahronian LG, Sennott EM, Van Allen EM, Wagle N, Kwak EL, Faris \\nJE, et al. Clinical acquired resistance to RAF inhibitor combinations \\nin BRAF-mutant colorectal cancer through MAPK pathway altera-\\ntions. Cancer Discov 2015;5:358–67.\\n\\t\\n28.\\t Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, \\net al. The protein data bank. Nucleic Acids Res 2000;28:235–42.\\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 9, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='cancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 9, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='Published OnlineFirst April 6, 2021.\\nCancer Discov \\n  \\nNoritaka Tanaka, Jessica J. Lin, Chendi Li, et al. \\n  \\nMAPK Reactivation\\n−\\nlterations Converging on RAS\\na Novel KRAS Switch-II Pocket Mutation and Polyclonal A\\nthrough\\n Inhibition \\nG12C\\nClinical Acquired Resistance to KRAS\\n  \\nUpdated version\\n  \\n \\n10.1158/2159-8290.CD-21-0365\\ndoi:\\nAccess the most recent version of this article at:\\n  \\nMaterial\\nSupplementary\\n  \\n \\nhttp://cancerdiscovery.aacrjournals.org/content/suppl/2021/04/03/2159-8290.CD-21-0365.DC1\\nAccess the most recent supplemental material at:\\n  \\n  \\n  \\n  \\n  \\n  \\n  \\nE-mail alerts\\n related to this article or journal.\\nSign up to receive free email-alerts\\n  \\nSubscriptions\\nReprints and \\n  \\n.\\npubs@aacr.org\\nDepartment at\\nTo order reprints of this article or to subscribe to the journal, contact the AACR Publications\\n  \\nPermissions\\n  \\nRightslink site. \\nClick on \"Request Permissions\" which will take you to the Copyright Clearance Center\\'s (CCC)\\n.', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 10, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''}),\n",
       " Document(page_content='.\\nhttp://cancerdiscovery.aacrjournals.org/content/early/2021/06/16/2159-8290.CD-21-0365\\nTo request permission to re-use all or part of this article, use this link\\nResearch. \\non July 28, 2021. © 2021 American Association for Cancer\\ncancerdiscovery.aacrjournals.org \\nDownloaded from \\nPublished OnlineFirst April 6, 2021; DOI: 10.1158/2159-8290.CD-21-0365', metadata={'source': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'file_path': '/Users/kkumbier/github/persisters/papers/Hata/2021_Cancer Discovery_Hata.pdf', 'page': 10, 'total_pages': 11, 'format': 'PDF 1.4', 'title': 'Clinical Acquired Resistance to KRASG12C Inhibition through a Novel KRAS Switch-II Pocket Mutation and Polyclonal Alterations Converging on RAS–MAPK Reactivation', 'author': '', 'subject': '', 'keywords': '', 'creator': 'Adobe InDesign CS6 (Windows)', 'producer': 'Adobe PDF Library 10.0.1', 'creationDate': \"D:20210616091024+05'30'\", 'modDate': \"D:20210728110726-07'00'\", 'trapped': ''})]"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "splitter.split_documents(pdf)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
